<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html id="nojs" xmlns="http://www.w3.org/1999/xhtml" xmlns:m="http://www.w3.org/1998/Math/MathML" xmlns:og="http://ogp.me/ns#" xml:lang="en-GB" lang="en-GB">	                                                                                                                                                                                                                                                         			                        <script>window.bmcIsMobile = "classic";</script>                    <head>    <title>BMC Bioinformatics | Full text | Inferring high-confidence human protein-protein interactions</title><link rel="shortcut icon" type="image/x-icon" href="/images/icon/1002/favicon.ico" /><meta http-equiv="X-UA-Compatible" content="IE=7" />                						<!-- <?xml version="1.0" encoding="UTF-8"?><rdf:RDF xmlns:cc="http://web.resource.org/cc/" xmlns="http://purl.org/rss/1.0/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:prism="http://prismstandard.org/namespaces/1.2/basic/"><cc:Work rdf:about="http://www.biomedcentral.com/1471-2105/13/79"><cc:license rdf:resource="http://creativecommons.org/licenses/by/2.0/"/></cc:Work><cc:License rdf:about="http://creativecommons.org/licenses/by/2.0/"><cc:permits rdf:resource="http://web.resource.org/cc/Reproduction"/><cc:permits rdf:resource="http://web.resource.org/cc/Distribution"/><cc:requires rdf:resource="http://web.resource.org/cc/Notice"/><cc:requires rdf:resource="http://web.resource.org/cc/Attribution"/><cc:permits rdf:resource="http://web.resource.org/cc/DerivativeWorks"/></cc:License><item rdf:about="http://www.biomedcentral.com/1471-2105/13/79"><title>Inferring high-confidence human protein-protein interactions</title><dc:title>Inferring high-confidence human protein-protein interactions</dc:title><dc:creator>Yu, Xueping</dc:creator><dc:creator>Wallqvist, Anders</dc:creator><dc:creator>Reifman, Jaques</dc:creator><dc:identifier>info:doi/10.1186/1471-2105-13-79</dc:identifier><dc:identifier>info:pmid/22558947</dc:identifier><dc:source>BMC Bioinformatics 2012, 13:79</dc:source><dc:date>2012-05-04</dc:date><prism:publicationName>BMC Bioinformatics</prism:publicationName><prism:publicationDate>2012-05-04</prism:publicationDate><prism:volume>13</prism:volume><prism:number>1</prism:number><prism:section>Methodology article</prism:section><prism:startingPage>79</prism:startingPage><prism:copyright>2012 Yu et al.; licensee BioMed Central Ltd.</prism:copyright></item></rdf:RDF> -->  <meta name="citation_journal_title" content="BMC Bioinformatics" /><meta name="citation_publisher" content="BioMed Central Ltd" />   <meta name="citation_author" content="Xueping Yu" />        <meta name="citation_author_institution" content="           Biotechnology High-Performance Computing Software Applications Institute, Telemedicine and Advanced Technology Research Center, U.S. Army Medical Research and Materiel Command, Ft. Detrick, MD, 21702, USA      " />    <meta name="citation_author" content="Anders Wallqvist" />        <meta name="citation_author_institution" content="           Biotechnology High-Performance Computing Software Applications Institute, Telemedicine and Advanced Technology Research Center, U.S. Army Medical Research and Materiel Command, Ft. Detrick, MD, 21702, USA      " />    <meta name="citation_author" content="Jaques Reifman" />        <meta name="citation_author_institution" content="           Biotechnology High-Performance Computing Software Applications Institute, Telemedicine and Advanced Technology Research Center, U.S. Army Medical Research and Materiel Command, Ft. Detrick, MD, 21702, USA      " />  <meta name="citation_title" content="Inferring high-confidence human protein-protein interactions" /><meta name="citation_volume" content="13" /><meta name="citation_issue" content="1" /><meta name="citation_date" content="2012-05-04" /><meta name="citation_firstpage" content="79" /><meta name="citation_doi" content="10.1186/1471-2105-13-79" /><meta name="title" content="Inferring high-confidence human protein-protein interactions" /><meta name="description" content="As numerous experimental factors drive the acquisition, identification, and interpretation of protein-protein interactions (PPIs), aggregated assemblies of human PPI data invariably contain experiment-dependent noise. Ascertaining the reliability of PPIs collected from these diverse studies and scoring them to infer high-confidence networks is a non-trivial task. Moreover, a large number of PPIs share the same number of reported occurrences, making it impossible to distinguish the reliability of these PPIs and rank-order them. For example, for the data analyzed here, we found that the majority (>83%) of currently available human PPIs have been reported only once." /><meta name="dc.title" content="Inferring high-confidence human protein-protein interactions" /><meta name="dc.creator" content="Yu, Xueping" /><meta name="dc.creator" content="Wallqvist, Anders" /><meta name="dc.creator" content="Reifman, Jaques" /><meta name="dc.description" content="As numerous experimental factors drive the acquisition, identification, and interpretation of protein-protein interactions (PPIs), aggregated assemblies of human PPI data invariably contain experiment-dependent noise. Ascertaining the reliability of PPIs collected from these diverse studies and scoring them to infer high-confidence networks is a non-trivial task. Moreover, a large number of PPIs share the same number of reported occurrences, making it impossible to distinguish the reliability of these PPIs and rank-order them. For example, for the data analyzed here, we found that the majority (>83%) of currently available human PPIs have been reported only once." /><meta name="dc.source" content="BMC Bioinformatics 2012 13:79" /><meta name="dc.format" content="text/html" /><meta name="dc.publisher" content="BioMed Central Ltd" /><meta name="dc.date" content="2012-05-04" /><meta name="dc.type" content="Methodology article" /><meta name="dc.identifier" content="10.1186/1471-2105-13-79" /><meta name="dc.language" content="en" /><meta name="dc.copyright" content="2012 Yu et al.; licensee BioMed Central Ltd." /><meta name="dc.rights" content="http://creativecommons.org/licenses/by/2.0/" /><meta name="dc.rightsAgent" content="reprints@biomedcentral.com" /><meta name="prism.issn" content="1471-2105" /><meta name="prism.publicationName" content="BMC Bioinformatics" /><meta name="prism.publicationDate" content="2012-05-04" /><meta name="prism.volume" content="13" /><meta name="prism.number" content="1" /><meta name="prism.section" content="Methodology article" /><meta name="prism.startingPage" content="79" /><meta name="prism.copyright" content="2012 Yu et al.; licensee BioMed Central Ltd." /><meta name="prism.rightsAgent" content="reprints@biomedcentral.com" /><link rel="alternate" type="application/pdf" title="PDF" href="http://www.biomedcentral.com/bmcbioinformatics/content/pdf/1471-2105-13-79.pdf" /><meta name="citation_pdf_url" content="http://www.biomedcentral.com/bmcbioinformatics/content/pdf/1471-2105-13-79.pdf" /><meta name="citation_abstract_html_url" content="http://www.biomedcentral.com/1471-2105/13/79/abstract" /><meta name="citation_fulltext_html_url" content="http://www.biomedcentral.com/1471-2105/13/79" /><meta name="citation_pdf_url" content="http://www.biomedcentral.com/bmcbioinformatics/content/pdf/1471-2105-13-79.pdf" /><link rel="alternate" type="application/pdf" title="PDF" href="http://www.biomedcentral.com/bmcbioinformatics/content/pdf/1471-2105-13-79.pdf" /><link rel='alternate' type='text/xml' title='XML version' href='http://www.biomedcentral.com/bmcbioinformatics/content/download/xml/1471-2105-13-79.xml' /><link rel="image_src" href="http://biomedcentral.com/bmcbioinformatics/content/figures/1471-2105-13-79-toc.gif" />    <link rel="section" href="http://www.biomedcentral.com/content/download/supplementary/1471-2105-13-79-s1.txt" title="Additional file 1" /><link rel="section" href="http://www.biomedcentral.com/content/download/supplementary/1471-2105-13-79-s2.txt" title="Additional file 2" /><link rel="section" href="http://www.biomedcentral.com/content/download/supplementary/1471-2105-13-79-s3.txt" title="Additional file 3" /><link rel="section" href="http://www.biomedcentral.com/content/download/supplementary/1471-2105-13-79-s4.txt" title="Additional file 4" /><link rel="section" href="http://www.biomedcentral.com/content/download/supplementary/1471-2105-13-79-s5.txt" title="Additional file 5" />                                <script type="text/javascript">// must run before cssdocument.documentElement.id = "js";</script>	                                                                                                                                                                                                                                                         			                        <script>window.bmcIsMobile = "classic";</script>            	                <link rel="stylesheet" type="text/css" href="/css/main-0.css" media="screen, print"/>                    <link rel="stylesheet" type="text/css" href="/css/plugins-0.css" media="screen, print"/>        						                <link rel="stylesheet" type="text/css" href="/css/articles-0.css" media="screen, print"/>    			        <link rel="stylesheet" type="text/css" href="/bmcbioinformatics/css/themes/1002.css?1330085853225" media="screen, print"/>                <link rel="stylesheet" type="text/css" href="/css/media/print-0.css" media="print"/>    <!--[if IE]>                    <link rel="stylesheet" type="text/css" href="/css/hacks/ie-0.css"/>    <![endif]--><!--[if IE 7]>                    <link rel="stylesheet" type="text/css" href="/css/hacks/ie7-0.css"/>    <![endif]--><script type="text/javascript">    //configuration    OAS_url = 'http://oas.biomedcentral.com/RealMedia/ads/';	// control of advert's position on page [AMC]        		    OAS_sitepage = "biomedcentral.com/bmcbioinformatics/article/10.1186/1471/2105/13/79";    			OAS_listpos = 'Top,x96,Bottom';							OAS_query = '';			    OAS_target = '_top';    //end of configuration    OAS_version = 10;    OAS_rn = '001234567890'; OAS_rns = '1234567890';    OAS_rn = new String (Math.random()); OAS_rns = OAS_rn.substring (2, 11);    function OAS_NORMAL(pos) {		document.write('<a href="' + OAS_url + 'click_nx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '!' + pos + '?' + OAS_query + '" target="' + OAS_target + '">');		document.write('<img src="' + OAS_url + 'adstream_nx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '!' + pos + '?' + OAS_query + '" border="0"/></a>');    }	function refererTerms(referer) {		referer = referer.split("q=");		referer = referer[1].split("&");		referer = referer[0];		if(referer.match("%20") != null && referer.match("%20").length >= 1) {			referer =  referer.replace(/%20/gi,"+");			return referer;		} else { return referer; }	} </script>  <div id="oas-campaign" style="display:none">biomedcentral.com/bmcbioinformatics/article/10.1186/1471/2105/13/79</div> <div id="oas-positions" style="display:none">Bottom,Top</div>  <script type="text/javascript">	// Set the version of JavaScript to 10 for a Browser 'Mozilla/3' on webTV     OAS_version = 11;    if ((navigator.userAgent.indexOf('Mozilla/3') != -1) || (navigator.userAgent.indexOf('Mozilla/4.0 WebTV') != -1)) {     OAS_version = 10; }    if (OAS_version >= 11) {    document.write('<scr' + 'ipt type="text/javascript" src="' + OAS_url + 'adstream_mjx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '?' + OAS_query + '"><\/script>'); }</script>	<script type="text/javascript">    document.write('');	    function OAS_AD(pos) {        if (OAS_version >= 11)         OAS_RICH(pos);        else         OAS_NORMAL(pos);    } </script><script type="text/javascript">    jsonLoaders = function(){        var _loaders = [];        return {            register: function(config){                config.prefix = config.prefix || '/webapi/1.0/blogs/';                config.feedLength = config.feedLength || 3;                _loaders.push(config);            },            list: function(){return _loaders}        };    }();    function totext() {	    document.write('<a href="javascript:returnToText();">Return to text</a>');    }    function retrieve_doi(){        var metas = document.getElementsByTagName('meta');        var i;        for (i = 0; i < metas.length; i++)            if (metas[i].getAttribute('name') == "citation_doi")                break;        var citation_doi = metas[i].getAttribute('content');        return citation_doi;    }</script>        <link title="Latest articles" rel="alternate" href="http://www.biomedcentral.com/bmcbioinformatics/latest/rss" type="application/rss+xml" exposeToHeader="true"/>    <link title="Most viewed" rel="alternate" href="http://www.biomedcentral.com/bmcbioinformatics/mostviewed/rss/" type="application/rss+xml" exposeToHeader="true"/>    <link title="Latest comments" rel="alternate" href="http://www.biomedcentral.com/bmcbioinformatics/latestcomments/rss" type="application/rss+xml" exposeToHeader="true"/></head><body class=" theme    "><div id="branding">            <dl class="google-ad wide ">	<dt class="hide">					<a class="banner-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising"></a>							</dt><dd><!-- OAS AD 'Top' begin --><script type="text/javascript">OAS_AD('Top');</script><!-- OAS AD 'Top' end --></dd></dl><noscript><dl class="google-ad wide noscript">      <dt class="hide"></dt>          <dd>        <a href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/biomedcentral.com/bmcbioinformatics/article/10.1186/1471/2105/13/79/11373465983@Top?">            <img src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/biomedcentral.com/bmcbioinformatics/article/10.1186/1471/2105/13/79/14357351733@Top?" alt="advert"/>        </a>    </dd></dl></noscript>    <div class="sup-panel-outer">        <div class="sup-panel-inner">            <ul id="login" class="login">            <li id="loginLink"><a href="/bmcbioinformatics/logon">Log on</a></li>    </ul>            <ul class="nav-sup">            <li id="BMC" class="current">            <a href="http://www.biomedcentral.com/"><span class="img-title">&nbsp;</span>            <span class="text-title">BioMed Central</span></a>        </li>		        		<li><a href="http://www.biomedcentral.com/journals"><span>Journals</span></a></li>										<li>				<a href="http://www.biomedcentral.com/gateways"><span>Gateways</span></a>			</li>					    	</ul>        </div>    </div>    <div class="branding-inner">					<div class="logo">			<a href="/bmcbioinformatics/"><img alt="BMC Bioinformatics" title="BMC Bioinformatics Logo" src="/sites/1002/images/logo.gif" /></a>						</div>						                            <div id="impact-factor" class="official">    <img src="/images/branding/official.gif" alt="official impact factor" title="2.75"/>    <span id="impact-factor-value">2.75</span></div>                                <div class="module search gray small  ">    <div class="module-inner padded">        <form method="get" class="searchForm"              action=" /quickSearchRedirectURL "              onsubmit="window.location.href=this['url'].value + '?terms=' + encodeURIComponent(this['terms'].value); return false;">            <fieldset class="search">			                <span>                    Search <select name="url">                                <option value="http://www.biomedcentral.com/bmcbioinformatics/search/results" selected>BMC Bioinformatics</option>                                <option value="http://www.biomedcentral.com/search/results">BioMed Central</option>                           </select> for                </span>                            <input id="searchTerms" name="terms" type="text" alt="Search terms" class="text " />                 <button class="w37" value="Go" type="submit">Go</button>            </fieldset>        </form>    </div>	<a class="advanced-search" href="/bmcbioinformatics/search">Advanced search</a></div>                         <div id="mobile-nav-list" >	<ul class="primary-nav">		<li     ><a href="/bmcbioinformatics/"><span id="home-tab">Home</span></a></li>														<li             class="current"    ><a href="/bmcbioinformatics/content"><span id="articles-tab"" >Articles</span></a></li>																					<li     ><a href="/bmcbioinformatics/authors/instructions"><span id="authors-tab">Authors</span></a></li>			<li     ><a href="/bmcbioinformatics/about/reviewers"><span id="reviewers-tab">Reviewers</span></a></li>						<li     ><a href="/bmcbioinformatics/about"><span id='about-tab'>About this journal</span></a></li>		<li     ><a href="/bmcbioinformatics/my"><span id="my-tab">My BMC Bioinformatics</span></a></li>										</ul></div>            </div></div>   <hr class="hide" /><div class="container">	<div id="content"  class="overflow-visible" >			     	<!--div class="branding-mobile">		<h1 class="logo">	    <a href="/bmcbioinformatics/">		    <img alt="BMC Bioinformatics" title="BMC Bioinformatics Logo" src="/sites/1002/images/logo.gif" />		</a>		</h1>	</div><br/-->				<div class="outline-wrapper block mobile-hidden" id="left-article-box"><ul id="box-outline" class="box"><ul id="outline" class="outline">   <li>      <a href="#">Top</a>   </li>   <li>      <a href="#abs">Abstract</a>   </li>   <li>      <a href="#sec1">Background</a>   </li>   <li>      <a href="#sec2">Results</a>   </li>   <li>      <a href="#sec3">Discussion</a>   </li>   <li>      <a href="#sec4">Conclusions</a>   </li>   <li>      <a href="#sec5">Materials and methods</a>   </li>   <li>      <a href="#sec6">Competing Interests</a>   </li>   <li>      <a href="#sec7">Authors’ contributions</a>   </li>   <li>      <a href="#ack">Acknowledgements</a>   </li>   <li class="lowest">      <a href="#refs">References</a>   </li></ul></ul><dl id="box-outline1"><dl class="google-ad  ">	<dt class="hide">							</dt><dd><!-- OAS AD 'x96' begin --><script type="text/javascript">OAS_AD('x96');</script><!-- OAS AD 'x96' end --></dd></dl><noscript><dl class="google-ad  noscript">      <dt class="hide"><a href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</dt>          <dd>        <a href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/biomedcentral.com/bmcbioinformatics/article/10.1186/1471/2105/13/79/14274811671@x96?">            <img src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/biomedcentral.com/bmcbioinformatics/article/10.1186/1471/2105/13/79/17237006942@x96?" alt="advert"/>        </a>    </dd></dl></noscript></dl></div><div class="mobile-hidden" id="right-panel"><div id="mobile-sidebar">	<div id="mobile-sidebar-tab">		<span>			<a href="#" id="mobile-sidebar-tab-link"><img style="opacity: 0.5;" src="/images/sidebar-hidden.png"><img style="display:none;" src="/images/sidebar-shown.png"></a>		</span>	</div>	<div id="mobile-sidebar-list"><div id="article-navigation-bar">    <div class="issue-information block">        <h5><a href="http://www.biomedcentral.com/bmcbioinformatics"><strong>BMC Bioinformatics</strong></a></h5>        <ul class="square normal">            <li><a href="http://www.biomedcentral.com/bmcbioinformatics/content/13">Volume 13</a></li>								        </ul>    </div>    <div class="article-information block" id="article-info">        <div id="viewing-options-links">	<h5>Viewing options</h5>	<ul class="square normal">                    <li>                             <a href="/1471-2105/13/79/abstract">Abstract</a>                        </li>        				<li>							<strong>Full text</strong>					</li>										<li class="pdfFileSize">			<a href="/content/pdf/1471-2105-13-79.pdf">PDF</a>			<span>(1.9MB)</span>		</li>				<li>                    </li>						<li>						<a href="/1471-2105/13/79/additional">Additional files</a>					</li>			</ul></div>		<div id="associated-material-links">    <h5>Associated material</h5>    <ul class="square normal">                        <li>            <a href="/pubmed/22558947">PubMed record</a>        </li>                            <li><a href="/1471-2105/13/79/about">About this article</a></li>                <li>                        <a href="/1471-2105/13/79/comments">Readers' comments </a>                    </li>        <li>                    </li>    </ul></div>		<div id="related-literature-links">    <h5>Related literature</h5>    <ul class="normal">                    <li> <a href="http://www.biomedcentral.com/1471-2105/13/79/about#citations"><h6>Cited by</h6></a></li>                <li id="google-blog-search"><a href="http://www.google.com/search?tbm=blg&hl=en&amp;source=hp&amp;biw=1280&amp;bih=899&amp;q=Inferring+high-confidence+human+protein-protein+interactions&amp;btnG=Search">on Google blog search</a></li>        <h6>Other articles by authors</h6>        <li>            <a class="collapser"><i>&nbsp;</i>on Google Scholar</a><ul id="authg" class="hidebeforeload">                         <li>                <a href="http://scholar.google.com/scholar?q=author%3A%22X+Yu%22">Yu X</a>            </li>                                <li>                <a href="http://scholar.google.com/scholar?q=author%3A%22A+Wallqvist%22">Wallqvist A</a>            </li>                                <li>                <a href="http://scholar.google.com/scholar?q=author%3A%22J+Reifman%22">Reifman J</a>            </li>            </ul>          </li>                <li>            		<a class="collapser"><i>&nbsp;</i>on PubMed</a>	<ul id="authpm" class="hidebeforeload"> 									<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Yu_X%20[Author]">Yu X</a>				</li>												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Wallqvist_A%20[Author]">Wallqvist A</a>				</li>												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Reifman_J%20[Author]">Reifman J</a>				</li>						</ul > 	        </li>    </ul>     <h6>Related articles/pages</h6>    <ul class="square normal">        <li><a href="http://www.google.co.uk/search?hl=en&amp;q=related:http://www.biomedcentral.com/1471-2105/13/79">on Google</a></li>        <li><a href="http://scholar.google.com/scholar?q=related:http://www.biomedcentral.com/1471-2105/13/79">on Google Scholar</a></li>                    <li><a href="/pubmed/related/22558947">on PubMed</a></li>            </ul></div>		                    	<h5>Tools</h5>	<ul class="square normal">									<li>						<a href="/1471-2105/13/79/citation">Download references</a>					</li>				<li><a href="/content/download/xml/1471-2105-13-79.xml">Download XML</a></li>				<li>						<a href="/1471-2105/13/79/email">Email to a friend</a>					</li>		<li><a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=456&doi=10.1186/1471-2105-13-79&volume=13&issue=1&title=Inferring+high-confidence+human+protein-protein+interactions&author_name=Xueping Yu&start_page=1&end_page=16">Order reprints</a></li>		<li>						<a href="/1471-2105/13/79/postcomment">Post a comment</a>					</li>	</ul>		<h5>Share this article</h5><ul id="social-networking-links">	<li>	<div class="fb-like" data-href="http://www.biomedcentral.com/1471-2105/13/79" data-send="false" data-layout="button_count" data-width="100" data-show-faces="false" data-action="recommend"></div>	</li>		<li>		<a href="http://twitter.com/share" data-url="http://www.biomedcentral.com/1471-2105/13/79" data-text="#bmcbioinformatics Inferring high-confidence human protein-protein interactions" class="twitter-share-button">Tweet</a>	</li>	<li>			<g:plusone size="medium" width="100"></g:plusone>	</li>	<li>		<a class="sharethis_title btn" href="#"><span class="icon">&nbsp;</span>More options...</a>	<ul class="iconlist hidebeforeload" id="sharethis">			<li>				<a onclick="largepopup(this.href,'citeulike_popup_post',800,600);return false" href="http://www.citeulike.org/posturl?url=http://www.biomedcentral.com/1471-2105/13/79"><span class="share-icons citeulike"></span>Citeulike</a>			</li>					<li>					<a href="http://www.connotea.org/addpopup?continue=confirm&amp;uri=http://dx.doi.org/doi:10.1186/1471-2105-13-79" onclick="largepopup(this.href,'Connotea',800,600);return false"><span class="share-icons connotea"></span>Connotea</a>	</li>				<li>				<a onclick="largepopup(this.href,'Del.icio.us',800,600);return false" href="http://del.icio.us/post?url=http://www.biomedcentral.com/1471-2105/13/79&title=Inferring high-confidence human protein-protein interactions"><span class="share-icons delicious"></span>Del.icio.us</a>	</li>			<li>				<a href="/1471-2105/13/79/email"><span class="share-icons email"></span>Email</a>	</li>			<li>				<a onclick="largepopup(this.href,'facebook',800,600);return false" href="http://www.facebook.com/sharer.php?u=http://www.biomedcentral.com/1471-2105/13/79"><span class="share-icons facebook"></span>Facebook</a>	</li>	<li class="gp">			<div class="googlehider mobile-hide"> 				<div class="g-plusone" data-annotation="none"></div>			</div>				<span class="share-icons googleplus mobile-hide"></span>				<div class="googlehider2 mobile-hide"> 				<div class="g-plusone mobile-hide" data-annotation="none"></div>			</div>				<span class="share-icons googleplus"></span>				<span class="gp-link"><a href="#">Google+</a></span>	</li>			<li>				<a onclick="largepopup(this.href,'mendeley',800,600);return false" href="http://www.mendeley.com/import/?url=http://www.biomedcentral.com/1471-2105/13/79"><span class="share-icons mendeley"></span>Mendeley</a>			</li>			<li>				<a onclick="largepopup(this.href,'twitter',800,600);return false" href="http://twitter.com/?status=%23bmcbioinformatics%0D%0AInferring%20high-confidence%20human%20protein-protein%20interactions%20http%3A%2F%2Fwww.biomedcentral.com%2F1471-2105%2F13%2F79"><span class="share-icons twitter"></span>Twitter</a>  			</li>		</ul></ul>    </div>							                        				</div>		<div class="mobile-sidebar-gradient top"></div>		<div class="mobile-sidebar-gradient bottom"></div>	</div>    </div></div><div class="rounded custom white article full-text">    <div class="t png_bg">        <div class="r png_bg"></div>    </div>    <div class="m">    <div class="r">        <div class="wrap-inner content">                    <div class="padded-inner">                            	                                                                                                                                                                                                                                                         			                        <script>window.bmcIsMobile = "classic";</script>            <div id="topmatter">		    				<a href="/about/access"><img alt="Open Access" src="/images/articles/openaccess-large.png" class="access mr15"/></a>			<a href="http://www.biomedcentral.com/about/mostviewed/"><img alt="Highly Accessed" src="/images/articles/highlyaccessed-large.png" class="access mr15"/></a>				<span class="articletype">Methodology article</span><h1>Inferring high-confidence human protein-protein interactions</h1>        	<div class="singleins">				<p class="authors">        																																																																																												        <strong>Xueping Yu</strong>, <strong>Anders Wallqvist</strong> and <strong>Jaques Reifman</strong><sup>&#42;</sup>	</p>			<div id="affiliations">		            <div class="module gray  inner">                <div class="module-inner padded-inner">		<ul>			<li>                                                                            		<p >							<span>*</span>									 Corresponding author:										            Jaques Reifman <a href='mailto:jaques.reifman@us.army.mil'>jaques.reifman@us.army.mil</a>    					</p>      </li>			                                                                                                    		</ul>				            <section> 						<div class="collapsible-content">							 <p class="options"><a class="affiliations-toggle" href="#"><i class="arrow"></i>Author Affiliations</a></p>				<div id="ins_container" style="display: block;">										    						    <p class ="singleInstitute">						                                        Biotechnology High-Performance Computing Software Applications Institute, Telemedicine and Advanced Technology Research Center, U.S. Army Medical Research and Materiel Command, Ft. Detrick, MD, 21702, USA                            </p>																				</div>										<p id="authoremails">            For all author emails, please <a href="/bmcbioinformatics/logon">log on</a>.    </p>						</div>			</section>				                </div>                    <div class="module-tl"></div>                    <div class="module-tr"></div>                    <div class="module-bl"></div>                    <div class="module-br"></div>            </div>		</div>	  </div>	<section class="cit">					<div class="collapsible-content">    <p><em>BMC Bioinformatics</em> 2012, <strong>13</strong>:79&nbsp;<span class="pseudotab">doi:10.1186/1471-2105-13-79</span></p>			        		        		<br/>		<p>The electronic version of this article is the complete one and can be found online at: <a href="http://www.biomedcentral.com/1471-2105/13/79">http://www.biomedcentral.com/1471-2105/13/79</a></p>		<br/>                    <table cellspacing="0" cellpadding="0">        <tbody>        <tr><td>Received:</td><td>5 October 2011</td></tr>                <tr><td>Accepted:</td><td>4 May 2012</td></tr>        <tr><td>Published:</td><td>4 May 2012</td></tr>        </tbody>        </table>    		<br/>        <p>            &copy; 2012 Yu et al.; licensee BioMed Central Ltd. <br />        </p>					<p>				This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href='http://creativecommons.org/licenses/by/2.0'>http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.			</p>		    	</div>	</section></div>    	<div id="article-body">		    <span class="mathjax" style="display:table-cell;vertical-align:center;padding:0;margin:0;float:right;position:relative;top:1.8em;left:0.8em;width:24.9em;line-height:1.4em;">   <form name="maths" method="get" style="float:right"><label for="math" style="margin-right:0.4em;position:relative;top:-0.2em">Formula display:</label><input type="hidden" name="fmt_math" value="no"><input type="checkbox" id="math" style="position:relative;left:0.1em" onClick="javascript:if(document.forms['maths'].elements['fmt_math_check'].checked) document.forms['maths'].elements['fmt_math'].value='yes'; document.forms['maths'].submit();" name="fmt_math_check" checked><label for="math" style="margin-left:4px;position:relative;top:.1em"><img src="/images/plugins/mathjax.png" style=""></label><a title="Mathematical formulae have been encoded as MathML and are displayed in this HTML version using MathJax in order to improve their display. Uncheck the box to turn MathJax off. This feature requires Javascript. For more information click on this button." href="http://www.biomedcentral.com/about/mathjax"><span style="width:19px;height:18px;position:relative;top:0.4em;margin-left:0.4em;background-position-y: -80px;background-position-x: 273px;" class="help-icon"></span></a></form></span><section><a name="abs"></a><h3>Abstract</h3>   <div class="collapsible-content">      <h4>Background</h4>      <p style="line-height:160%">As numerous experimental factors drive the acquisition, identification, and interpretation         of protein-protein interactions (PPIs), aggregated assemblies of human PPI data invariably         contain experiment-dependent noise. Ascertaining the reliability of PPIs collected         from these diverse studies and scoring them to infer high-confidence networks is a         non-trivial task. Moreover, a large number of PPIs share the same number of reported         occurrences, making it impossible to distinguish the reliability of these PPIs and         rank-order them. For example, for the data analyzed here, we found that the majority         (&gt;83%) of currently available human PPIs have been reported only once.      </p>      <h4>Results</h4>      <p style="line-height:160%">In this work, we proposed an unsupervised statistical approach to score a set of diverse,         experimentally identified PPIs from nine primary databases to create subsets of high-confidence         human PPI networks. We evaluated this ranking method by comparing it with other methods         and assessing their ability to retrieve protein associations from a number of diverse         and independent reference sets. These reference sets contain known biological data         that are either directly or indirectly linked to interactions between proteins. We         quantified the average effect of using ranked protein interaction data to retrieve         this information and showed that, when compared to randomly ranked interaction data         sets, the proposed method created a larger enrichment (~134%) than either ranking         based on the hypergeometric test (~109%) or occurrence ranking (~46%).      </p>      <h4>Conclusions</h4>      <p style="line-height:160%">From our evaluations, it was clear that ranked interactions were always of value because         higher-ranked PPIs had a higher likelihood of retrieving high-confidence experimental         data. Reducing the noise inherent in aggregated experimental PPIs via our ranking         scheme further increased the accuracy and enrichment of PPIs derived from a number         of biologically relevant data sets. These results suggest that using our high-confidence         protein interactions at different levels of confidence will help clarify the topological         and biological properties associated with human protein networks.      </p>   </div></section><span id="keywords">   <h5 class="inline">Keywords: </h5>High confidence; Human protein interaction network; Protein-protein interactions</span><section><a name="sec1"></a><h3>Background</h3>   <div class="collapsible-content">      <p style="line-height:160%">The development of high-throughput techniques during the last decade has led to an         unprecedented increase in the volume of identified human protein-protein interactions         (PPIs). The currently available individual PPI data sets can be roughly categorized         into three sets: <em>1</em>) proteome-wide, large-scale screenings aimed at investigating all possible PPIs <a name="d16426e143"></a><a name="d16426e145"></a><a name="d16426e147"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>-<a onclick="LoadInParent('#B3'); return false;" href="#B3">3</a>], <em>2</em>) semi-large-scale screenings aimed at investigating the interactions between a specific         group of proteins (typically in a pathway) and all other proteins <a name="d16426e154"></a><a name="d16426e156"></a>[<a onclick="LoadInParent('#B4'); return false;" href="#B4">4</a></a>,<a onclick="LoadInParent('#B5'); return false;" href="#B5">5</a>], and <em>3</em>) small-scale, traditional studies aimed at detecting specific PPIs among biologically         interesting proteins, e.g., oncogenes and their regulators. Although this latter set         is still numerically dominant (~80% of all PPIs belong to this set), examples of the         first two types of investigations are expanding rapidly.      </p>      <p style="line-height:160%">Given this extensive resource of known human PPIs and their continuous accelerated         growth, how to globally analyze and aggregate the data remain a challenge. Statistical         methods for inferring confidence of protein interactions can be broadly divided into         two groups <a name="d16426e165"></a><a name="d16426e167"></a><a name="d16426e169"></a>[<a onclick="LoadInParent('#B6'); return false;" href="#B6">6</a>-<a onclick="LoadInParent('#B8'); return false;" href="#B8">8</a>]: scoring schemes that rely on the interaction data themselves (e.g., affinity purification/mass-spectrometry         [AP/MS] data or yeast two-hybrid [Y2H] data) and scoring schemes that require additional         data sources not directly related to the interactions <em>per se</em> (e.g., functional annotation or gene expression data). Herein, we address the question         of how to extract high-confidence PPIs while relying only on the aggregated interaction         data themselves.      </p>      <p style="line-height:160%">The most intuitive approach to infer high-confidence PPIs is to score PPIs based on         the number of times an interaction has been reported <a name="d16426e178"></a><a name="d16426e180"></a><a name="d16426e182"></a>[<a onclick="LoadInParent('#B9'); return false;" href="#B9">9</a>-<a onclick="LoadInParent('#B11'); return false;" href="#B11">11</a>]. However, using the number of times a PPI has been reported (occurrence) across different         studies as the metric of reliability could be influenced by numerous unknowable experimental         factors, e.g., recent studies have demonstrated that such factors may result in decreased         reliability of PPIs containing frequently studied proteins <a name="d16426e186"></a>[<a onclick="LoadInParent('#B12'); return false;" href="#B12">12</a>]. Moreover, a large number of PPIs share the same number of reported occurrences,         making it impossible to use occurrence alone to establish the reliability of these         PPIs and rank-order them. For example, for the data analyzed here, we found that the         majority (&gt;83%) of currently available human PPIs have been reported only once.      </p>      <p style="line-height:160%">Herein, we propose an unsupervised statistical approach to score and rank a set of         diverse, experimentally identified PPIs. We applied this methodology to human PPIs         (non-physical associations excluded) aggregated from nine publicly available primary         databases that exclusively contain experimental data (Additional file <a name="d16426e191"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/suppl/S1">1</a>): the Biomolecular Interaction Network Database (BIND) <a name="d16426e195"></a>[<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a>], the Biological General Repository for Interaction Datasets (BioGRID) <a name="d16426e199"></a>[<a onclick="LoadInParent('#B14'); return false;" href="#B14">14</a>], the Database of Interacting Proteins (DIP) <a name="d16426e203"></a>[<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>], the Human Protein Reference Database (HPRD) <a name="d16426e207"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>], IntAct <a name="d16426e212"></a>[<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>], the Molecular INTeraction database (MINT) <a name="d16426e216"></a>[<a onclick="LoadInParent('#B18'); return false;" href="#B18">18</a>], the mammalian PPI database of the Munich Information Center on Protein Sequences         (MIPS) <a name="d16426e220"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>], PDZBase (a PPI database for PDZ-domains) <a name="d16426e224"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>], and Reactome <a name="d16426e228"></a>[<a onclick="LoadInParent('#B21'); return false;" href="#B21">21</a>]. Our method re-normalizes the importance of frequently occurring proteins among PPIs         to avoid giving added (and potentially artificial) weight to those interactions. We         estimated the importance of a PPI by comparing the actual observed occurrence of a         PPI with its occurrence in a randomized sample. This calculation gauges the likelihood         that the interaction occurs by chance in the set of all observed PPIs. Using these         estimates, we rank-ordered the aggregated input PPI data set, allowing us to create         high-confidence subsets based on a given rank threshold. At the lowest ranked threshold,         all interactions are included and there is no difference between the ranked data and         the original set of PPIs.      </p><div class="figs">      <div class="addfile "><p><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/suppl/S1','S1',630,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/suppl/S1"></a><strong>Additional file 1.</strong> Aggregated human physical PPI data. This tab-delimited text file contains the aggregated         PPI data based on publications collected from nine databases: BIND, BioGRID, DIP,         HPRD, IntAct, MINT, MIPS, PDZBase, and Reactome. </p><p>         	Format: TXT          	Size: 2.4MB <a href="/content/supplementary/1471-2105-13-79-s1.txt">Download file</a><span class="open-data-icon"><a href="/about/access/#opendata"><img title="Open Data" alt="Open Data" src="/bmcimages/article/opendata.gif"></a></span></div></div>      <p style="line-height:160%">The presented scoring and ranking procedure can be seen as an extension of our previous         effort to infer high-confidence interactions from the affinity purification raw data,         termed interaction detection based on shuffling (IDBOS) <a name="d16426e244"></a>[<a onclick="LoadInParent('#B22'); return false;" href="#B22">22</a>], and, in the following, we will also refer to our scoring and ranking scheme as IDBOS.         Our proposed procedure shares similarities to estimating probabilities of observed         interactions above a random background based on the hypergeometric distribution <a name="d16426e248"></a>[<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>], with the distinction that the IDBOS-generated probability density distribution functions         correct for biases toward self-interaction among frequently studied proteins. Although         other methods exist for assigning confidence scores to PPIs, these generally require         additional data or reference sets <a name="d16426e252"></a><a name="d16426e254"></a>[<a onclick="LoadInParent('#B24'); return false;" href="#B24">24</a></a>,<a onclick="LoadInParent('#B25'); return false;" href="#B25">25</a>], or <em>a priori</em> assumptions of network topology <a name="d16426e261"></a>[<a onclick="LoadInParent('#B26'); return false;" href="#B26">26</a>]. To the best of our knowledge, this is the first application of an unsupervised probabilistic         scoring and ranking scheme to create subsets of unbiased high-confidence human PPI         networks.      </p>      <p style="line-height:160%">We evaluated the improvement in using IDBOS-ranked PPI data by comparing it with other         methods and assessing their ability to retrieve biological associations from a number         of diverse and independent reference sets. These reference sets contain known biological         data that are either directly (e.g., crystallographically determined protein complexes)         or indirectly (e.g., co-expressed genes) linked to interactions between proteins.         The hypothesis we tested was that sets of highly ranked PPIs are enriched in biological         associations as determined from the diverse reference sets. We quantified the average         effect of using ranked protein interaction data to retrieve this information and showed         that, when compared to randomly ranked interaction data sets, IDBOS created a larger         enrichment (~134%) than either ranking based on the hypergeometric test (~109%) or         occurrence ranking (~46%).      </p>      <p style="line-height:160%">From our evaluations, it was clear that ranked interactions were always of value because         higher-ranked PPIs had a higher likelihood of retrieving biologically relevant data.         Statistically removing the biasing factors inherent in aggregated PPI data via the         IDBOS-ranking scheme further increased the accuracy and enrichment of biological information         associated with PPIs.      </p>   </div></section><section><a name="sec2"></a><h3>Results</h3>   <div class="collapsible-content">      <h4>Statistical scoring of human PPIs from literature data</h4>      <p style="line-height:160%">We used the collection of human experimental and physical PPIs to create a set of         116,134 reported interactions, containing 80,980 unique physical associations between         13,369 distinct proteins (see Materials and methods). Out of the unique PPIs, 13,554         interactions were observed in more than one experiment. The number of times a PPI         has been reported in the literature is an important metric for inferring high-confidence         interactions <a name="d16426e279"></a><a name="d16426e281"></a><a name="d16426e283"></a>[<a onclick="LoadInParent('#B9'); return false;" href="#B9">9</a></a>,<a onclick="LoadInParent('#B10'); return false;" href="#B10">10</a>,<a onclick="LoadInParent('#B27'); return false;" href="#B27">27</a>], but it could also be dependent on other factors <a name="d16426e287"></a><a name="d16426e289"></a>[<a onclick="LoadInParent('#B12'); return false;" href="#B12">12</a>,<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>]. The observed number of interactions of a protein is partly reflective of how often         it has been studied (popularity). For example, the top five connected proteins in         the PPI data are G-protein beta subunit (GNB1), G-protein gamma subunit (GNGT1), G-protein         alpha subunit (GNAL), ubiquitin C (UBC), and tumor protein p53 (TP53), having 2,280,         2,248, 2,243, 1,899, and 1,097 reported interactions, respectively. To normalize this         popularity bias, we compared the observed number of protein interactions with statistics         derived from the corresponding probability density distribution functions generated         from randomized data. In the generation of the random interaction sets, we kept each         protein’s reported number of interactions fixed and, thus, we expect the corresponding         interaction probabilities of proteins with a high (low) number of reported interactions         also to be high (low).      </p>      <p style="line-height:160%">Note that the reported number of interactions involving a protein refers to the total         number of observed PPIs in the literature involving that specific protein. This is         different from a protein’s degree, which is defined as the number of unique interacting         protein partners. Thus, while TP53 is associated with 1,097 observed PPIs, its degree         is reduced to 478 due to the multiple observations of many of the involved interactions.      </p>      <p style="line-height:160%">Our calculations followed our previously described procedures for generating Z-scores         from sets of interacting protein pairs <a name="d16426e297"></a>[<a onclick="LoadInParent('#B22'); return false;" href="#B22">22</a>]. Briefly, the aggregated PPI data tabulates all pairs and the number of their occurrence         in the literature. From this list we counted, for each unique pair between proteins         <em>i</em> and <em>j</em>, how many times it occurred <em>O</em><sub><em>ij</em></sub>. Randomized versions of the original PPI list were then created under the conditions         that: <em>1</em>) the protein identifiers and the number of times they occur are preserved and <em>2</em>) no interactions are allowed between proteins of the same identifier, i.e., self-interactions         are not allowed. We generated <em>M</em> = 10<sup>6</sup> randomized PPI lists and calculated the average number of times each PPI from the         original list occurred &lt; <em>R</em><sub><em>ij</em></sub> &gt; and its standard deviation σ<sub><em>R</em></sub>. The corresponding Z-score was then calculated as,      </p>      <p style="line-height:160%" class="inlinenumber"><mml:math xmlns:mml="http://www.w3.org/1998/Math/MathML"> <mml:mrow> <mml:msub> <mml:mi>Z</mml:mi> <mml:mi mathvariant="italic">ij</mml:mi> </mml:msub> <mml:mo>=</mml:mo> <mml:mfrac> <mml:mrow> <mml:msub> <mml:mi>O</mml:mi> <mml:mi mathvariant="italic">ij</mml:mi> </mml:msub> <mml:mo>&#8722;</mml:mo> <mml:mo>&lt;</mml:mo> <mml:msub> <mml:mi>R</mml:mi> <mml:mi mathvariant="italic">ij</mml:mi> </mml:msub> <mml:mo>&gt;</mml:mo> </mml:mrow> <mml:msub> <mml:mi>&#963;</mml:mi> <mml:mi>R</mml:mi> </mml:msub> </mml:mfrac> </mml:mrow> </mml:math></p>      <p style="line-height:160%">We further used the randomized data to estimate the p-value p<sub><em>ij</em></sub> of observing an interaction between proteins <em>i</em> and <em>j</em> in the original data set. Briefly, for each unique pair in the original PPI list,         we calculated the number of times it occurred in each of the <em>n</em> = 1, … ,<em>M</em> random realizations and created the normalized probability density function PDF<sub><em>ij</em></sub>, i.e., the probability of finding <span class="inlinenumber"><a title="View MathML" onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M2','MathML',630,470);return false;" href="http://www.biomedcentral.com/1471-2105/13/79/mathml/M2"><img class="mathimg" src="/content/inline/1471-2105-13-79-i2.gif" alt="<a onClick=&#34;popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M2','MathML',630,470);return false;&#34; target=&#34;_blank&#34; href=&#34;http://www.biomedcentral.com/1471-2105/13/79/mathml/M2&#34;&gt;View MathML</a&gt;"></a></span> occurrences by random chance. We estimated the p-value for each actual interaction         as,      </p>      <p style="line-height:160%" class="inlinenumber">         <div style="display:table;width:100%;*display:inline">            <p class="inlinenumber"><a title="View MathML" onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M3','MathML',630,470);return false;" href="http://www.biomedcentral.com/1471-2105/13/79/mathml/M3"><img class="mathimg" src="/content/inline/1471-2105-13-79-i3.gif" alt="<a onClick=&#34;popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M3','MathML',630,470);return false;&#34; target=&#34;_blank&#34; href=&#34;http://www.biomedcentral.com/1471-2105/13/79/mathml/M3&#34;&gt;View MathML</a&gt;"></a></p><span style="font-size:80%;text-align:right;display:table-cell;vertical-align:center;*float:right;*display:inline;*position:relative;*top:6px">(2)</span></div>      </p>      <p style="line-height:160%">Z-scores and p-values (Z<sub><em>ij</em></sub> and p<sub><em>ij</em></sub>) are legitimate metrics for ranking an observed PPI, although they are associated         with different numerical properties and uncertainties because of the limitations imposed         on using a finite number of randomizations.      </p>      <h4>Assessing p-values and Z-scores as PPI metrics</h4>      <p style="line-height:160%">To assess the p-values and Z-scores as metrics in identifying and ranking high-confidence         protein interactions in biological data sets, we compared the scored interactions         with an independent PPI data set that was not part of the aggregated human data set.         We used the human PPIs derived from experimentally determined structural protein complexes         deposited in the Protein Data Bank (PDB) as outlined in the Materials and methods         (Additional file <a name="d16426e568"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/suppl/S2','',630,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/suppl/S2">2</a>). To assess our scored data using this data set as a benchmark, we selected only         those interacting protein pairs from the scored human PPI data where both proteins         appeared in the PDB-derived PPI data set. We termed these pairs as “judgeable,” i.e.,         the proteins appear in the benchmark data set, and we can judge whether the interaction         occurred or not. Thus, for a given score, we determined how many of our judgeable         PPIs were interacting and non-interacting within the benchmark set. Figure <a name="d16426e571"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/figure/F1">1A-B</a> shows the frequency of interacting and non-interacting pairs as a function of Z-scores         and p-values. Interactions associated with high Z-scores and small p-values were enriched         with benchmark interactions, indicating that both metrics were effective in extracting         high-confidence PPIs from the aggregated literature data.      </p><div class="figs">      <div class="addfile "><p><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/suppl/S2','S2',630,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/suppl/S2"></a><strong>Additional file 2.</strong> Direct interactions supported by known complexes in PDB. This tab-delimited file contains         the protein pairs that have direct contact in at least one protein complex in PDB.         </p><p>         	Format: TXT          	Size: 7KB <a href="/content/supplementary/1471-2105-13-79-s2.txt">Download file</a><span class="open-data-icon"><a href="/about/access/#opendata"><img title="Open Data" alt="Open Data" src="/bmcimages/article/opendata.gif"></a></span></div>      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/figure/F1','F1',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/figure/F1"><img align="top" src="/content/figures/1471-2105-13-79-1.gif" alt="thumbnail" class="thumbnail"><strong>Figure 1.</strong></a> <strong>IDBOS scoring schemes.</strong> The method presented is an extension to the interaction detection based on shuffling         (IDBOS) method used for mass spectrometry co-purification data <a name="d16426e597"></a>[<a onclick="LoadInParent('#B22'); return false;" href="#B22">22</a>]. We compared the set of known protein-protein interactions (PPIs) with randomized         versions, which preserve the number of interactions per protein, to obtain a Z-score         and a p-value for each interaction. These quantities are schematically outlined at         the top, where a randomized probability density distribution function (PDF) is used         to illustrate the p-value and Z-score calculations for a particular interaction between         proteins <em>i</em> and <em>j</em>. To evaluate these scoring schemes, we analyzed interactions derived from crystallographic         complexes in the PDB. Each human PPI was compared to a PPI derived from protein structure         data in the PDB and assigned to one of two subsets: <em>interactions</em> or <em>non-interactions</em>. If the PPI was present in the PDB interaction data set, the pair was assigned to         the <em>interactions</em> set, otherwise the pair was assigned to the <em>non-interactions</em> set. It is reasonable to assume that the first subset should be enriched with actual         PPIs. (A) Distribution of Z-scores corresponding to “interactions” and “non-interactions”         assigned to PDB-derived PPIs, and (B) the corresponding p-value distributions. We         found that both p-values and Z-scores could distinguish these subsets, suggesting         that they are useful metrics.      </div></div>      <p style="line-height:160%">The procedure for shuffling the data allowed us to compare the frequency of each observed         interaction <em>O</em><sub><em>ij</em></sub> to that interaction’s probability density distribution function PDF<sub><em>ij</em></sub>. This distribution is different for each interaction and depends on the number of         distinct proteins and interactions present in the entire network. Lacking an analytical         expression to generate the exact distribution functions, the procedure outlined in         Equations 1 and 2 allowed us to generate estimates for both Z-scores and p-values         for each interacting pair. We found that using either only p-values or only Z-scores         to be inferior to using the combined information (data not shown), and, hence, we         aggregated these two metrics by converting the value of each metric into a rank and         generating the average rank from both metrics for each interaction. Interactions with         the same p-value (or Z-score) were assigned the same rank. If interactions had the         same rank in the final averaged-rank list, these interactions were considered equivalent         and analyzed together. The ranked data are provided in the Supplementary materials         (Additional file <a name="d16426e632"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/suppl/S3','',630,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/suppl/S3">3</a>). Instead of assigning post-priori probabilities to already observed interactions,         we only used ranks and comparisons between ranked data to gauge the biological information         contained in these subsets.      </p><div class="figs">      <div class="addfile "><p><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/suppl/S3','S3',630,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/suppl/S3"></a><strong>Additional file 3.</strong> IDBOS Ranked PPI data. This tab-delimited file contains the aggregated PPI data set         with the corresponding IDBOS ranking. </p><p>         	Format: TXT          	Size: 4.5MB <a href="/content/supplementary/1471-2105-13-79-s3.txt">Download file</a><span class="open-data-icon"><a href="/about/access/#opendata"><img title="Open Data" alt="Open Data" src="/bmcimages/article/opendata.gif"></a></span></div></div>      <h4>Top-ranked IDBOS PPIs are different from the most frequently reported PPIs</h4>      <p style="line-height:160%">Table <a name="d16426e651"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/table/T1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/table/T1">1</a> shows the 20 top-ranked PPIs that were identified using the IDBOS system (sorted         by average rank), and Table <a name="d16426e654"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/table/T2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/table/T2">2</a> shows the most frequently reported PPIs (sorted by PPI occurrence). As expected,         the most frequently reported PPIs involved ubiquitously studied proteins, such as         those mentioned above (UBC and TP53) as well as growth factor receptor bound 2 (GRB2),         which had 916 reported interactions. These interactions, however, were not observed         in the top PPIs scored by the IDBOS procedure. Instead, the top PPIs of the IDBOS         set were enriched with infrequently observed interactions. Although we could not independently         assess whether these interactions are more “real” than the frequently observed ones,         they are partly supported by independent literature citations. In fact, the proteins         in these interactions could be classified into two groups. The first group comprised         proteins that had the same specific function or were subunits of the same protein         complex, such as branched chain keto acid E1 alpha (BCKDHA) and branched chain keto         acid E1 beta (BCKDHB) <a name="d16426e658"></a><a name="d16426e660"></a>[<a onclick="LoadInParent('#B3'); return false;" href="#B3">3</a>,<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>], and dynein cytoplasmic 2 intermediate chain 1 (D2LIC) and dynein cytoplasmic 2 heavy         chain 1 (DNCH2) <a name="d16426e664"></a><a name="d16426e666"></a>[<a onclick="LoadInParent('#B30'); return false;" href="#B30">30</a></a>,<a onclick="LoadInParent('#B31'); return false;" href="#B31">31</a>]. The second group comprised proteins that had a ligand/receptor relationship, such         as inducible T cell co-stimulator (ICOS) and inducible T cell co-stimulator ligand         (ICOSLG) <a name="d16426e670"></a><a name="d16426e672"></a><a name="d16426e674"></a>[<a onclick="LoadInParent('#B32'); return false;" href="#B32">32</a>-<a onclick="LoadInParent('#B34'); return false;" href="#B34">34</a>], or gastric inhibitory polypeptide (GIP) and gastric inhibitory polypeptide receptor         (GIPR) <a name="d16426e679"></a><a name="d16426e681"></a><a name="d16426e683"></a><a name="d16426e685"></a>[<a onclick="LoadInParent('#B35'); return false;" href="#B35">35</a>-<a onclick="LoadInParent('#B38'); return false;" href="#B38">38</a>]. Among the top 20 PPIs of the IDBOS set, two seemingly unrelated protein pairs were         actually closely related. The proteins in the LTC4S/MGST1 interacting pair are actually         two of the six members of the membrane-associated proteins in the eicosanoid and glutathione         metabolism (MAPEG) family. Similarly, the top-ranked interaction between L-threonine         dehydrogenase (TDH) and aminoacetone synthetase (alias of GCAT) catalyzes the conversion         of L-threonine to glycine <a name="d16426e689"></a>[<a onclick="LoadInParent('#B39'); return false;" href="#B39">39</a>].      </p><div class="figs">      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/table/T1','T1',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/table/T1"><strong>Table 1.</strong></a> <strong>Top 20 protein-protein interactions from IDBOS ranking</strong></div>      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/table/T2','T2',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/table/T2"><strong>Table 2.</strong></a> <strong>Top 20 protein-protein interactions from occurrence ranking</strong></div></div>      <p style="line-height:160%">Not only were the top-ranked sets different, but the remaining bulk of the interactions         also showed considerable changes in rank when the IDBOS p-values were non-zero (See         Additional files <a name="d16426e2243"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/suppl/S3','',630,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/suppl/S3">3</a> and <a name="d16426e2246"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/suppl/S4','',630,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/suppl/S4">4</a>). To evaluate the effect of this re-ranking of interactions, we then asked whether         these rankings have any impact in retrieving biological information. We addressed         this question by comparing the ability of the two schemes to identify known and inferred         biological relationships based on rank. The hypothesis we tested was that higher-ranked         subsets of the data sets are better at retrieving biological information, and that         IDBOS ranking provides a more efficient way of retrieving this information than ranking         solely based on frequency of occurrence.      </p><div class="figs">      <div class="addfile "><p><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/suppl/S4','S4',630,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/suppl/S4"></a><strong>Additional file 4.</strong> Occurrence Ranked PPI data. This tab-delimited file contains the aggregated PPI data         set with the corresponding occurrence ranking. </p><p>         	Format: TXT          	Size: 1.7MB <a href="/content/supplementary/1471-2105-13-79-s4.txt">Download file</a><span class="open-data-icon"><a href="/about/access/#opendata"><img title="Open Data" alt="Open Data" src="/bmcimages/article/opendata.gif"></a></span></div></div>      <h4>Evaluation of ranking schemes as measures of identifying interacting proteins</h4>      <p style="line-height:160%">To assess different PPI ranking schemes, we constructed six benchmark reference sets         derived from high-quality experimental studies as detailed in Materials and methods.         These independent data sets comprise PPIs detected using <em>1</em>) far-Western blotting, <em>2</em>) isothermal titration calorimetry, <em>3</em>) nuclear magnetic resonance, <em>4</em>) surface plasmon resonance, <em>5</em>) direct interactions from protein complex structures from the PDB, and <em>6</em>) homologous human PPIs derived from actual mouse PPI data. These benchmark sets tested         the ability of the ranked data to retrieve known interactions derived from a variety         of experimentally determined PPI data sets.      </p>      <p style="line-height:160%">Using these six benchmark reference sets, we compared the IDBOS-ranked data set with         a set ranked by PPI frequency of occurrence, a set ranked using the hypergeometric         test <a name="d16426e2287"></a>[<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>] (see Materials and methods, Additional file <a name="d16426e2290"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/suppl/S5','',630,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/suppl/S5">5</a>), and two random data sets. The random data sets included a set in which the PPIs         themselves were retained, but their ranks were assigned randomly, and a set where         both the interactions themselves were randomized and were assigned a random rank.         In addition, we included the results obtained by directly using the nine individual         data sources that formed the aggregated collection of human PPIs. The objective here         was to assess each ranking method’s capability to retrieve true interactions in each         of the reference sets, as a function of rank, i.e., higher-ranked data should contain         a larger fraction of true interactions than lower-ranked data.      </p><div class="figs">      <div class="addfile "><p><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/suppl/S5','S5',630,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/suppl/S5"></a><strong>Additional file 5.</strong> Hypergeometric Ranked PPI data. This tab-delimited file contains the aggregated PPI         data set with the corresponding hypergeometric ranking. </p><p>         	Format: TXT          	Size: 2.3MB <a href="/content/supplementary/1471-2105-13-79-s5.txt">Download file</a><span class="open-data-icon"><a href="/about/access/#opendata"><img title="Open Data" alt="Open Data" src="/bmcimages/article/opendata.gif"></a></span></div></div>      <p style="line-height:160%">To make a fair comparison within each benchmark reference set, for each set we defined         a protein pair from the aggregated data set as a judgeable interaction if both proteins         appeared in the reference set; otherwise, we assumed that the pair could not be judged         by the reference set. If a judgeable interaction was observed in the reference set,         we termed it a true interaction. A good scheme would rank true interactions before         other judgeable interactions. To quantify the ability to retrieve true interactions         in each scored set, we extracted the judgeable subset and, using each corresponding         rank as a threshold, we counted the numbers of judgeable and true interactions with         scores above the threshold. We defined the number of true interactions as <em>coverage</em> and the fraction of true interactions among those judgeable interactions as <em>accuracy</em>.      </p>      <p style="line-height:160%">Figure <a name="d16426e2313"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/figure/F2">2</a> shows the results of this analysis for the six interaction benchmark reference sets.         As expected, the completely random PPI data set had no capability to retrieve direct         protein interaction data. For the network created by assigning random ranks to existing         PPIs, the accuracy was almost uniform at any given coverage.      </p><div class="figs">      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/figure/F2','F2',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/figure/F2"><img align="top" src="/content/figures/1471-2105-13-79-2.gif" alt="thumbnail" class="thumbnail"><strong>Figure 2.</strong></a> <strong>Retrieving protein-protein interactions.</strong> The proposed interaction detection based on shuffling (IDBOS) ranking scheme was         compared with hypergeometric test ranking, frequency of occurrence ranking, one randomly         ranked set (where the observed protein-protein interactions [PPIs] take random ranks),         a randomly rewired interaction set, and nine individual data sets. We compared these         ranking schemes using benchmark reference sets derived from six independent experimental         data sets, namely: (<strong>A</strong>) far-Western blotting, (<strong>B</strong>) isothermal titration calorimetry, (<strong>C</strong>) nuclear magnetic resonance, (<strong>D</strong>) surface plasmon resonance, (<strong>E</strong>) protein pairs in direct contact with each other within protein complex structures         collected from PDB (contacts determined using the jointly buried surface area), and         (<strong>F</strong>) mouse homologous protein interactions. We assumed that each reference set could         only judge a PPI when each of its two proteins actually appeared in this reference         set (judgeable). We assessed performance by <em>coverage</em> (number of true interactions) and <em>accuracy</em> (fraction of true interactions among judgeable interactions) as a function of varying         ranking thresholds (when applicable). The higher-ranked thresholds are to the left         in the plots. In the set scored by occurrence alone, multiple PPIs possessed the same         score, and the second right-most symbol entry corresponds to the collection of PPIs         with an observed occurrence ≥2.      </div></div>      <p style="line-height:160%">IDBOS, hypergeometric, and frequency of occurrence ranking were each associated with         enhanced accuracy at higher ranks, thus indicating that true interactions ranked before         other judgeable interactions. We assessed the overall performance of the ranking methods         by calculating relative improvement compared to using no ranking, which is equivalent         to assigning random ranks to the existing data. For the <em>accuracy</em> as a function of <em>coverage</em> plots shown in Figure <a name="d16426e2362"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/figure/F2">2</a>, we calculated the average <em>accuracy</em><span class="inlinenumber"><a title="View MathML" onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M4','MathML',630,470);return false;" href="http://www.biomedcentral.com/1471-2105/13/79/mathml/M4"><img class="mathimg" src="/content/inline/1471-2105-13-79-i4.gif" alt="<a onClick=&#34;popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M4','MathML',630,470);return false;&#34; target=&#34;_blank&#34; href=&#34;http://www.biomedcentral.com/1471-2105/13/79/mathml/M4&#34;&gt;View MathML</a&gt;"></a></span>over all ranks <em>r</em> as,      </p>      <p style="line-height:160%" class="inlinenumber">         <div style="display:table;width:100%;*display:inline">            <p class="inlinenumber"><a title="View MathML" onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M5','MathML',630,470);return false;" href="http://www.biomedcentral.com/1471-2105/13/79/mathml/M5"><img class="mathimg" src="/content/inline/1471-2105-13-79-i5.gif" alt="<a onClick=&#34;popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M5','MathML',630,470);return false;&#34; target=&#34;_blank&#34; href=&#34;http://www.biomedcentral.com/1471-2105/13/79/mathml/M5&#34;&gt;View MathML</a&gt;"></a></p><span style="font-size:80%;text-align:right;display:table-cell;vertical-align:center;*float:right;*display:inline;*position:relative;*top:6px">(3)</span></div>      </p>      <p style="line-height:160%">where <em>n</em>(<em>r</em>) is the number of true PPIs at rank <em>r</em> and <em>N</em> is the sum of all <em>n</em>(<em>r</em>)’s. The gain of using IDBOS, hypergeometric, or occurrence as a ranking method was         estimated by comparing the average <em>accuracy</em><span class="inlinenumber"><a title="View MathML" onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M6','MathML',630,470);return false;" href="http://www.biomedcentral.com/1471-2105/13/79/mathml/M6"><img class="mathimg" src="/content/inline/1471-2105-13-79-i6.gif" alt="<a onClick=&#34;popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M6','MathML',630,470);return false;&#34; target=&#34;_blank&#34; href=&#34;http://www.biomedcentral.com/1471-2105/13/79/mathml/M6&#34;&gt;View MathML</a&gt;"></a></span>to the randomly ranked data <span class="inlinenumber"><a title="View MathML" onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M7','MathML',630,470);return false;" href="http://www.biomedcentral.com/1471-2105/13/79/mathml/M7"><img class="mathimg" src="/content/inline/1471-2105-13-79-i7.gif" alt="<a onClick=&#34;popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M7','MathML',630,470);return false;&#34; target=&#34;_blank&#34; href=&#34;http://www.biomedcentral.com/1471-2105/13/79/mathml/M7&#34;&gt;View MathML</a&gt;"></a></span>as,      </p>      <p style="line-height:160%" class="inlinenumber">         <div style="display:table;width:100%;*display:inline">            <p class="inlinenumber"><a title="View MathML" onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M8','MathML',630,470);return false;" href="http://www.biomedcentral.com/1471-2105/13/79/mathml/M8"><img class="mathimg" src="/content/inline/1471-2105-13-79-i8.gif" alt="<a onClick=&#34;popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M8','MathML',630,470);return false;&#34; target=&#34;_blank&#34; href=&#34;http://www.biomedcentral.com/1471-2105/13/79/mathml/M8&#34;&gt;View MathML</a&gt;"></a></p><span style="font-size:80%;text-align:right;display:table-cell;vertical-align:center;*float:right;*display:inline;*position:relative;*top:6px">(4)</span></div>      </p>      <p style="line-height:160%">Table <a name="d16426e2532"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/table/T3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/table/T3">3</a> lists the gains in average accuracies in each reference set for the different ranking         schemes. The IDBOS ranking scheme shows the greatest increase compared to using randomly         assigned ranks, achieves a two-fold increase in accuracy compared with using frequency         of occurrence ranking, and consistently outperforms the hypergeometric ranking. In         practice, one should also note that selecting any high-accuracy subset of PPIs based         on ranked interaction data creates a smaller data set and lowers the overall PPI coverage.      </p><div class="figs">      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/table/T3','T3',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/table/T3"><strong>Table 3.</strong></a> <strong>Evaluation of different ranking schemes</strong></div></div>      <p style="line-height:160%">The independent scores (Z-scores or p-values) were typically better than, or equivalent         to, the hypergeometric method: Z-score ranking was close to the IDBOS, and better         than the hypergeometric, whereas the p-value ranking was not as good as the hypergeometric.         For example, using the PDB benchmarking set, the gains of the IDBOS, Z-score, hypergeometric,         and p-value methods relative to the average <em>accuracy</em> of the randomly ranked data were 258%, 253%, 217%, and 216%, respectively; using         the mouse homologous benchmarking set, the gains were 222%, 211%, 198%, and 155%,         respectively. Thus, the IDBOS combined rankings consistently outperformed all attempted         scoring schemes.      </p>      <h4>IDBOS versus hypergeometric test rankings</h4>      <p style="line-height:160%">The observed differences in Table <a name="d16426e2805"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/table/T3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/table/T3">3</a> and Figure <a name="d16426e2808"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/figure/F2">2</a> between the IDBOS and hypergeometric test rankings warrants further comment. The         fundamental difference between the IDBOS and hypergeometric test rankings lies in         how they account for interactions between self-interacting proteins. For the IDBOS         randomized data, each pair of proteins was re-assigned an interaction partner such         that no self-interacting protein pairs remained in the final set used to construct         the probability density distribution functions. In the hypergeometric test, self-interacting         protein pairs were assigned finite probabilities of occurrence based on a background         distribution for each protein pair, which was different from IDBOS. Conceptually,         the sum of the probability of observing all interactions with a given protein A among         all other proteins p<sub>AA</sub>, p<sub>AB</sub>, p<sub>AC</sub>, etc., was the same in both schemes. However, the constraint that p<sub>AA</sub> was zero in IDBOS and p<sub>AA</sub> was non-zero in the hypergeometric test, re-distributed the probabilities such that         any interaction probability p<sub>AB</sub> between protein A and another protein B could be different in the two schemes. This         strongly influenced the probability of detecting proteins that occurred with high         frequency in the data set.      </p>      <p style="line-height:160%">The effect of including or excluding self-interacting protein pairs was magnified         in the evaluation of interactions involving popular proteins. In the hypergeometric         test, the likelihood of randomly generating self-interacting protein pairs is roughly         proportional to the square of the number of times the protein appears in the data         set. In practice, this leads to an underestimate of the occurrence of non-identity         pairs in the random data, and this increases the significance attached to the observed         non-identity PPIs involving popular proteins. The effect on ranking PPIs was considerable,         e.g., among the top 20 occurrence-ranked interactions listed in Table <a name="d16426e2832"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/table/T2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/table/T2">2</a>, 11 appeared in the top 20 hypergeometric test ranking scheme. In fact, the first         occurrence-ranked interaction between MDM2 and TP53 was still the second ranked interaction         based on the hypergeometric test. Figure <a name="d16426e2835"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/figure/F3">3</a> shows the overlap of interactions among top-ranked PPIs based on IDBOS, the hypergeometric         test, and frequency of occurrence rankings. As exemplified above, there was a noteworthy         overlap (&gt;0.40) between the ranking results in the frequency of occurrence and the         hypergeometric test rankings for all ranks tested. In contrast, the first 10<sup>3</sup> top-ranked IDBOS PPIs showed low overlap (~0.10) with the frequency of occurrence         ranked PPIs, but higher similarity (~0.36) to the ranked PPIs identified using the         hypergeometric test. Thus, the high overlap between occurrence and hypergeometric         rankings suggested that the inadvertent biases introduced by frequent investigations         of popular proteins could not be completely disentangled by the hypergeometric-ranking         scheme. Furthermore, we verified that consensus ranking schemes that included the         average rank of all three ranking methods, or combining IDBOS and hypergeometric test         rankings only, did not increase the accuracy beyond using IDBOS rankings (data not         shown). Instead, we contend that the corresponding probability density distribution         functions generated by IDBOS are the most appropriate to gauge unbiased PPIs in this         data set.      </p><div class="figs">      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/figure/F3','F3',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/figure/F3"><img align="top" src="/content/figures/1471-2105-13-79-3.gif" alt="thumbnail" class="thumbnail"><strong>Figure 3.</strong></a> <strong>Overlap of top-ranked protein interactions.</strong> The overlap of top-ranked protein-protein interactions (PPIs), based on a pairwise         comparison of three interaction evaluation schemes: interaction detection based on         shuffling (IDBOS), hypergeometric test ranking, and frequency of occurrence ranking.         The overlap between hypergeometric and occurrence rankings was considerable. The right-most         point in the graph corresponds to the case where all interactions are included and,         by definition, the three schemes overlap. For intermediate rankings, IDBOS shows considerably         lower overlap with either method, indicative of distinct and different sets of top-ranked         PPIs. The right-most curve (Random) shows the expected overlap fraction of top-ranking         PPIs from two completely random rankings, emphasizing that even though the overlap         fraction between IDBOS and Occurrence ranking was low, it was considerably higher         than what would be expected by chance alone.      </div></div>      <h4>Enrichment of known domain-domain interactions</h4>      <p style="line-height:160%">The hypothesis that PPIs are mediated by a smaller set of specific domain-domain interactions         (DDIs) that are repeatedly used, can be exploited by inferring DDIs from known PPIs         and then predicting novel PPIs from the inferred DDI set <a name="d16426e2861"></a><a name="d16426e2863"></a>[<a onclick="LoadInParent('#B40'); return false;" href="#B40">40</a></a>,<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>]. A high-confidence PPI set, such as the collection of highly ranked PPIs, is more         likely to contain reliable DDIs. To confirm this, we investigated the fraction of         known DDIs among all candidate domain pairs as a function of the rank threshold. As         before, we only considered judgeable protein pairs whose domains appear in the set         of known DDIs. The known DDI set was inferred from the PDB crystal structures <a name="d16426e2867"></a>[<a onclick="LoadInParent('#B42'); return false;" href="#B42">42</a>]. As two random controls, we also randomly ranked and randomly rewired the judgeable         protein pairs. Figure <a name="d16426e2870"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/figure/F4">4</a> shows that more stringent ranking thresholds (represented by fewer “Top judgeable         PPIs” in Figure <a name="d16426e2873"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/figure/F4">4</a>) resulted in higher enrichments of DDIs among all domain pairs between interacting         proteins for both the IDBOS and frequency of occurrence rankings. This behavior was         in contrast with the two random controls, which did not show such enrichments. Table <a name="d16426e2876"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/table/T4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/table/T4">4</a> quantifies the improvement gained by ranking the data relative to no rankings. Within         the same top number of PPIs, the IDBOS set had a higher enrichment of known DDIs than         sets ranked either by the hypergeometric test or by frequency of occurrence, indicating         a potential application of the IDBOS-ranked PPI sets for the inference of accurate         DDIs.      </p><div class="figs">      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/figure/F4','F4',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/figure/F4"><img align="top" src="/content/figures/1471-2105-13-79-4.gif" alt="thumbnail" class="thumbnail"><strong>Figure 4.</strong></a> <strong>Recovery of known domain interactions from protein interactions.</strong> The proposed interaction detection based on shuffling (IDBOS) ranking scheme was         compared with the frequency of occurrence ranking, one randomly ranked set (where         the observed protein-protein interactions (PPIs) take random ranks), and a randomly         rewired interaction set in identifying known domain-domain interactions (DDIs). We         assessed performance by calculating the fraction of true DDIs among the judgeable         interactions as a function of varying ranking thresholds. In the set scored by occurrence         alone, multiple PPIs possessed the same score, and the second right-most symbol corresponds         to the collection of PPIs with an observed occurrence ≥2.      </div>      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/table/T4','T4',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/table/T4"><strong>Table 4.</strong></a> <strong>Domain-domain interaction enrichment</strong></div></div>      <h4>Evaluation of ranking PPIs as a means of retrieving biological information</h4>      <p style="line-height:160%">Up until now, the different scoring schemes were used to evaluate reference data sets         that can be considered to be directly related to interacting proteins. We next evaluated         the improvement that could be gained by using the differently ranked data sets to         rank-order the interactions in reference data sets that are presumed to be enriched         with interacting proteins, i.e., in sets of proteins that share the same Kyoto Encyclopedia         of Genes and Genomes (KEGG) pathway, are implicated in the same disease, share Gene         Ontology (GO) function, or tissue mRNA expression levels.      </p>      <h4>Case I: Enrichment of KEGG co-pathway gene pairs</h4>      <p style="line-height:160%">Genes encoding interacting proteins are more likely to be part of the same pathway.         Hence, the fraction of PPIs that are annotated to belong to the same pathway should         be larger in sets containing higher-ranked PPIs. We used KEGG pathway data <a name="d16426e3005"></a>[<a onclick="LoadInParent('#B43'); return false;" href="#B43">43</a>] to define co-pathway gene pairs, as outlined in the Materials and methods. For each         investigated PPI set, we excluded protein pairs involving proteins that did not participate         in any KEGG pathway (these were deemed to be non-judgeable). Figure <a name="d16426e3008"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/figure/F5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/figure/F5">5A</a> shows the results of this analysis. We confirmed the correlation between co-pathway         membership of a protein pair and its rank threshold in both the IDBOS and the frequency         of occurrence ranking schemes, as well as the absence of correlation in the two random         controls. Overall, IDBOS ranking outperformed frequency of occurrence ranking by a         factor of four for enrichment of co-pathway gene pairs among selected top-ranking         PPIs (Table <a name="d16426e3011"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/table/T5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/table/T5">5</a>), and the gain seen in hypergeometric ranking was substantially smaller than that         of IDBOS ranking. There is a discontinuity in the curve for the frequency of occurrence         ranking scheme, which was caused by hundreds of interactions between three subunits         of G protein (GNAL, GNB1, and GNGT1) and olfactory proteins which were supported by         seven publications (Additional file <a name="d16426e3014"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/suppl/S1">1</a>). We reviewed these publications and found that this literature support was weak,         with some based solely on gene expression evidence of those olfactory genes <a name="d16426e3018"></a>[<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>]. Among these hundreds of PPIs, those involved with GNB1 (beta subunit) or GNGT1 (gamma         subunit) were not supported by KEGG pathway data, resulting in a sharp drop in accuracy.      </p><div class="figs">      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/figure/F5','F5',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/figure/F5"><img align="top" src="/content/figures/1471-2105-13-79-5.gif" alt="thumbnail" class="thumbnail"><strong>Figure 5.</strong></a> <strong>Recovery of biological co-annotations from protein interactions.</strong> The proposed interaction detection based on shuffling (IDBOS) ranking scheme was         compared with the frequency of occurrence ranking, one randomly ranked set (where         the observed protein-protein interactions (PPIs) take random ranks), and a randomly         rewired interaction set in retrieving biological relationships from diverse reference         sets. The reference sets were: (<strong>A</strong>) co-pathway gene pairs, (<strong>B</strong>) co-disease susceptibility gene pairs, (<strong>C</strong>) functionally related gene pairs, and (<strong>D</strong>) tissue co-expressed gene pairs. We assessed performance by calculating the enrichment         fraction of true PPIs among the judgeable interactions as a function of varying ranking         thresholds. In the set scored by occurrence alone, multiple PPIs possessed the same         score, and the second right-most symbol corresponds to the collection of PPIs with         an observed occurrence ≥2.      </div>      <div class="table"><p><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/table/T5','T5',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/table/T5"><strong>Table 5.</strong></a> <strong>Evaluation of ranked interactions to detect biological relationships</strong></div></div>      <h4>Case II: Enrichment of co-disease gene pairs</h4>      <p style="line-height:160%">Genomic variations may underlie different susceptibilities to disease, e.g., individuals         with specific single nucleotide polymorphisms (SNPs) can develop, or be pre-disposed         to, a particular disease phenotype. Furthermore, genes encoding interacting proteins         are more likely to occur within the same disease classification <a name="d16426e3228"></a>[<a onclick="LoadInParent('#B45'); return false;" href="#B45">45</a>]. Using the gene co-disease data extracted by Goh et al. <a name="d16426e3232"></a>[<a onclick="LoadInParent('#B45'); return false;" href="#B45">45</a>] from the Online Mendelian Inheritance in Man (OMIM) dataset <a name="d16426e3236"></a>[<a onclick="LoadInParent('#B46'); return false;" href="#B46">46</a>], we investigated the enrichment of co-disease gene pairs as a function of rank threshold         (Figure <a name="d16426e3239"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/figure/F5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/figure/F5">5B</a>, Table <a name="d16426e3242"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/table/T5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/table/T5">5</a>). A gene pair was termed a co-disease gene pair if their SNPs led to susceptibility         to the same disease. With this co-disease gene pair set, we also extracted judgeable         protein pairs whose encoding genes appeared in this co-disease set (i.e., genes that         have an actual disease annotation). Similar to the previous analyses, the IDBOS-scored         set showed an enhanced probability of identifying existing disease-susceptible genes         as compared with the frequency of occurrence ranking scheme, hypergeometric-based         rankings, or random controls.      </p>      <h4>Case III: Enrichment of GO co-function gene pairs</h4>      <p style="line-height:160%">Figure <a name="d16426e3251"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/figure/F5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/figure/F5">5C</a> shows the ability of the IDBOS and frequency of occurrence ranking schemes to identify         PPIs whose proteins share the same gene ontology (GO) function. Both ranking schemes         displayed a similar rank-dependent enrichment of co-function proteins compared to         the randomly ranked (all ranks have the same importance) and randomly rewired (interactions         are random) data set. Table <a name="d16426e3254"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/table/T5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/table/T5">5</a> indicates that the improvements of the ranking schemes compared to randomly ranked         data are relatively modest in retrieving functionally related gene pairs. As pointed         out by Gillis and Pavlidis <a name="d16426e3258"></a>[<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>], the ability of PPI networks to distinguish co-functionality is largely influenced         by the multifunctional nature of the gene annotations schemes themselves. In this         case, the IDBOS, hypergeometric, and occurrence ranking schemes themselves will also         have less influence on the retrieval of functionally related protein pairs compared         to the annotation scheme itself.      </p>      <h4>Case IV: Enrichment of tissue co-expression gene pairs</h4>      <p style="line-height:160%">Finally, we examined the occurrence of interacting protein pairs in a large data set         that maps out the global mRNA expression levels for ~20,000 human genes in 32 normal         tissues <a name="d16426e3269"></a>[<a onclick="LoadInParent('#B47'); return false;" href="#B47">47</a>]. For detail on the construction of this reference set, see Materials and methods.         The biological hypothesis tested here examined co-expression of two mRNAs in the same         tissue as indicative that the corresponding proteins had higher probability of interaction.         Figure <a name="d16426e3272"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/figure/F5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/figure/F5">5D</a> shows the fraction of PPIs retrieved as a function of ranking scheme. Overall, the         fraction of PPIs contained in the co-expression data set was lower than that of the         other biological reference sets examined above. However, the investigated ranking         schemes were able to produce ranked data sets that contain a slightly larger fraction         of co-expressed PPIs than using no ranking (Table <a name="d16426e3275"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/table/T5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/table/T5">5</a>).      </p>      <h4>Topological differences of ranked PPI networks</h4>      <p style="line-height:160%">Data sets of protein interactions are commonly represented as graphs or networks of         protein interactions, and applying network topology metrics to characterize the biological         role of PPIs has attracted wide attention <a name="d16426e3285"></a><a name="d16426e3287"></a><a name="d16426e3289"></a><a name="d16426e3291"></a>[<a onclick="LoadInParent('#B9'); return false;" href="#B9">9</a></a>,<a onclick="LoadInParent('#B10'); return false;" href="#B10">10</a>,<a onclick="LoadInParent('#B22'); return false;" href="#B22">22</a>,<a onclick="LoadInParent('#B48'); return false;" href="#B48">48</a>]. The presence of confounding factors, e.g., protein abundance <a name="d16426e3295"></a>[<a onclick="LoadInParent('#B49'); return false;" href="#B49">49</a>] or popularity <a name="d16426e3299"></a>[<a onclick="LoadInParent('#B12'); return false;" href="#B12">12</a>], can strongly influence the topological properties of the network. Here, we used         subsets of highly ranked PPIs derived from using either IDBOS or occurrence ranking         to select smaller high-confidence networks. Figure <a name="d16426e3302"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/figure/F6">6A</a> shows the overall networks constructed from the top-ranking PPIs corresponding to         4,425, 6,561, and 13,554 interactions selected based on the number of PPIs that have         more than or equal to four, three, and two reported occurrences, respectively. Figure <a name="d16426e3305"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/figure/F6">6B</a> shows the corresponding degree distribution, i.e., the distribution of the number         of unique interacting partners each protein has (Degree), as well as the overall degree         distribution for the entire network (All).      </p><div class="figs">      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/figure/F6','F6',800,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/figure/F6"><img align="top" src="/content/figures/1471-2105-13-79-6.gif" alt="thumbnail" class="thumbnail"><strong>Figure 6.</strong></a> <strong>Network topology of ranked interactions</strong>. (<strong>A</strong>) We reconstructed the corresponding top-ranked protein-protein interactions (PPIs)         networks using both interaction detection based on shuffling (IDBOS) ranking and frequency         of occurrence ranking. The columns of 4,425, 6,561, and 13,554 top-ranked interactions         corresponds to selecting PPIs with ≥ 4, ≥ 3, and ≥ 2 reported occurrences, respectively.         These interactions were distributed among roughly twice as many proteins using IDBOS         ranking than in occurrence ranking. (<strong>B</strong>) The degree distribution for each selected PPI network was analyzed and compared         to the distribution of all aggregated interactions (All). The higher-ranked PPI data         sets were associated with fewer proteins that had a large number of interacting partners         (hubs). As visualized in (<strong>A</strong>), the top-ranked networks were instead characterized by more evenly distributed interactions         in a loosely interconnected network.      </div></div>      <p style="line-height:160%">Selecting highly ranked subsets of PPIs, using either IDBOS or occurrence, had the         effect of reducing the number of high-degree proteins (hubs) in the overall network.         This effect was most pronounced in the top-ranked IDBOS networks, which effectively         down-weights interactions of popular proteins. The resulting top-ranked IDBOS networks         were not dominated by hub interactions, but show a more distributed interaction network,         with distinct topological and scaling properties.      </p>   </div></section><section><a name="sec3"></a><h3>Discussion</h3>   <div class="collapsible-content">      <p style="line-height:160%">Single experiments designed for large-scale detection of PPIs judge multiple observations         of the same PPI as evidence for the confidence of the interaction. Multiple observations,         across many different experiments, of a particular PPI also lend confidence that this         interaction occurs in nature. One would then conclude that if one aggregated all known         PPI experiments, the more times a particular PPI occurred, the more confident one         would be of the interaction. We found a somewhat counterintuitive result in that ranking         PPIs in order of the number of times a PPI has been observed was actually not the         optimal way of assessing the importance of frequently reported interactions. Instead,         we used the IDBOS method, which ranks interacting protein pairs in the observed PPI         data sets as compared to random occurrences derived from an empirically reconstructed         probability density distribution function specific to each interaction. We then used         these ranks to order the PPI data and showed that this approach consistently identified         more known protein associations in numerous benchmark reference sets than using either         hypergeometric test rankings or simply frequency of occurrence. The improvement of         IDBOS over the hypergeometric test results stems from the differences in the underlying         random distributions in the two schemes. While the hypergeometric test does not impose         any limitations on self-interacting proteins, by construction, the IDBOS procedure         sets their probability of occurrence to zero even for the random case. The input data         contain no self-interacting proteins; therefore, the corresponding random probability         density distribution functions generated by IDBOS seem to be the most appropriate         to gauge unbiased PPIs in this data set.      </p>      <p style="line-height:160%">We evaluated the improvement in using ranked PPI data to identify known protein relationships         in eleven independent reference sets, including direct and indirect readouts of protein         interactions. We used a large number of different reference sets to perform as comprehensive         an analysis as possible, as each reference set only covers a specific subset of protein         interactions. It was clear that ranking interactions was of value, because higher-ranked         PPIs had better success at retrieving protein associations. Across all sets investigated,         occurrence ranking improved the performance by 46% compared to using non-ranked data,         hypergeometric ranking achieved a 109% increase, and the IDBOS ranking scheme achieved         a 134% increase. This conformed to the assumption that aggregating and creating high-confidence         subsets of data creates added value, albeit at a cost of reducing the overall number         of PPIs that can be considered.      </p>   </div></section><section><a name="sec4"></a><h3>Conclusions</h3>   <div class="collapsible-content">      <p style="line-height:160%">We have developed a statistical approach to infer subsets of high-confidence human         PPIs, and showed that ranked data can consistently enrich the accuracy of the retrieved         PPI data in these sets. Our IDBOS method was more successful in ranking interactions         than using either the number of times an interaction has been observed across experiments         or rankings based on the hypergeometric test. Furthermore, using either IDBOS or hypergeometric         scoring schemes generates unique ranks for almost all interactions, as opposed to         the frequency of occurrence method, in which many interactions have the same integer         score corresponding to the number of observed occurrences. The IDBOS ranking increased         accuracy and enrichment of protein interaction data associated with PPIs by more than         two-fold compared to simply ranking interactions based on observed occurrences. We         achieved this improvement by comparing the observed interaction data with a probability         density distribution function that does not inflate the statistical importance of         interactions associated with frequently studied proteins. These results suggest that         using our high-confidence protein interactions at different levels of confidence could         help clarify the dependence on confidence on topological and biological properties         associated with human protein networks.      </p>   </div></section><section><a name="sec5"></a><h3>Materials and methods</h3>   <div class="collapsible-content">      <h4>Statistical scoring of PPIs from aggregated experimental data</h4>      <p style="line-height:160%">We downloaded the collection of PPIs in October 2011 from the nine databases that         covers the bulk of all known experimentally determined PPIs. Databases of non-primary         nature, i.e., containing aggregated data and/or predicted and inferred interactions,         were excluded. From this collection, we extracted the subset of human physical PPIs,         consisting of PPIs from both large-, semi-large-, and small-scale experimental studies.         Because Reactome has no standardized annotations describing physical associations         or direct interactions, instead we extracted PPIs annotated as “direct complexes.”         We treated each interaction reported by each study (identified by a unique publication         ID) as a unique record by deleting redundant copies arising from the overlaps among         the nine databases. In total, we analyzed 13,369 proteins, 80,980 PPIs, and 116,134         records (Additional file <a name="d16426e3356"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/suppl/S1">1</a>). The computational procedure to generate 10<sup>6</sup> random realizations of the PPI data set and compute the corresponding p-values and         Z-scores took ~2,000 minutes on a dual core Xeon Irwindale 3.6 GHz 64-bit Linux server         equipped with 4 GB of RAM. We used fractional rankings, i.e., PPIs that had the same         score received the same ranking number, which is the mean of what they would have         received under ordinal rankings. The Supplementary material provides the scored and         ranked PPI data set, with equivalently ranked interactions tabulated in arbitrary         order.      </p>      <p style="line-height:160%">To rank interactions based on an alternative statistical method compared to IDBOS         and frequency of occurrence, we included ranks based on a hypergeometric test <a name="d16426e3365"></a>[<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>]. For two proteins, <em>i</em> and <em>j</em>, given that the interaction between <em>i</em> and <em>j</em> occurred <em>N</em><sub><em>ij</em></sub> times, the probability (<em>p</em><sub><em>ij</em></sub>) for these two proteins to have this or a larger number of interactions by chance         was approximated using a hypergeometric distribution as follows:      </p>      <p style="line-height:160%" class="inlinenumber">         <div style="display:table;width:100%;*display:inline">            <p class="inlinenumber"><a title="View MathML" onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M9','MathML',630,470);return false;" href="http://www.biomedcentral.com/1471-2105/13/79/mathml/M9"><img class="mathimg" src="/content/inline/1471-2105-13-79-i9.gif" alt="<a onClick=&#34;popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M9','MathML',630,470);return false;&#34; target=&#34;_blank&#34; href=&#34;http://www.biomedcentral.com/1471-2105/13/79/mathml/M9&#34;&gt;View MathML</a&gt;"></a></p><span style="font-size:80%;text-align:right;display:table-cell;vertical-align:center;*float:right;*display:inline;*position:relative;*top:6px">(5)</span></div>      </p>      <p style="line-height:160%">where <em>N</em><sub><em>i</em></sub> (<em>N</em><sub><em>j</em></sub>) is the number of times protein <em>i</em> (<em>j</em>) was observed in the aggregated data set, and <em>N</em> is the total number of interactions in the aggregated data set of PPIs. This formulation         is correct in the limit of <em>N</em> &gt; &gt; <em>N</em><sub><em>i</em></sub><em>N</em><sub><em>j</em></sub>, which was satisfied in this data set. Finally, we ranked the PPI data based on the         calculated probabilities.      </p>      <h4>PDB interactions</h4>      <p style="line-height:160%">We mapped all PDB sequences (downloaded in October 2010) to the human protein sequences         stored in the UniProtKB/Swiss-Prot database (downloaded in June 2008) and obtained         649 protein complexes that contained at least two different human proteins. We then         calculated the contact area of each intra-complex human protein pair using the program         EMPIRE <a name="d16426e3542"></a>[<a onclick="LoadInParent('#B50'); return false;" href="#B50">50</a>], to determine which protein pairs interact. For protein pairs occurring in multiple         complexes, we selected the pair with the largest contact area as an interacting pair.         We collected 281 direct protein interactions between 563 proteins, with contact areas         ranging from 0.1 to 183 nm<sup>2</sup>, as a reference set for known human PPIs derived from structural data (Additional         file <a name="d16426e3548"></a><a onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/suppl/S2','',630,470); return false;" href="http://www.biomedcentral.com/1471-2105/13/79/suppl/S2">2</a>).      </p>      <h4>Reference sets from the HIPPIE database</h4>      <p style="line-height:160%">We extracted four reference sets from the Human Integrated Protein-Protein Interaction         rEferenc (HIPPIE) data set (<a href="http://cbdm.mdc-berlin.de/tools/hippie/">http://cbdm.mdc-berlin.de/tools/hippie/</a> <a href="http://www.webcitation.org/query.php?url=http://cbdm.mdc-berlin.de/tools/hippie/&amp;refdoi=10.1186/1471-2105-13-79" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a> information.php, January 2012), each of which consists of PPIs detected by a top-scored         experiment technique (score = 10, specified with European Bioinformatics Institute         (EBI) term ID, and number of PPIs &gt;100). The extracted data set contained PPIs derived         from <em>1</em>) far-Western blotting, <em>2</em>) isothermal titration calorimetry, <em>3</em>) nuclear magnetic resonance, and <em>4</em>) surface plasmon resonance experiments.      </p>      <h4>Homologous human PPIs from mouse data</h4>      <p style="line-height:160%">We extracted the subset of mouse physical PPIs from the above collection and constructed         a homologous human PPI set according to the sequence homology between mouse and human         proteins defined by the National Center for Biotechnology Information (NCBI) homology         mapping scheme (<a href="ftp://ftp.ncbi.nih.gov/pub/HomoloGene/">ftp://ftp.ncbi.nih.gov/pub/HomoloGene/</a> <a href="http://www.webcitation.org/query.php?url=ftp://ftp.ncbi.nih.gov/pub/HomoloGene/&amp;refdoi=10.1186/1471-2105-13-79" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a>, October 2009). We used this homologous human PPI set (containing 3,148 interactions         between 5,632 proteins) as a reference data set reflective of direct protein interactions.      </p>      <h4>Domain annotation and interaction data</h4>      <p style="line-height:160%">We used the Pfam-A families of the Pfam21.0 database as the source for domain annotation         <a name="d16426e3588"></a>[<a onclick="LoadInParent('#B51'); return false;" href="#B51">51</a>]. We also downloaded the DDI set inferred from PDB crystal structures from the iPfam         database <a name="d16426e3592"></a>[<a onclick="LoadInParent('#B42'); return false;" href="#B42">42</a>]. This DDI set contains 4,030 interactions between 2,837 Pfam-A domains.      </p>      <h4>Co-disease susceptibility gene pairs</h4>      <p style="line-height:160%">We used the disease susceptibility gene data of Goh et al. <a name="d16426e3602"></a>[<a onclick="LoadInParent('#B45'); return false;" href="#B45">45</a>], which was constructed by processing OMIM raw data <a name="d16426e3606"></a>[<a onclick="LoadInParent('#B46'); return false;" href="#B46">46</a>]. There were 43,249 pairs between 3,670 genes whose two genes were susceptible to         at least one common disease. We used these pairs to represent the co-disease susceptibility         gene pairs.      </p>      <h4>Co-function gene pairs</h4>      <p style="line-height:160%">We downloaded the GO annotation data (<a href="http://www.geneontology.org">http://www.geneontology.org</a> <a href="http://www.webcitation.org/query.php?url=http://www.geneontology.org&amp;refdoi=10.1186/1471-2105-13-79" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a>, June 2010) to identify a set of functionally related gene pairs as a reference set.         For each gene, we expanded its GO annotation list by including all ancestors of each         member in the list. For two genes, <em>g</em> and <em>h</em>, given that they had <em>n</em><sub><em>gh</em></sub> GO annotations in common, the probability (<em>p</em><sub><em>gh</em></sub>) for these two genes to have this or larger annotation overlap by chance was estimated         from the hypergeometric distribution as follows:      </p>      <p style="line-height:160%" class="inlinenumber">         <div style="display:table;width:100%;*display:inline">            <p class="inlinenumber"><a title="View MathML" onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M10','MathML',630,470);return false;" href="http://www.biomedcentral.com/1471-2105/13/79/mathml/M10"><img class="mathimg" src="/content/inline/1471-2105-13-79-i10.gif" alt="<a onClick=&#34;popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M10','MathML',630,470);return false;&#34; target=&#34;_blank&#34; href=&#34;http://www.biomedcentral.com/1471-2105/13/79/mathml/M10&#34;&gt;View MathML</a&gt;"></a></p><span style="font-size:80%;text-align:right;display:table-cell;vertical-align:center;*float:right;*display:inline;*position:relative;*top:6px">(6)</span></div>      </p>      <p style="line-height:160%">where <em>N</em><sub><em>g</em></sub> (<em>N</em><sub><em>h</em></sub>) is the number of GO annotations of gene <em>g</em> (<em>h</em>) and <em>T</em> is the total number of unique GO annotations. We used –log(<em>p</em><sub><em>gh</em></sub>) as the score and chose the top 1% of all gene pairs as a reference set, resulting         in 1,502,420 co-function pairs for this reference set.      </p>      <h4>Co-pathway gene pairs</h4>      <p style="line-height:160%">The KEGG pathway data file (hsa_gene_map.tab) was downloaded from its Web site (<a href="ftp://ftp.genome.jp/pub/kegg/pathway/">ftp://ftp.genome.jp/pub/kegg/pathway/</a> <a href="http://www.webcitation.org/query.php?url=ftp://ftp.genome.jp/pub/kegg/pathway/&amp;refdoi=10.1186/1471-2105-13-79" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a>, June 2010), which lists the pathways that each annotated gene participates in. Similar         to the co-function score described above, the co-pathway score was also computed using         the hypergeometric distribution method. We chose the top 1% as the reference set of         significant co-pathway gene pairs (139,841 pairs).      </p>      <h4>Tissue co-expression of gene pairs</h4>      <p style="line-height:160%">We used a global expression data set containing mRNA expression levels for ~20,000         human genes in 32 normal tissues derived from massively parallel signature sequencing         <a name="d16426e3788"></a>[<a onclick="LoadInParent('#B47'); return false;" href="#B47">47</a>]. The raw data set documents the abundance values of each short sequence signature         tag (with a total of 182,727 tags) in 32 tissues. We mapped these tags to the regions         of the human genome (hg18) that encode genes, in both orientations. We obtained 105,512         hits in the gene regions in both orientations and, among them, 68,855 hits in the         gene orientation (p-value &lt; 10<sup>−2,000</sup>), indicating that the tags were able to distinguish the transcribed orientation from         the non-transcribed orientation in the genome. Assigning the tags that hit a gene         region and orientation to the corresponding gene, we obtained a set of tags for each         gene, resulting in a total of 14,516 genes having non-empty tag sets. We summed up         the abundance tissue profiles of the tags of a gene to create its raw expression profile.      </p>      <p style="line-height:160%">Furthermore, we computed the statistical significance of a gene being preferentially         expressed in a tissue (termed “tissue specificity”) using an approach similar to that         of Yu <em>et al.</em><a name="d16426e3799"></a>[<a onclick="LoadInParent('#B52'); return false;" href="#B52">52</a>], which identified tissue-specific genes from the NCBI Expressed Sequence Tag database.         Let <em>e</em><sub><em>k</em></sub>(<em>g</em>) be the expression level of gene <em>g</em> in tissue <em>k</em>. The total expression of gene <em>g</em> in all 32 tissues is,      </p>      <p style="line-height:160%" class="inlinenumber">         <div style="display:table;width:100%;*display:inline">            <p class="inlinenumber"><a title="View MathML" onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M11','MathML',630,470);return false;" href="http://www.biomedcentral.com/1471-2105/13/79/mathml/M11"><img class="mathimg" src="/content/inline/1471-2105-13-79-i11.gif" alt="<a onClick=&#34;popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M11','MathML',630,470);return false;&#34; target=&#34;_blank&#34; href=&#34;http://www.biomedcentral.com/1471-2105/13/79/mathml/M11&#34;&gt;View MathML</a&gt;"></a></p><span style="font-size:80%;text-align:right;display:table-cell;vertical-align:center;*float:right;*display:inline;*position:relative;*top:6px">(7)</span></div>      </p>      <p style="line-height:160%">and the expected total expression of all genes in tissue <em>k</em> is,      </p>      <p style="line-height:160%" class="inlinenumber">         <div style="display:table;width:100%;*display:inline">            <p class="inlinenumber"><a title="View MathML" onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M12','MathML',630,470);return false;" href="http://www.biomedcentral.com/1471-2105/13/79/mathml/M12"><img class="mathimg" src="/content/inline/1471-2105-13-79-i12.gif" alt="<a onClick=&#34;popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M12','MathML',630,470);return false;&#34; target=&#34;_blank&#34; href=&#34;http://www.biomedcentral.com/1471-2105/13/79/mathml/M12&#34;&gt;View MathML</a&gt;"></a></p><span style="font-size:80%;text-align:right;display:table-cell;vertical-align:center;*float:right;*display:inline;*position:relative;*top:6px">(8)</span></div>      </p>      <p style="line-height:160%">If we randomly throw <em>E</em>(<em>g</em>) darts into 32 areas of sizes <em>E</em><sub><em>k</em></sub>, <em>k</em> = 1, 2, …32, and each dart has a probability,      </p>      <p style="line-height:160%" class="inlinenumber">         <div style="display:table;width:100%;*display:inline">            <p class="inlinenumber"><a title="View MathML" onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M13','MathML',630,470);return false;" href="http://www.biomedcentral.com/1471-2105/13/79/mathml/M13"><img class="mathimg" src="/content/inline/1471-2105-13-79-i13.gif" alt="<a onClick=&#34;popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M13','MathML',630,470);return false;&#34; target=&#34;_blank&#34; href=&#34;http://www.biomedcentral.com/1471-2105/13/79/mathml/M13&#34;&gt;View MathML</a&gt;"></a></p><span style="font-size:80%;text-align:right;display:table-cell;vertical-align:center;*float:right;*display:inline;*position:relative;*top:6px">(9)</span></div>      </p>      <p style="line-height:160%">of hitting area <em>k</em>, we would expect to see <em>E</em>(<em>g</em>) <em>q</em><sub><em>k</em></sub> darts in this area with variance <em>E</em>(<em>g</em>)<em>q</em><sub><em>k</em></sub>(<em>1-q</em><sub><em>k</em></sub>). Similarly, if gene <em>g</em> were equally expressed across all tissues, the expected expression level in tissue         <em>i</em> would be <em>E</em>(<em>g</em>)<em>q</em><sub><em>k</em></sub> with variance <em>E</em>(<em>g</em>)<em>q</em><sub><em>k</em></sub>(<em>1-q</em><sub><em>k</em></sub>). Thus, we used the corresponding <em>Z</em>-score,      </p>      <p style="line-height:160%" class="inlinenumber">         <div style="display:table;width:100%;*display:inline">            <p class="inlinenumber"><a title="View MathML" onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M14','MathML',630,470);return false;" href="http://www.biomedcentral.com/1471-2105/13/79/mathml/M14"><img class="mathimg" src="/content/inline/1471-2105-13-79-i14.gif" alt="<a onClick=&#34;popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M14','MathML',630,470);return false;&#34; target=&#34;_blank&#34; href=&#34;http://www.biomedcentral.com/1471-2105/13/79/mathml/M14&#34;&gt;View MathML</a&gt;"></a></p><span style="font-size:80%;text-align:right;display:table-cell;vertical-align:center;*float:right;*display:inline;*position:relative;*top:6px">(10)</span></div>      </p>      <p style="line-height:160%">as the tissue specificity of gene <em>g</em> in tissue <em>k</em>. Accordingly, we defined the tissue co-expression score of genes <em>g</em> and <em>h</em> as,      </p>      <p style="line-height:160%" class="inlinenumber">         <div style="display:table;width:100%;*display:inline">            <p class="inlinenumber"><a title="View MathML" onclick="popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M15','MathML',630,470);return false;" href="http://www.biomedcentral.com/1471-2105/13/79/mathml/M15"><img class="mathimg" src="/content/inline/1471-2105-13-79-i15.gif" alt="<a onClick=&#34;popup('http://www.biomedcentral.com/1471-2105/13/79/mathml/M15','MathML',630,470);return false;&#34; target=&#34;_blank&#34; href=&#34;http://www.biomedcentral.com/1471-2105/13/79/mathml/M15&#34;&gt;View MathML</a&gt;"></a></p><span style="font-size:80%;text-align:right;display:table-cell;vertical-align:center;*float:right;*display:inline;*position:relative;*top:6px">(11)</span></div>      </p>      <p style="line-height:160%">We chose the top 1% of these scored gene pairs as the reference set of co-expressed         gene pairs to evaluate the PPI scoring approaches. There were 1,053,529 co-expression         pairs in this set.      </p>   </div></section><section><a name="sec6"></a><h3>Competing Interests</h3>   <div class="collapsible-content">      <p style="line-height:160%">The authors declare that they have no competing interests.</p>   </div></section><section><a name="sec7"></a><h3>Authors’ contributions</h3>   <div class="collapsible-content">      <p style="line-height:160%">XY, AW, and JR conceived of the study and participated in its design and coordination.         XY collected the data, developed the algorithms, and performed the calculations. AW         and XY drafted the original manuscript, which was edited by JR. All authors read and         approved the final manuscript.      </p>   </div></section><section><a name="ack"></a><h3>Acknowledgements</h3>   <div class="collapsible-content">      <p style="line-height:160%">We thank Drs. Jacob Feala, Vesna Memišević, and Xin Hu for helpful discussions. The         authors were supported by the Military Operational Medicine Research Program of the         U.S. Army Medical Research and Materiel Command, Ft. Detrick, Maryland, as part of         the U.S. Army's Network Science Initiative. The opinions and assertions contained         herein are the private views of the authors and are not to be construed as official         or as reflecting the views of the U.S. Army or the U.S. Department of Defense. This         paper has been approved for public release with unlimited distribution.      </p>   </div></section><section><a name="refs"></a><h3>References</h3>   <div id="article-references" class="collapsible-content">      <ol id="references">         <li id="B1">            <p><a name="B1"></a> Rual JF,  Venkatesan K,  Hao T,  Hirozane-Kishikawa T,  Dricot A,  Li N,  Berriz GF,  Gibbons FD,  Dreze M,  Ayivi-Guedehoussou N, <em>et al</em>.: <strong> Towards a proteome-scale map of the human protein-protein interaction network. </strong><p><em>Nature</em> 2005,  <strong>437</strong>(7062)<strong>:</strong>1173-1178.  <a target="_blank" href="/pubmed/16189514">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16189514">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B1" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B1','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B2">            <p><a name="B2"></a> Stelzl U,  Worm U,  Lalowski M,  Haenig C,  Brembeck FH,  Goehler H,  Stroedicke M,  Zenkner M,  Schoenherr A,  Koeppen S, <em>et al</em>.: <strong> A human protein-protein interaction network: a resource for annotating the proteome. </strong><p><em>Cell</em> 2005,  <strong>122</strong>(6)<strong>:</strong>957-968.  <a target="_blank" href="/pubmed/16169070">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16169070">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B2" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B2','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B3">            <p><a name="B3"></a> Ewing RM,  Chu P,  Elisma F,  Li H,  Taylor P,  Climie S,  McBroom-Cerajewski L,  Robinson MD,  O'Connor L,  Li M, <em>et al</em>.: <strong> Large-scale mapping of human protein-protein interactions by mass spectrometry. </strong><p><em>Mol Syst Biol</em> 2007,  <strong>3</strong><strong>:</strong>89.  <a target="_blank" href="/pubmed/17353931">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17353931">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17353931">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B3" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B3','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B4">            <p><a name="B4"></a> Jeronimo C,  Forget D,  Bouchard A,  Li Q,  Chua G,  Poitras C,  Therien C,  Bergeron D,  Bourassa S,  Greenblatt J, <em>et al</em>.: <strong> Systematic analysis of the protein interaction network for the human transcription                  machinery reveals the identity of the 7SK capping enzyme. </strong><p><em>Mol Cell</em> 2007,  <strong>27</strong>(2)<strong>:</strong>262-274.  <a target="_blank" href="/pubmed/17643375">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17643375">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B4" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B4','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B5">            <p><a name="B5"></a> Sowa ME,  Bennett EJ,  Gygi SP,  Harper JW: <strong> Defining the human deubiquitinating enzyme interaction landscape. </strong><p><em>Cell</em> 2009,  <strong>138</strong>(2)<strong>:</strong>389-403.  <a target="_blank" href="/pubmed/19615732">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19615732">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19615732">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B5" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B5','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B6">            <p><a name="B6"></a> Suthram S,  Shlomi T,  Ruppin E,  Sharan R,  Ideker T: <strong> A direct comparison of protein interaction confidence assignment schemes. </strong><p><em>BMC Bioinforma</em> 2006,  <strong>7</strong><strong>:</strong>360.  <a target="_blank" href="http://dx.doi.org/10.1186/1471-2105-7-360">BioMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B6" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B6','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B7">            <p><a name="B7"></a> Schelhorn SE,  Mestre J,  Albrecht M,  Zotenko E: <strong> Inferring physical protein contacts from large-scale purification data of protein                  complexes. </strong><p><em>Mol Cell Proteomics</em> 2011,  <strong>10</strong>(6)<strong>:</strong>M110 004929.  <a target="_blank" href="/pubmed/21451165">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21451165">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21451165">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B7" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B7','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B8">            <p><a name="B8"></a> Yu X,  Ivanic J,  Memisevic V,  Wallqvist A,  Reifman J: <strong> Categorizing biases in high-confidence high-throughput protein-protein interaction                  data sets. </strong><p><em>Mol Cell Proteomics</em> 2011,  <strong>11</strong><strong>:</strong>M111 012500.                <p>in press</p> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B8" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B8','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B9">            <p><a name="B9"></a> Wodak SJ,  Pu S,  Vlasblom J,  Seraphin B: <strong> Challenges and rewards of interaction proteomics. </strong><p><em>Mol Cell Proteomics</em> 2009,  <strong>8</strong>(1)<strong>:</strong>3-18.  <a target="_blank" href="/pubmed/18799807">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18799807">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B9" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B9','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B10">            <p><a name="B10"></a> Yu H,  Braun P,  Yildirim MA,  Lemmens I,  Venkatesan K,  Sahalie J,  Hirozane-Kishikawa T,  Gebreab F,  Li N,  Simonis N, <em>et al</em>.: <strong> High-quality binary protein interaction map of the yeast interactome network. </strong><p><em>Science</em> 2008,  <strong>322</strong>(5898)<strong>:</strong>104-110.  <a target="_blank" href="/pubmed/18719252">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18719252">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18719252">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B10" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B10','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B11">            <p><a name="B11"></a> Hakes L,  Pinney JW,  Robertson DL,  Lovell SC: <strong> Protein-protein interaction networks and biology–what's the connection? </strong><p><em>Nat Biotechnol</em> 2008,  <strong>26</strong>(1)<strong>:</strong>69-72.  <a target="_blank" href="/pubmed/18183023">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18183023">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B11" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B11','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B12">            <p><a name="B12"></a> Pfeiffer T,  Hoffmann R: <strong> Large-scale assessment of the effect of popularity on the reliability of research. </strong><p><em>PLoS One</em> 2009,  <strong>4</strong>(6)<strong>:</strong>e5996.  <a target="_blank" href="/pubmed/19551148">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19551148">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19551148">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B12" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B12','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B13">            <p><a name="B13"></a> Bader GD,  Betel D,  Hogue CW: <strong> BIND: the Biomolecular Interaction Network Database. </strong><p><em>Nucleic Acids Res</em> 2003,  <strong>31</strong>(1)<strong>:</strong>248-250.  <a target="_blank" href="/pubmed/12519993">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12519993">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=12519993">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B13" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B13','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B14">            <p><a name="B14"></a> Stark C,  Breitkreutz BJ,  Reguly T,  Boucher L,  Breitkreutz A,  Tyers M: <strong> BioGRID: a general repository for interaction datasets. </strong><p><em>Nucleic Acids Res</em> 2006,  <strong>34</strong>(Database issue)<strong>:</strong>D535-539.  <a target="_blank" href="/pubmed/16381927">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16381927">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=16381927">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B14" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B14','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B15">            <p><a name="B15"></a> Salwinski L,  Miller CS,  Smith AJ,  Pettit FK,  Bowie JU,  Eisenberg D: <strong> The Database of Interacting Proteins: 2004 update. </strong><p><em>Nucleic Acids Res</em> 2004,  <strong>32</strong>(Database issue)<strong>:</strong>D449-451.  <a target="_blank" href="/pubmed/14681454">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=14681454">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=14681454">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B15" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B15','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B16">            <p><a name="B16"></a> Peri S,  Navarro JD,  Amanchy R,  Kristiansen TZ,  Jonnalagadda CK,  Surendranath V,  Niranjan V,  Muthusamy B,  Gandhi TK,  Gronborg M, <em>et al</em>.: <strong> Development of human protein reference database as an initial platform for approaching                  systems biology in humans. </strong><p><em>Genome Res</em> 2003,  <strong>13</strong>(10)<strong>:</strong>2363-2371.  <a target="_blank" href="/pubmed/14525934">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=14525934">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=14525934">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B16" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B16','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B17">            <p><a name="B17"></a> Aranda B,  Achuthan P,  Alam-Faruque Y,  Armean I,  Bridge A,  Derow C,  Feuermann M,  Ghanbarian AT,  Kerrien S,  Khadake J, <em>et al</em>.: <strong> The IntAct molecular interaction database in 2010. </strong><p><em>Nucleic Acids Res</em> 2009,  <strong>38</strong>(Database issue)<strong>:</strong>D525-531.  <a target="_blank" href="/pubmed/19850723">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19850723">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19850723">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B17" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B17','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B18">            <p><a name="B18"></a> Chatr-aryamontri A,  Ceol A,  Palazzi LM,  Nardelli G,  Schneider MV,  Castagnoli L,  Cesareni G: <strong> MINT: the Molecular INTeraction database. </strong><p><em>Nucleic Acids Res</em> 2007,  <strong>35</strong>(Database issue)<strong>:</strong>D572-574.  <a target="_blank" href="/pubmed/17135203">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17135203">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17135203">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B18" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B18','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B19">            <p><a name="B19"></a> Pagel P,  Kovac S,  Oesterheld M,  Brauner B,  Dunger-Kaltenbach I,  Frishman G,  Montrone C,  Mark P,  Stumpflen V,  Mewes HW, <em>et al</em>.: <strong> The MIPS mammalian protein-protein interaction database. </strong><p><em>Bioinformatics</em> 2005,  <strong>21</strong>(6)<strong>:</strong>832-834.  <a target="_blank" href="/pubmed/15531608">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15531608">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B19" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B19','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B20">            <p><a name="B20"></a> Beuming T,  Skrabanek L,  Niv MY,  Mukherjee P,  Weinstein H: <strong> PDZBase: a protein-protein interaction database for PDZ-domains. </strong><p><em>Bioinformatics</em> 2005,  <strong>21</strong>(6)<strong>:</strong>827-828.  <a target="_blank" href="/pubmed/15513994">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15513994">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B20" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B20','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B21">            <p><a name="B21"></a> Vastrik I,  D'Eustachio P,  Schmidt E,  Gopinath G,  Croft D,  de Bono B,  Gillespie M,  Jassal B,  Lewis S,  Matthews L, <em>et al</em>.: <strong> Reactome: a knowledge base of biologic pathways and processes. </strong><p><em>Genome Biol</em> 2007,  <strong>8</strong>(3)<strong>:</strong>R39.  <a target="_blank" href="/pubmed/17367534">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/gb-2007-8-3-r39">BioMed&nbsp;Central&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17367534">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B21" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B21','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B22">            <p><a name="B22"></a> Yu X,  Ivanic J,  Wallqvist A,  Reifman J: <strong> A novel scoring approach for protein co-purification data reveals high interaction                  specificity. </strong><p><em>PLoS Comput Biol</em> 2009,  <strong>5</strong>(9)<strong>:</strong>e1000515.  <a target="_blank" href="/pubmed/19779545">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19779545">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19779545">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B22" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B22','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B23">            <p><a name="B23"></a> Hart GT,  Lee I,  Marcotte ER: <strong> A high-accuracy consensus map of yeast protein complexes reveals modular nature of                  gene essentiality. </strong><p><em>BMC Bioinforma</em> 2007,  <strong>8</strong><strong>:</strong>236.  <a target="_blank" href="http://dx.doi.org/10.1186/1471-2105-8-236">BioMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B23" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B23','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B24">            <p><a name="B24"></a> Deane CM,  Salwinski L,  Xenarios I,  Eisenberg D: <strong> Protein interactions: two methods for assessment of the reliability of high throughput                  observations. </strong><p><em>Mol Cell Proteomics</em> 2002,  <strong>1</strong>(5)<strong>:</strong>349-356.  <a target="_blank" href="/pubmed/12118076">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12118076">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B24" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B24','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B25">            <p><a name="B25"></a> Deng M,  F Sun,  T Chen: <strong> Assessment of the reliability of protein-protein interactions and protein function                  prediction. </strong><p><em>Pacific Symposium on Biocomputing Pacific Symposium on Biocomputing</em> 2003,  <strong>8</strong>(140)<strong>:</strong>151-4376.  <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B25" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B25','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B26">            <p><a name="B26"></a> Goldberg DS,  Roth FP: <strong> Assessing experimentally derived interactions in a small world. </strong><p><em>Proc Natl Acad Sci U S A</em> 2003,  <strong>100</strong>(8)<strong>:</strong>4372-4376.  <a target="_blank" href="/pubmed/12676999">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12676999">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=12676999">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B26" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B26','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B27">            <p><a name="B27"></a> Bossi A,  Lehner B: <strong> Tissue specificity and the human protein interaction network. </strong><p><em>Mol Syst Biol</em> 2009,  <strong>5</strong><strong>:</strong>260.  <a target="_blank" href="/pubmed/19357639">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19357639">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19357639">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B27" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B27','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B28">            <p><a name="B28"></a> Gillis J,  Pavlidis P: <strong> The impact of multifunctional genes on "guilt by association" analysis. </strong><p><em>PLoS One</em> 2011,  <strong>6</strong>(2)<strong>:</strong>e17258.  <a target="_blank" href="/pubmed/21364756">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21364756">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21364756">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B28" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B28','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B29">            <p><a name="B29"></a> Wynn RM,  Kato M,  Machius M,  Chuang JL,  Li J,  Tomchick DR,  Chuang DT: <strong> Molecular mechanism for regulation of the human mitochondrial branched-chain alpha-ketoacid                  dehydrogenase complex by phosphorylation. </strong><p><em>Structure</em> 2004,  <strong>12</strong>(12)<strong>:</strong>2185-2196.  <a target="_blank" href="/pubmed/15576032">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15576032">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B29" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B29','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B30">            <p><a name="B30"></a> Grissom PM,  Vaisberg EA,  McIntosh JR: <strong> Identification of a novel light intermediate chain (D2LIC) for mammalian cytoplasmic                  dynein 2. </strong><p><em>Mol Biol Cell</em> 2002,  <strong>13</strong>(3)<strong>:</strong>817-829.  <a target="_blank" href="/pubmed/11907264">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11907264">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=11907264">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B30" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B30','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B31">            <p><a name="B31"></a> Mikami A,  Tynan SH,  Hama T,  Luby-Phelps K,  Saito T,  Crandall JE,  Besharse JC,  Vallee RB: <strong> Molecular structure of cytoplasmic dynein 2 and its distribution in neuronal and ciliated                  cells. </strong><p><em>J Cell Sci</em> 2002,  <strong>115</strong>(Pt 24)<strong>:</strong>4801-4808.  <a target="_blank" href="/pubmed/12432068">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12432068">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B31" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B31','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B32">            <p><a name="B32"></a> Cabello OA,  Eliseeva E,  He WG,  Youssoufian H,  Plon SE,  Brinkley BR,  Belmont JW: <strong> Cell cycle-dependent expression and nucleolar localization of hCAP-H. </strong><p><em>Mol Biol Cell</em> 2001,  <strong>12</strong>(11)<strong>:</strong>3527-3537.  <a target="_blank" href="/pubmed/11694586">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11694586">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=11694586">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B32" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B32','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B33">            <p><a name="B33"></a> Wang S,  Zhu G,  Chapoval AI,  Dong H,  Tamada K,  Ni J,  Chen L: <strong> Costimulation of T cells by B7-H2, a B7-like molecule that binds ICOS. </strong><p><em>Blood</em> 2000,  <strong>96</strong>(8)<strong>:</strong>2808-2813.  <a target="_blank" href="/pubmed/11023515">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11023515">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B33" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B33','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B34">            <p><a name="B34"></a> Wang S,  Zhu G,  Tamada K,  Chen L,  Bajorath J: <strong> Ligand binding sites of inducible costimulator and high avidity mutants with improved                  function. </strong><p><em>J Exp Med</em> 2002,  <strong>195</strong>(8)<strong>:</strong>1033-1041.  <a target="_blank" href="/pubmed/11956294">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11956294">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=11956294">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B34" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B34','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B35">            <p><a name="B35"></a> Volz A,  Goke R,  Lankat-Buttgereit B,  Fehmann HC,  Bode HP,  Goke B: <strong> Molecular cloning, functional expression, and signal transduction of the GIP-receptor                  cloned from a human insulinoma. </strong><p><em>FEBS Lett</em> 1995,  <strong>373</strong>(1)<strong>:</strong>23-29.  <a target="_blank" href="/pubmed/7589426">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=7589426">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B35" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B35','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B36">            <p><a name="B36"></a> Gallwitz B,  Witt M,  Morys-Wortmann C,  Folsch UR,  Schmidt WE: <strong> GLP-1/GIP chimeric peptides define the structural requirements for specific ligand-receptor                  interaction of GLP-1. </strong><p><em>Regul Pept</em> 1996,  <strong>63</strong>(1)<strong>:</strong>17-22.  <a target="_blank" href="/pubmed/8795084">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=8795084">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B36" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B36','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B37">            <p><a name="B37"></a> Manhart S,  Hinke SA,  McIntosh CH,  Pederson RA,  Demuth HU: <strong> Structure-function analysis of a series of novel GIP analogues containing different                  helical length linkers. </strong><p><em>Biochemistry</em> 2003,  <strong>42</strong>(10)<strong>:</strong>3081-3088.  <a target="_blank" href="/pubmed/12627975">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12627975">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B37" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B37','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B38">            <p><a name="B38"></a> Yamada Y,  Seino Y: <strong> Physiology of GIP–a lesson from GIP receptor knockout mice. </strong><p><em>Horm Metab Res</em> 2004,  <strong>36</strong>(11–12)<strong>:</strong>771-774.  <a target="_blank" href="/pubmed/15655707">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15655707">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B38" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B38','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B39">            <p><a name="B39"></a> Tressel T,  Thompson R,  Zieske LR,  Menendez MI,  Davis L: <strong> Interaction between L-threonine dehydrogenase and aminoacetone synthetase and mechanism                  of aminoacetone production. </strong><p><em>J Biol Chem</em> 1986,  <strong>261</strong>(35)<strong>:</strong>16428-16437.  <a target="_blank" href="/pubmed/3536927">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=3536927">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B39" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B39','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B40">            <p><a name="B40"></a> Ta HX,  Holm L: <strong> Evaluation of different domain-based methods in protein interaction prediction. </strong><p><em>Biochem Biophys Res Commun</em> 2009,  <strong>390</strong>(3)<strong>:</strong>357-362.  <a target="_blank" href="/pubmed/19800868">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19800868">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B40" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B40','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B41">            <p><a name="B41"></a> Gupta S,  Wallqvist A,  Bondugula R,  Ivanic J,  Reifman J: <strong> Unraveling the conundrum of seemingly discordant protein-protein interaction datasets. </strong><p><em>Conf Proc IEEE Eng Med Biol Soc</em> 2010,  <strong>2010</strong><strong>:</strong>783-786.  <a target="_blank" href="/pubmed/21096109">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21096109">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B41" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B41','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B42">            <p><a name="B42"></a> Finn RD,  Marshall M,  Bateman A: <strong> iPfam: visualization of protein-protein interactions in PDB at domain and amino acid                  resolutions. </strong><p><em>Bioinformatics</em> 2005,  <strong>21</strong>(3)<strong>:</strong>410-412.  <a target="_blank" href="/pubmed/15353450">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15353450">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B42" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B42','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B43">            <p><a name="B43"></a> Kanehisa M,  Goto S: <strong> KEGG: kyoto encyclopedia of genes and genomes. </strong><p><em>Nucleic Acids Res</em> 2000,  <strong>28</strong>(1)<strong>:</strong>27-30.  <a target="_blank" href="/pubmed/10592173">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10592173">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=10592173">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B43" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B43','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B44">            <p><a name="B44"></a> Zhang X,  De la Cruz O,  Pinto JM,  Nicolae D,  Firestein S,  Gilad Y: <strong> Characterizing the expression of the human olfactory receptor gene family using a                  novel DNA microarray. </strong><p><em>Genome Biol</em> 2007,  <strong>8</strong>(5)<strong>:</strong>R86.  <a target="_blank" href="/pubmed/17509148">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/gb-2007-8-5-r86">BioMed&nbsp;Central&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17509148">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B44" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B44','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B45">            <p><a name="B45"></a> Goh KI,  Cusick ME,  Valle D,  Childs B,  Vidal M,  Barabasi AL: <strong> The human disease network. </strong><p><em>Proc Natl Acad Sci U S A</em> 2007,  <strong>104</strong>(21)<strong>:</strong>8685-8690.  <a target="_blank" href="/pubmed/17502601">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17502601">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17502601">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B45" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B45','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B46">            <p><a name="B46"></a> Hamosh A,  Scott AF,  Amberger JS,  Bocchini CA,  McKusick VA: <strong> Online Mendelian Inheritance in Man (OMIM), a knowledgebase of human genes and genetic                  disorders. </strong><p><em>Nucleic Acids Res</em> 2005,  <strong>33</strong>(Database issue)<strong>:</strong>D514-517.  <a target="_blank" href="/pubmed/15608251">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15608251">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15608251">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B46" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B46','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B47">            <p><a name="B47"></a> Jongeneel CV,  Delorenzi M,  Iseli C,  Zhou D,  Haudenschild CD,  Khrebtukova I,  Kuznetsov D,  Stevenson BJ,  Strausberg RL,  Simpson AJ, <em>et al</em>.: <strong> An atlas of human gene expression from massively parallel signature sequencing (MPSS). </strong><p><em>Genome Res</em> 2005,  <strong>15</strong>(7)<strong>:</strong>1007-1014.  <a target="_blank" href="/pubmed/15998913">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15998913">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15998913">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B47" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B47','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B48">            <p><a name="B48"></a> Pierre S,  Scholich K: <strong> Toponomics: studying protein-protein interactions and protein networks in intact tissue. </strong><p><em>Mol Biosyst</em> 2010,  <strong>6</strong>(4)<strong>:</strong>641-647.  <a target="_blank" href="/pubmed/20237641">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20237641">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B48" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B48','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B49">            <p><a name="B49"></a> Ivanic J,  Yu X,  Wallqvist A,  Reifman J: <strong> Influence of protein abundance on high-throughput protein-protein interaction detection. </strong><p><em>PLoS One</em> 2009,  <strong>4</strong>(6)<strong>:</strong>e5815.  <a target="_blank" href="/pubmed/19503833">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19503833">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19503833">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B49" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B49','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B50">            <p><a name="B50"></a> Liang S,  Liu S,  Zhang C,  Zhou Y: <strong> A simple reference state makes a significant improvement in near-native selections                  from structurally refined docking decoys. </strong><p><em>Proteins</em> 2007,  <strong>69</strong>(2)<strong>:</strong>244-253.  <a target="_blank" href="/pubmed/17623864">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17623864">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17623864">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B50" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B50','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B51">            <p><a name="B51"></a> Bateman A,  Coin L,  Durbin R,  Finn RD,  Hollich V,  Griffiths-Jones S,  Khanna A,  Marshall M,  Moxon S,  Sonnhammer EL, <em>et al</em>.: <strong> The Pfam protein families database. </strong><p><em>Nucleic Acids Res</em> 2004,  <strong>32</strong>(Database issue)<strong>:</strong>D138-141.  <a target="_blank" href="/pubmed/14681378">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=14681378">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=14681378">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B51" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B51','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>         <li id="B52">            <p><a name="B52"></a> Yu X,  Lin J,  Zack DJ,  Qian J: <strong> Computational analysis of tissue-specific combinatorial gene regulation: predicting                  interaction between transcription factors in human tissues. </strong><p><em>Nucleic Acids Res</em> 2006,  <strong>34</strong>(17)<strong>:</strong>4925-4936.  <a target="_blank" href="/pubmed/16982645">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16982645">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=16982645">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2105-13-79&amp;bibl=B52" onclick="popup('/sfx_links?ui=1471-2105-13-79&amp;bibl=B52','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>               <p class="totext"><script type="text/javascript">totext()</script></p>            </p>         </li>      </ol>   </div></section>	</div>	                    </div>                </div>    </div>    </div>    <div class="b png_bg">        <div class="r png_bg"></div>    </div></div>		</div></div>    <dl class="google-ad wide ">	<dt class="hide">					<a class="banner-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising"></a>							</dt><dd><!-- OAS AD 'Bottom' begin --><script type="text/javascript">OAS_AD('Bottom');</script><!-- OAS AD 'Bottom' end --></dd></dl><noscript><dl class="google-ad  noscript">      <dt class="hide"><a href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</dt>          <dd>        <a href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/biomedcentral.com/bmcbioinformatics/article/10.1186/1471/2105/13/79/17493322870@Bottom?">            <img src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/biomedcentral.com/bmcbioinformatics/article/10.1186/1471/2105/13/79/17452094629@Bottom?" alt="advert"/>        </a>    </dd></dl></noscript>    <hr class="hide" />	    <div id="footer">					<span class="views"><a href="?fmt_view=mobile">Mobile view</a> | <strong>Desktop view</strong></span>		<div class="content">    <div class="rounded ">    <div class="t png_bg">        <div class="r png_bg"></div>    </div>    <div class="m">    <div class="r">        <div class="wrap-inner ">                	    <ul class="desktop">                                                    				        <li class="noborder"><a href="http://www.biomedcentral.com/about/tandc">Terms and Conditions</a></li><li><a href="http://www.biomedcentral.com/about/privacy">Privacy statement</a></li><li><a href="http://www.biomedcentral.com/presscenter">Press</a></li>	<li><a href="http://www.biomedcentral.com/advertisers">Information for advertisers</a></li><li><a href="http://www.biomedcentral.com/about/bmcjobs">Jobs at BMC</a></li><li><a href="http://www.biomedcentral.com/support">Support</a></li><li><a href="http://www.biomedcentral.com/about/contact">Contact us</a></li>        	    </ul>    <p id="copyright">&copy; 2012 	BioMed Central Ltd unless otherwise stated. Part of Springer Science+Business Media.</p>                    </div>    </div>    </div>    <div class="b png_bg">        <div class="r png_bg"></div>    </div></div>	</div>			</div>    <div id="analytics">		<script type="text/javascript" src="http://www.google-analytics.com/ga.js" ></script>	    <script type="text/javascript">        var pageTracker = _gat._getTracker("UA-72326-1");		_gat._anonymizeIp();        pageTracker._setLocalRemoteServerMode();        pageTracker._initData();        pageTracker._setCustomVar(1,"Design","new",3);        		pageTracker._trackPageview();	</script></div>    <script src="//ajax.googleapis.com/ajax/libs/jquery/1.8.2/jquery.min.js"></script><script type="text/javascript">if (typeof jQuery == 'undefined') { document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.min.js' type='text/javascript'%3E%3C/script%3E")); }document.documentElement.id = "js";var site = {			abbreviation: "/bmcbioinformatics",	            type: "BMC_CORE",        portalId: "9001",                section: "browse",                page: "fulltext",    											homePortal: "biomedcentral.com",		  			siteShowAds: "true",				showAds: "true",				url: "biomedcentral.com/bmcbioinformatics",			portal: false,	        adpage_url: "biomedcentral.com/bmcbioinformatics/article/10.1186/1471/2105/13/79",        id: "1002",	name: "BMC Bioinformatics"	};if(typeof quoteSearchUri == "undefined") { quoteSearchUri=null; }</script>        <script type="text/javascript" src="/javascript/journals/behaviours-0.js"></script>         <script type="text/javascript" src="/javascript/plugins/jquery.mousewheel-0.js"></script> 	        <script type="text/javascript" src="/javascript/journals/browse-0.js"></script>	        <script type="text/javascript" src="/javascript/plugins/flowplayer-3.2.6.min-0.js"></script>		        <script type="text/javascript" src="/javascript/plugins/mobile-0.js"></script>		        <script type="text/javascript" src="/javascript/plugins/collapsible-content-0.js"></script>        <script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.8.23/jquery-ui.min.js"></script>				            <script type="text/javascript" src="/javascript/articles/articles-0.js"></script>            <script type="text/javascript" src="/javascript/plugins/jmol-0.js"></script>            <script type="text/javascript" src="/javascript/plugins/jquery.fancybox-1.3.1.pack-0.js"></script>		        <script type="text/javascript" src="/javascript/plugins/social_sharing-0.js"></script>		<div id="fb-root"></div><!--[if lte IE 6]>            <script type="text/javascript" src="/javascript/plugins/DD_belatedPNG_0.0.8a-0.js"></script>    <script type="text/javascript">        DD_belatedPNG.fix('.png_bg, ul.primary-nav li a, ul.primary-nav li a span, ul.secondary-nav li a, ul.secondary-nav li a span, ul li i, .plus-button, .facebook, .twitter, .fancy-bg, #fancybox-close');    </script>	                <link rel="stylesheet" type="text/css" href="/css/hacks/ie6-0.css"/>    <![endif]--><!-- Claudia Whitcombe wanted this comment here --></body></html>