<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns:mml="http://www.w3.org/1998/Math/MathML" xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><head><script async="" src="bioinformaticsTest_files/cbgapi.loaded_0"></script>
      <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
      <title>Telescoper: de novo assembly of highly repetitive regions </title>
      <meta name="googlebot" content="NOODP">
      <meta name="HW.ad-path" content="/cgi/content/full/28/18/i311">
      <meta content="/bioinfo/28/18/i311.atom" name="HW.identifier">
      <meta name="DC.Format" content="text/html">
      <meta name="DC.Language" content="en">
      <meta content="Telescoper: de novo assembly of highly repetitive regions" name="DC.Title">
      <meta content="10.1093/bioinformatics/bts399" name="DC.Identifier">
      <meta content="2012-09-15" name="DC.Date">
      <meta content="Oxford University Press" name="DC.Publisher">
      <meta content="Ma'ayan Bresler" name="DC.Contributor">
      <meta content="Sara Sheehan" name="DC.Contributor">
      <meta content="Andrew H. Chan" name="DC.Contributor">
      <meta content="Yun S. Song" name="DC.Contributor">
      <meta content="Bioinformatics" name="citation_journal_title">
      <meta content="Bioinformatics" name="citation_journal_abbrev">
      <meta content="1367-4803" name="citation_issn">
      <meta content="1460-2059" name="citation_issn">
      <meta name="citation_author" content="Ma'ayan Bresler">
      <meta name="citation_author_institution" content="1Department of EECS">
      <meta name="citation_author" content="Sara Sheehan">
      <meta name="citation_author_institution" content="1Department of EECS">
      <meta name="citation_author" content="Andrew H. Chan">
      <meta name="citation_author_institution" content="1Department of EECS">
      <meta name="citation_author" content="Yun S. Song">
      <meta name="citation_author_institution" content="1Department of EECS">
      <meta name="citation_author_institution" content="2Department of Statistics, University of California, Berkeley, CA 94720, USA">
      <meta content="Telescoper: de novo assembly of highly repetitive regions" name="citation_title">
      <meta content="09/15/2012" name="citation_date">
      <meta content="28" name="citation_volume">
      <meta content="18" name="citation_issue">
      <meta content="i311" name="citation_firstpage">
      <meta content="i317" name="citation_lastpage">
      <meta content="28/18/i311" name="citation_id">
      <meta content="bioinfo;28/18/i311" name="citation_mjid">
      <meta content="10.1093/bioinformatics/bts399" name="citation_doi">
      <meta content="http://bioinformatics.oxfordjournals.org/content/28/18/i311.abstract" name="citation_abstract_html_url">
      <meta content="http://bioinformatics.oxfordjournals.org/content/28/18/i311.full" name="citation_fulltext_html_url">
      <meta content="http://bioinformatics.oxfordjournals.org/content/28/18/i311.full.pdf" name="citation_pdf_url">
      <meta content="http://bioinformatics.oxfordjournals.org/content/28/18/i311" name="citation_public_url">
      <meta content="22962446" name="citation_pmid">
      <meta name="citation_section" content="Original Paper">
      <meta name="robots" content="noarchive,nofollow">
      <meta name="googlebot" content="noarchive">
      <link href="http://bioinformatics.oxfordjournals.org/content/28/18/i306.short" rel="prev">
      <link href="http://bioinformatics.oxfordjournals.org/content/28/18/i318.short" rel="next">
      <link rel="stylesheet" type="text/css" media="all" href="bioinformaticsTest_files/hw-global.css">
      <link rel="stylesheet" type="text/css" media="print" href="bioinformaticsTest_files/hw-print.css">
      <link rel="stylesheet" type="text/css" media="all" href="bioinformaticsTest_files/hw-local-global.css">
      <link rel="stylesheet" type="text/css" media="all" href="bioinformaticsTest_files/hw-publisher-global.css">
      <link rel="stylesheet" type="text/css" media="all" href="bioinformaticsTest_files/hw-publisher-ac.css">
      <link rel="stylesheet" type="text/css" media="all" href="bioinformaticsTest_files/hw-page-content.css">
      <link rel="stylesheet" type="text/css" media="all" href="bioinformaticsTest_files/jquery.css">
      <link rel="stylesheet" type="text/css" media="all" href="bioinformaticsTest_files/hw-global-colexpand.css">
      <link rel="stylesheet" type="text/css" media="all" href="bioinformaticsTest_files/hw-publisher-page-content.css"><script type="text/javascript" id="session-d76048021e1">var callbackToken='51251E2D7D88F74';</script><script type="text/javascript" src="bioinformaticsTest_files/jquery-min.js"></script><script type="text/javascript" src="bioinformaticsTest_files/hw-shared.js"></script><script type="text/javascript" src="bioinformaticsTest_files/hw-publisher-site.js"></script><script type="text/javascript" src="bioinformaticsTest_files/local-js-vars.js"></script><script type="text/javascript" src="bioinformaticsTest_files/hw-content.js"></script><script type="text/javascript" src="bioinformaticsTest_files/hw-mathjax.js"></script><script type="text/javascript" src="bioinformaticsTest_files/jquery_003.js"></script><script type="text/javascript" src="bioinformaticsTest_files/jquery_002.js"></script><script type="text/javascript" src="bioinformaticsTest_files/jquery.js"></script><script type="text/javascript" src="bioinformaticsTest_files/hw-col-expand.js"></script><script type="text/javascript" src="bioinformaticsTest_files/hw-publisher-article-dynamic-elements.js"></script><script type="text/javascript" src="bioinformaticsTest_files/hw-mathjax.htm"></script><script type="text/x-mathjax-config;executed=true">
            MathJax.Hub.Config({
                tex2jax: {
                  inlineMath: [["$","$"],["\\(","\\)"]],
                  skipTags: ["body"],
                  processClass: "mml-math"
                }
            });
        </script><script type="text/javascript" src="bioinformaticsTest_files/MathJax.js"></script><link rel="stylesheet" type="text/css" media="all" href="bioinformaticsTest_files/hw20.css">
      <link rel="stylesheet" type="text/css" media="all" href="bioinformaticsTest_files/h20-oup-header.css">
      <link rel="stylesheet" type="text/css" media="all" href="bioinformaticsTest_files/h20-oup-footer.css">
      <link rel="stylesheet" type="text/css" media="all" href="bioinformaticsTest_files/h20-oup-sidebars.css">
      <link type="text/css" media="all" rel="stylesheet" href="bioinformaticsTest_files/bioinformatics_h20.css">
      <link rel="stylesheet" type="text/css" href="bioinformaticsTest_files/print.css" media="print"><script type="text/javascript" src="bioinformaticsTest_files/main.js"></script><script type="text/javascript" src="bioinformaticsTest_files/oup_ad_size.js"></script><script src="bioinformaticsTest_files/common"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Hover_Arrow {position: absolute; width: 15px; height: 11px; cursor: pointer}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 2px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 2px 2em; background: transparent}
.MathJax_MenuTitle {background-color: #CCCCCC; margin: -1px -1px 1px -1px; text-align: center; font-style: italic; font-size: 80%; color: #444444; padding: 2px 0; overflow: hidden}
.MathJax_MenuArrow {position: absolute; right: .5em; color: #666666}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuRadioCheck {position: absolute; left: 1em}
.MathJax_MenuLabel {padding: 2px 2em 4px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #CCCCCC; margin: 4px 1px 0px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: Highlight; color: HighlightText}
.MathJax_Menu_Close {position: absolute; width: 31px; height: 31px; top: -15px; left: -15px}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1px; bottom: 2px; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><script src="bioinformaticsTest_files/policy"></script><script src="bioinformaticsTest_files/custom-script.js"></script><script src="bioinformaticsTest_files/oxford-www"></script></head>
   <body class="general_page journal bioinformatics">
            <div class="article fulltext-view" itemprop="articleBody"><span class="highwire-journal-article-marker-start"></span><h1 id="article-title-1" itemprop="headline">Telescoper: <em>de novo</em> assembly of highly repetitive regions
               </h1>
               <div class="contributors">
                  <ol class="contributor-list" id="contrib-group-1">
                     <li class="contributor" id="contrib-1" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="name" itemprop="name"><a class="name-search" href="http://bioinformatics.oxfordjournals.org/search?author1=Ma%27ayan+Bresler&amp;sortspec=date&amp;submit=Submit">Ma'ayan Bresler</a></span><a id="xref-aff-1-1" class="xref-aff" href="#aff-1"><sup>1</sup></a>, 
                     </li>
                     <li class="contributor" id="contrib-2" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="name" itemprop="name"><a class="name-search" href="http://bioinformatics.oxfordjournals.org/search?author1=Sara+Sheehan&amp;sortspec=date&amp;submit=Submit">Sara Sheehan</a></span><a id="xref-aff-1-2" class="xref-aff" href="#aff-1"><sup>1</sup></a>, 
                     </li>
                     <li class="contributor" id="contrib-3" itemprop="author" itemscope="itemscope" itemtype="http://schema.org/Person"><span class="name" itemprop="name"><a class="name-search" href="http://bioinformatics.oxfordjournals.org/search?author1=Andrew+H.+Chan&amp;sortspec=date&amp;submit=Submit">Andrew H. Chan</a></span><a id="xref-aff-1-3" class="xref-aff" href="#aff-1"><sup>1</sup></a> and 
                     </li>
                     <li class="last" id="contrib-4"><span class="name"><a class="name-search" href="http://bioinformatics.oxfordjournals.org/search?author1=Yun+S.+Song&amp;sortspec=date&amp;submit=Submit">Yun S. Song</a></span><a id="xref-aff-1-4" class="xref-aff" href="#aff-1"><sup>1</sup></a><span class="xref-sep">,</span><a id="xref-aff-2-1" class="xref-aff" href="#aff-2"><sup>2</sup></a><span class="xref-sep">,</span><a id="xref-corresp-1-1" class="xref-corresp" href="#corresp-1"><sup>*</sup></a></li>
                  </ol><p class="affiliation-list-reveal"><a href="#" class="view-more">+</a> Author Affiliations</p>
                  <ol class="affiliation-list hideaffil">
                     <li class="aff"><a id="aff-1" name="aff-1"></a><address><sup>1</sup>Department of EECS
                        </address>
                     </li>
                     <li class="aff"><a id="aff-2" name="aff-2"></a><address><sup>2</sup>Department of Statistics, University of California, Berkeley, CA 94720, USA
                        </address>
                     </li>
                  </ol>
                  <ol class="corresp-list">
                     <li class="corresp" id="corresp-1"><a class="rev-xref" href="#xref-corresp-1-1">↵</a>*To whom correspondence should be addressed.
                     </li>
                  </ol>
               </div>
               <div class="section abstract" id="abstract-1" itemprop="description">
                  <div class="section-nav">
                     <div class="nav-placeholder">&nbsp;</div><a href="#sec-1" title="1 INTRODUCTION" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>Abstract</h2>
                  
                  <p id="p-2"><strong>Motivation:</strong> With advances
 in sequencing technology, it has become faster and cheaper to obtain 
short-read data from which to assemble
                     genomes. Although there has been considerable 
progress in the field of genome assembly, producing high-quality <em>de novo</em>
 assemblies from short-reads remains challenging, primarily because of 
the complex repeat structures found in the genomes
                     of most higher organisms. The telomeric regions of 
many genomes are particularly difficult to assemble, though much could
                     be gained from the study of these regions, as their
 evolution has not been fully characterized and they have been linked to
                     aging.
                  </p>
                  
                  <p id="p-3"><strong>Results:</strong> In this article,
 we tackle the problem of assembling highly repetitive regions by 
developing a novel algorithm that iteratively
                     extends long paths through a series of read-overlap
 graphs and evaluates them based on a statistical framework. Our 
algorithm,
                     Telescoper, uses short- and long-insert libraries 
in an integrated way throughout the assembly process. Results on real 
and
                     simulated data demonstrate that our approach can 
effectively resolve much of the complex repeat structures found in the 
telomeres
                     of yeast genomes, especially when longer 
long-insert libraries are used.
                  </p>
                  
                  <p id="p-4"><strong>Availability:</strong> Telescoper is publicly available for download at sourceforge.net/p/telescoper.
                  </p>
                  
                  <p id="p-5"><strong>Contact:</strong> <a href="mailto:yss@eecs.berkeley.edu">yss@eecs.berkeley.edu</a></p>
                  
                  <p id="p-6"><strong>Supplementary Information:</strong> <a href="http://bioinformatics.oxfordjournals.org/cgi/content/full/bts399/DC1">Supplementary data</a> are available at <em>Bioinformatics</em> online.
                  </p>
                  
               </div>
               <div class="section intro" id="sec-1">
                  <div class="section-nav"><a href="#abstract-1" title="Abstract" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-2" title="2 TERMINOLOGY" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>1 INTRODUCTION</h2>
                  
                  <p id="p-7">Recent advances in high-throughput 
sequencing (HTS) technologies have dramatically reduced the cost of 
producing reference
                     genomes for new species and characterizing 
whole-genome variations in multiple individuals of a population. 
However, the assemblies
                     produced by current algorithms are often 
incomplete. <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alkan <em>et al.</em> (2011)</a> report that a <em>de novo</em> shotgun assembly of the human genome using short-reads is 16% shorter than the reference assembled using more laborious means,
                     and that <em>&lt;</em> 1% of segmental duplications
 are represented. Indeed, it is well recognized that there is room for 
better algorithmic use
                     of the data, especially for repetitive regions, 
which are one of the primary challenges in assembly. Telomeres are 
particularly
                     complex and repetitive, and thus very difficult to 
assemble correctly. Not only does each telomere contain repeats within
                     itself, but often telomeres on different 
chromosomes are very similar. Existing assembly algorithms thus 
frequently fail to
                     assemble telomeric regions from short-read data. 
Due to this lack of complete assembly, telomere evolution has not been 
fully
                     characterized, though a great deal is to be gained 
from it, as telomeres have been linked to ageing (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">McEachern <em>et al.</em>, 2000</a>).
 High-quality telomere assemblies could help us learn more about the 
variation in telomeres within and between species.
                     In addition, characterizing telomere gene families 
and their regulation could help us clarify the function of telomeres and
                     how they change as we age.
                  </p>
                  
                  <p id="p-8">Genome assembly is the challenge of 
piecing together reads to reconstruct the original genome. Reads are 
obtained from various
                     technologies, with varying read length, error rates
 and coverage. Sanger-chemistry reads range in length from around 500 to
                     1000 bases. Newer technologies such as Illumina, 
Complete Genomics (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Drmanac <em>et al.</em>, 2010</a>), Helicos (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Harris <em>et al.</em>, 2008</a>), 454 Life Sciences (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Margulies <em>et al.</em>, 2005</a>), SOLiD (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">McKernan <em>et al.</em>, 2009</a>) and Ion Torrent (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Rothberg <em>et al.</em>, 2011</a>)
 provide reads at vastly lower costs for greater throughput, but at the 
expense of length. Initial improvements in assembly
                     from short-read data focused on how to process the 
sheer quantity of data and how to detect overlaps. The de Bruijn graph
                     proved a useful data structure for this purpose (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Pevzner <em>et al.</em>, 2001</a>) and is used by pioneering short-read assemblers such as Velvet (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Zerbino and Birney, 2008</a>) and EULER-USR (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Chaisson <em>et al.</em>, 2009</a>), and subsequent assemblers including SOAPdenovo (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Li <em>et al.</em>, 2010</a>), ALLPATHS 2 (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">MacCallum <em>et al.</em>, 2009</a>), ABySS (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Simpson <em>et al.</em>, 2009</a>) and Cortex (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Iqbal <em>et al.</em>, 2012</a>).
                  </p>
                  
                  <p id="p-9">Many HTS platforms produce paired-end or 
mate-pair reads, which we collectively refer to as read-pairs. The 
paired nature
                     of these reads constitutes a powerful source of 
information, significantly facilitating genome assembly. Improved use of
 read-pair
                     information lies at the heart of recent works such 
asALLPATHS-LG (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Gnerre <em>et al.</em>, 2011</a>), the PE-Assembler (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ariyaratne and Sung, 2011</a>) and the Paired de Bruijn Graph (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Medvedev <em>et al.</em>, 2011</a>), innovations of which are primarily in earlier use of short-insert read-pairs.
                  </p>
                  
                  <p id="p-10">ALLPATHS-LG requires reads of length 
around 100 bases sequenced from short fragments of length ≈ 180 bp so 
that, on average,
                     each read-pair overlaps by ∼20 bases. This means 
that in general each read-pair can be merged into a single longer read 
corresponding
                     to the fragment. A drawback of this approach is in 
the very specific type of data required, which differs from the standard
                     library construction of fragments 300–500 bp in 
size. The PE-Assembler builds short stretches in non-repetitive regions 
first,
                     similar to unitigs (see <a id="xref-sec-2-1" class="xref-sec" href="#sec-2">Section 2</a> for a definition) in a de Bruijn graph, and then extends these iteratively using reads with mates that map to the increasing
                     already-assembled portion. (A similar idea is also used in <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Parrish <em>et al.</em> (2011</a>)
 for resequencing with a reference, where insertions are assembled as 
iterative extensions of existing sequences.) The Paired
                     de Bruijn Graph method entails building a so-called
 A-Bruijn graph in which vertices track pairs of reads instead of single
                     reads, with two vertices being merged only if the 
merging is consistent with the associated pairs of reads. To our 
knowledge,
                     this method remains largely theoretical at this 
time, and it has been tested only on simulated data with perfect reads.
                  </p>
                  
                  <p id="p-11">A theoretical observation from <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Medvedev <em>et al.</em> (2011)</a> is that longer long-insert libraries can substantially improve assembly. Recent innovations (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Peng <em>et al.</em>, 2012</a>) in library construction may bring such libraries into the mainstream, so it is timely to develop algorithms that take full
                     advantage of such data.
                  </p>
                  
                  <p>In this article, we describe a new algorithm to improve <em>de novo</em>
 assembly of highly repetitive regions. Although the ideas presented 
here are applicable to the assembly of any genomic region,
                     our algorithm was developed with the specific aim 
of assembling highly repetitive regions such as telomeres. In our 
method,
                     which we name Telescoper, we incorporate the 
following three algorithmic ideas, the latter two of which make novel 
use of
                     read-pairs:
                     
                  </p>
                  <ol class="list-ord" id="list-1">
                     <li id="list-item-1">
                        <p id="p-13"><strong>Iterative extensions:</strong> a seed sequence is extended iteratively using reads localized to a particular region by their mates, thus allowing for gradual
                           extension into difficult regions. See <a id="xref-sec-4-1" class="xref-sec" href="#sec-4">Section 3.1</a> for details. As mentioned above, this idea is not new, but it has not yet been fully exploited in a well-used algorithm,
                           despite several potential advantages.
                        </p>
                     </li>
                     <li id="list-item-2">
                        <p id="p-14"><strong>Simultaneous use of short-insert read-pairs in a statistical framework:</strong> rather than using read-pair information pair by pair to untangle the read-graph, we build extensions through the graph and
                           <em>simultaneously</em> consider all read-pairs mapping to each extension to choose the most probable extension. See <a id="xref-sec-5-1" class="xref-sec" href="#sec-5">Section 3.2.</a></p>
                     </li>
                     <li id="list-item-3">
                        <p id="p-15"><strong>Simultaneous use of long-insert libraries:</strong>
 rather than using long-insert read-pairs only for scaffolding or for 
filling in gaps between easily assembled contigs, our
                           iterative extension procedure uses 
long-insert reads during assembly. We look for support of assembled 
sequence at all insert
                           sizes, so that incorrect assembly can result 
only if the repetitive structure spans all libraries. See <a id="xref-sec-6-1" class="xref-sec" href="#sec-6">Section 3.3</a> for further details.
                        </p>
                     </li>
                  </ol>
                  
                  <p id="p-16">Each of the above ideas helps to resolve 
repetitive regions. Implicit throughout our algorithm is the principle 
that in order
                     to assemble difficult regions, one cannot make only
 safe simplifications, but must also explore several alternative 
extensions
                     and use downstream analysis to find and reject 
false extensions.
                  </p>
                  
                  <p id="p-17">We tested the performance of our method on both real and simulated data from the telomeres of the <em>Saccharomyces cerevisiae</em>
 genome, which consists of 16 chromosomes. This is a particularly 
challenging problem since all such telomeres have a core
                     repetitive component called X (≈ 475 bp long) as 
well as several combinatorial repeats and sometimes a larger repetitive 
component
                     (see Saccharomyces Genome Database, <a href="http://www.yeastgenome.org/">www.yeastgenome.org</a>). In addition, because <em>S.cerevisiae</em> underwent an ancient genome duplication (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Kellis <em>et al.</em>, 2004</a>),
 telomeric regions of different chromosomes typically share highly 
similar repetitive regions. We show that Telescoper is
                     capable of generating more complete and continuous 
assemblies in the telomeric regions than other state-of-the-art <em>de novo</em> assembly algorithms, especially when longer long-insert libraries are used.
                  </p>
                  
               </div>
               <div class="section" id="sec-2">
                  <div class="section-nav"><a href="#sec-1" title="1 INTRODUCTION" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-3" title="3 METHODS" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>2 TERMINOLOGY</h2>
                  
                  <p>We adopt the following terms commonly used to describe the output of sequencing technologies and the resulting assemblies:
                     
                  </p>
                  <ul class="list-unord" id="list-2">
                     <li id="list-item-4">
                        <p id="p-19"><strong>Read-pair:</strong> refers 
to a pair of sequenced reads from a fragment. The fragment size 
determines the distance between the two reads, often
                           called the ‘insert’ size. The insert 
distribution is frequently approximated by a normal distribution. We use
 the term read-pair
                           regardless of whether the insert is short or 
long.
                        </p>
                     </li>
                     <li id="list-item-5">
                        <p id="p-20"><strong>Mate:</strong> refers to the partner of a read <em>R</em> in a read-pair. When <em>R</em>
 is oriented with respect to a sequence, we know its mate's relative 
position and can refer to it as a ‘left-mate’ or ‘right-mate’
                           (or, as a ‘left-read’ or ‘right-read’).
                        </p>
                     </li>
                     <li id="list-item-6">
                        <p id="p-21"><strong>Contig:</strong> a 
sequence, which ideally belongs to the original genome, produced from 
assembling a group of reads. The standard output
                           from an assembly algorithm is a set of 
contigs. Contigs are often ordered to produce ‘scaffolds’, which may 
contain stretches
                           of unknown sequence between the contigs.
                        </p>
                     </li>
                     <li id="list-item-7">
                        <p id="p-22"><strong>Read-overlap graph:</strong> also called a read-graph, is a graph in which each vertex is a read and directed edges between reads represent overlapping
                           sequence, i.e. in the error-free case, the last <em>k</em> bases of one read are the same as the first <em>k</em> bases of its neighbor read, where <em>k</em> is greater than some threshold.
                        </p>
                     </li>
                     <li id="list-item-8">
                        <p id="p-23"><strong>Unitig:</strong> a path through the read-graph that can be unambiguously merged into a single sequence. A ‘unitig graph’is an extension of
                           the read-overlap graph idea (similarly for a unitig path), where the vertices are now unitigs.
                        </p>
                     </li>
                  </ul>
                  
               </div>
               <div class="section methods" id="sec-3">
                  <div class="section-nav"><a href="#sec-2" title="2 TERMINOLOGY" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-8" title="4 EMPIRICAL RESULTS" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>3 METHODS</h2>
                  
                  <p id="p-24">We have two main aims in our algorithm: 
(i) rather than performing a greedy read-by-read assembly procedure, we 
build a number
                     of alternative extensions, and score them according
 to the alignment of read-pairs to each extension and (ii) we use 
long-insert
                     read-pairs not only for scaffolding or gap filling 
but also as part of the assembly itself, to check that the local 
assembly
                     is consistent on a longer scale.
                  </p>
                  
                  <p id="p-25">Our algorithm begins with a set of 
non-repetitive ‘seed strings’, as could be taken from a reference 
genome, if it exists,
                     or be assembled from a de Bruijn graph. At present,
 we use seeds of length 500 bp from the reference, at position 40 kb 
from
                     the end of the chromosome, although contigs 
produced from any other algorithm could be used. The goal is to then 
independently
                     extend each contig to produce a more complete 
assembly.
                  </p>
                  
                  <p id="p-26">A high-level overview of the algorithm is illustrated in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. The algorithm proceeds by extending the end of the contig iteratively by a fixed amount, <em>N</em><sub>new</sub>, per iteration, as detailed in <a id="xref-sec-4-2" class="xref-sec" href="#sec-4">Section 3.1.</a> We fix the extension length (usually a few hundred bases) as a conservative measure. Because multiple extensions are frequently
                     possible, the result is an ‘extension graph’ (e-graph) in which each extension node (e-node) contains <em>N</em><sub>new</sub>
 bases of new sequence that serve as a possible extension for that 
e-node's parent. A path from the root (the seed string)
                     to a leaf represents a series of extensions that 
form a single lengthened contig. The aim is for the e-graph to contain a
                     path corresponding to the true sequence, ideally 
terminating close to the end of the desired chromosome, and for this 
path
                     to be identifiable as the best.
                     
                     
                  </p>
                  <div id="F1" class="fig pos-float odd">
                     <img src="bioinformaticsSupFiles/F1_large.jpg" width=800/>
                     </div>
                     <div class="fig-caption"><span class="fig-label">Fig. 1.</span> 
                        <p id="p-27" class="first-child">High-level description of the algorithm. Beginning with the seed string <em>S</em><sub>0</sub>,
 the algorithm iteratively performs the steps described to construct an 
e-graph data structure, from which a contig or contigs
                           can be read. For simplicity, only a few 
example arcs are shown; in reality, red arcs are present between each 
consecutive
                           pair of e-nodes, and orange arcs can be 
present between a given e-node and any of its preceding e-nodes
                        </p>
                        <div class="sb-div caption-clear"></div>
                     </div>
                  </div>
                  
                  <p id="p-28">Our algorithm will be most tractable if 
the e-graph is sparse, so at each iteration, there are as few extensions
 as possible
                     (and the true extension is among them). The 
criteria for pruning and terminating the e-graph are discussed in <a id="xref-sec-7-1" class="xref-sec" href="#sec-7">Section 3.4.</a>
 We first explain our methods for (i) listing possible extensions for a 
given e-node in the e-graph, (ii) scoring each extension
                     based on the alignment of short-insert read-pairs 
and (iii) scoring each extension based on the alignment of long-insert 
read-pairs.
                  </p>
                  
                  <p id="p-29">In the following description, we assume without loss of generality that we are extending to the right.</p>
                  
                  <div id="sec-4" class="subsection">
                     <h3>3.1 Iterative extension of assembly</h3>
                     
                     <p id="p-30">The extension step consists of finding
 possible extensions of a given e-node; the extensions will in turn 
become e-nodes themselves.
                        We fix the length of each e-node so that most 
right-reads in the new extension will have left-mates mapping to the 
e-node
                        rather than behind it. In our implementation, we
 choose this length, denoted <em>N</em><sub>tot</sub>, to be the mean insert length plus the standard deviation of the short-insert library. In the case of multiple short-insert
                        libraries, one can use the largest short-insert length for computing <em>N</em><sub>tot</sub>.
                     </p>
                     
                     <p id="p-31">The extension step is depicted in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>.
 It begins by mapping all the left-reads to the e-node to obtain 
right-mates extending off the right end of the e-node into
                        unknown region yet to be assembled, i.e. the 
left-mate maps to the e-node and the right-mate dangles off the end, as 
illustrated
                        in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2a</a>. We say that these right-mates form a read ‘cloud’.
                        
                        
                     </p>
                     <div id="F2" class="fig pos-float odd">
                     <img src="bioinformaticsSupFiles/F2.large.jpg" width="800"/>
                        
                        </div>
                        <div class="fig-caption"><span class="fig-label">Fig. 2.</span> 
                           <p id="p-32" class="first-child">Illustration of Step 1 of <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, finding an e-node <em>S</em>'s possible extensions. (<strong>a</strong>) A read ‘cloud’ consists of those right-reads with left-mates that map to <em>S</em>. (<strong>b</strong>) The reads in the cloud are then error corrected and organized into a read-graph, which is in turn converted into a unitig
                              graph. (<strong>c</strong>) Paths through the unitig graph correspond to possible extensions
                           </p>
                           <div class="sb-div caption-clear"></div>
                        </div>
                     </div>
                     
                     <p id="p-33">The reads in the read cloud are error-corrected, then used to construct a read-overlap graph, which is transformed into a
                        unitig graph as depicted in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2b</a>. More details on error-correction and read-overlap graph construction are provided in the <a href="http://bioinformatics.oxfordjournals.org/cgi/content/full/bts399/DC1">Supplementary Material</a>. The unitig graph encodes a list of candidate extensions for the contig, as illustrated in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2c</a>. Each new e-node consists of <em>N</em><sub>new</sub> bases of new extension plus (<em>N</em><sub>tot</sub> – <em>N</em><sub>new</sub>) bases from the end of the previous e-node.
                     </p>
                     
                     <p id="p-34">There are several advantages to this 
localized assembly. First, it reduces ambiguities caused by repeats. For
 a read-pair
                        from another location to interfere with the area
 under construction, its left-read must map to the previous e-node while
 the
                        right-read must overlap with another read in the
 read cloud. Second, because it restricts assembly to a small region, 
there
                        is ample memory to store complicated information
 about the reads and their relationships. This information can be thrown
 out
                        as we move to other regions of the graph. This 
local use of information enables more complex use of read-pairs, as 
described
                        in <a id="xref-sec-5-2" class="xref-sec" href="#sec-5">Sections 3.2</a> and <a id="xref-sec-6-2" class="xref-sec" href="#sec-6">3.3</a>.
                     </p>
                     
                  </div>
                  
                  <div id="sec-5" class="subsection">
                     <h3>3.2 Simultaneous use of short-insert read-pairs in statistical scoring of extensions</h3>
                     
                     <p id="p-35">Although existing assembly algorithms 
make use of read-pairs in various ways, the information contained in 
read-pairs has
                        not yet been fully exploited. In other 
assemblers, read-pairs are used primarily to connect unitigs with 
expected insert sizes.
                        We can obtain additional power by scoring 
potential extensions according to the features derived from the aligned 
read-pairs.
                     </p>
                     
                     <p id="p-36">We first evaluate extensions based on the likelihood of gaps in short-insert read-pair coverage. Each extension consists of
                        an ordered sequence of unitigs, as in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2c</a>. Each right-read in an assembled unitig will have a left-mate mapping to earlier sequence in the previous e-node. The set
                        of left-mates associated with reads in unitig <em>U</em> is denoted <em>M<sub>U</sub></em> (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Fig. 3a</a>).
                        
                        
                     </p>
                     <div id="F3" class="fig pos-float odd">
                        <img src="bioinformaticsSupFiles/F3.large.jpg" width=800/>
                        </div>
                        <div class="fig-caption"><span class="fig-label">Fig. 3.</span> 
                           <p id="p-37" class="first-child">Computing the expected number of left-reads mapping back from a unitig <em>U</em><sub>2</sub> to the previous e-node <em>S</em>. (<strong>a</strong>) <em>M</em><sub><em>U</em><sub>2</sub></sub> denotes the set of reads mapping from unitig <em>U</em><sub>2</sub> to the previous e-node <em>S</em>. (<strong>b</strong>) For a right-read <em>R</em><sub>r</sub> located at position t in unitig <em>U</em><sub>2</sub>, the probability of its left-mate <em>R<sub>l</sub></em> mapping to <em>S</em> at a distance <em>x</em> behind <em>U</em><sub>2</sub> is <em>h</em>(<em>x</em> + <em>t</em>), where <em>h</em>(·) is the expected insert distribution. (<strong>c</strong>) The expected number of reads at position <em>x</em> behind unitig <em>U</em><sub>2</sub> is given by <em>f<sub>U</sub></em> (<em>x</em>) defined in <a id="xref-disp-formula-1-1" class="xref-disp-formula" href="#disp-formula-1">Equation (1)</a></p>
                           <div class="sb-div caption-clear"></div>
                        </div>
                     </div>
                     
                     <p id="p-38">In our model, we make the simplifying assumption of a uniform coverage distribution. Let <em>x</em> denote the distance from the right end of a left read relative to the start of unitig <em>U</em>, as pictured in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3b</a>. We denote by <em>f<sub>U</sub></em> (<em>x</em>) the expected number of left-reads in <em>M<sub>U</sub></em> spanning position <em>x</em> (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Fig. 3c</a>). We compute <em>f<sub>U</sub></em> (<em>x</em>) by convolving the expected insert distribution <em>h</em>(·) with the uniform distribution over the stretch of <em>U</em> on which right mates can begin:
                        <span class="disp-formula" id="disp-formula-1"><img class="graphic" alt="Formula" src="bioinformaticsTest_files/graphic-4.gif"><span class="disp-formula-label">(1)</span></span>
                        where <em>L</em>(<em>U</em>) is the length of <em>U</em>, <em>ℓ</em> is the read length and <em>λ</em> is the probability of a read arriving at position <em>t</em>; note that <em>λ</em> is equal to <em>C/</em>(2<em>ℓ</em>), where <em>C</em> is the coverage. False unitigs will typically have gaps in the empirical distribution <span class="inline-formula" id="inline-formula-1"><img class="inline-graphic" alt="Graphic" src="bioinformaticsTest_files/inline-graphic-1.gif"></span>, as illustrated in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4b</a>. Let Gap(<em>U</em>) denote the set of such gaps associated with <em>U</em> . For a gap <em>g</em> ∈ Gap(<em>U</em>) of length ≥ <em>ℓ/</em>2, we compute a penalty equal to the number of mates expected in <em>g</em>, obtained by summing <em>f<sub>U</sub></em> (<em>x</em>) over <em>g</em>'s coordinates. The preliminary score for an extension is then the sum of these penalties over all gaps and all unitigs in
                        the extension:
                        <span class="disp-formula" id="disp-formula-2"><img class="graphic" alt="Formula" src="bioinformaticsTest_files/graphic-5.gif"><span class="disp-formula-label">(2)</span></span>
                        To produce a final score <em>P</em><sub>ext</sub> for each possible extension, we add <em>p</em><sub>ext</sub> to a ‘contig gap penalty’, equal to <em>λ</em> times the largest gap size (denoted by <em>g<sub>c</sub></em> in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Fig. 4c</a>) between two adjacent unitigs, i.e. the expected number of reads to fall in that gap. The best extensions (i.e. those with
                        the lowest <em>P</em><sub>ext</sub> scores) are kept, as described in more detail in <a id="xref-sec-7-2" class="xref-sec" href="#sec-7">Section 3.4.</a>
                        
                        
                     </p>
                     <div id="F4" class="fig pos-float odd">
                     <img src="bioinformaticsSupFiles/F4.large.jpg" width="800"/>
                        <div class="fig-inline"><a class="fig-inline-link" href="http://bioinformatics.oxfordjournals.org/content/28/18/i311/F4.expansion.html"><img alt="Fig. 4." src="bioinformaticsTest_files/F4.gif"></a><div class="callout"><span>View larger version:</span><ul class="callout-links">
                                 <li><a class="fig-inline-link" href="http://bioinformatics.oxfordjournals.org/content/28/18/i311/F4.expansion.html">In this page</a></li>
                                 <li><a target="_blank" class="in-nw-vis" href="http://bioinformatics.oxfordjournals.org/content/28/18/i311/F4.expansion.html">In a new window</a></li>
                              </ul>
                              <ul class="fig-services">
                                 <li class="ppt-link"><a href="http://bioinformatics.oxfordjournals.org/powerpoint/28/18/i311/F4">Download as PowerPoint Slide</a></li>
                              </ul>
                           </div>
                        </div>
                        <div class="fig-caption"><span class="fig-label">Fig. 4.</span> 
                           <p id="p-39" class="first-child">Illustration of Step 2 of <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, scoring an e-node's possible extensions using short-insert read-pairs. (<strong>a</strong>) The penalty for unitig <em>U</em><sub>2</sub> is 0 because no gaps of size ≥<em>ℓ/</em>2 exist (where <em>ℓ</em> is the read length). (<strong>b</strong>) The penalty for unitig <em>U</em><sub>3</sub> is <em>&gt;</em> 0 because a gap, denoted <em>g</em>, of size ≥ <em>ℓ/</em>2 exists. (<strong>c</strong>) The size of contig gap <em>g<sub>c</sub></em> is the distance between the reads that define the end and start of two adjacent unitigs
                           </p>
                           <div class="sb-div caption-clear"></div>
                        </div>
                     </div>
                     
                  </div>
                  
                  <div id="sec-6" class="subsection">
                     <h3>3.3 Simultaneous use of long-insert libraries</h3>
                     
                     <p id="p-40">Telescoper uses all libraries 
simultaneously during assembly, rather than using long-insert libraries 
only during scaffolding
                        or gap-filling, as is typical in other assembly 
algorithms. The main idea is that once long paths have been formed in 
the
                        e-graph, any further extension can be evaluated 
on the basis of its agreement with the current e-graph according to each
 library.
                        Having produced and pruned a set of extensions 
using just the short-insert library in Steps 1 and 2 of our algorithm (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Fig. 1</a>),
 the third step aims to confirm that a proposed extension is supported 
by read-pair information from all other libraries
                        simultaneously. For there to be ambiguity in 
extension choice, there must be repeats at lengths corresponding to all 
library
                        sizes.
                     </p>
                     
                     <p id="p-41">To test for long-insert read-pair 
support of a potential extension, we first gather all read-pairs of 
which right reads map
                        to the extension and left-reads map to the 
previous e-nodes in the path up the e-graph. Then, if the right reads 
fully cover
                        the proposed extension, even possibly without 
overlaps, we consider the extension to be fully supported. Partial 
support is
                        computed as a linear function of the fraction of
 the extension that is covered by the right reads. This support measure 
is
                        then multiplied by the short-insert score <em>P</em><sub>ext</sub> to obtain a single final score.
                     </p>
                     
                  </div>
                  
                  <div id="sec-7" class="subsection">
                     <h3>3.4 Choosing extensions for continuation</h3>
                     
                     <p id="p-42">For a given e-node, upon finding all its possible extensions, at most <em>B</em> top scoring (the lower the better) extensions are retained for computational tractability. In our analysis, we use <em>B</em>
 = 4. We create a new e-node for each of these top scoring extensions 
and assign a running score equal to the sum of its extension
                        score and its parent e-node's running score. 
Then, at each depth in the e-node graph, the <em>B</em> top scoring e-nodes are marked for pursuit.
                     </p>
                     
                     <p id="p-43">An e-node is terminated if it cannot be lengthened by the extension operation, if its extension score plus the scores of two
                        previous ancestral extensions exceeds a threshold, or if a specified maximum depth is reached.
                     </p>
                     
                     <p id="p-44">To track the parallel success of 
alternative e-node paths and keep their number in check, we use breadth 
first search to explore
                        the e-graph. If two different sequences of 
e-nodes end with equivalent e-nodes at a particular depth, we allow the 
two e-nodes
                        to merge. This kind of merging of e-nodes 
reduces the computational burden.
                     </p>
                     
                  </div>
                  
               </div>
               <div class="section results" id="sec-8">
                  <div class="section-nav"><a href="#sec-3" title="3 METHODS" class="prev-section-link"><span>Previous Section</span></a><a href="#sec-11" title="5 CONCLUSION" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>4 EMPIRICAL RESULTS</h2>
                  
                  <p id="p-45">In this section, we compare Telescoper's performance with that of other short-read assembly algorithms, including ABySS (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Simpson <em>et al.</em>, 2009</a>), ALLPATHS 2 (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">MacCallum <em>et al.</em>, 2009</a>), SGA (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Simpson and Durbin, 2012</a>), SOAPdenovo (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Li <em>et al.</em>, 2010</a>) and Velvet (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Zerbino and Birney, 2008</a>).
                  </p>
                  
                  <p id="p-46">Because of limited space, we focus on short-read data in the ensuing discussion. However, as detailed in the <a href="http://bioinformatics.oxfordjournals.org/cgi/content/full/bts399/DC1">Supplementary Material</a>,
 we also considered a combination of short-insert short-read data and 
long-insert Sanger read data, and observed that Telescoper
                     compares favourably with other algorithms, 
including Celera (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Myers <em>et al.</em>, 2000</a>), which was designed for Sanger reads.
                  </p>
                  
                  <div id="sec-9" class="subsection methods">
                     <h3>4.1 Data and experiment setup</h3>
                     
                     <p id="p-47">We studied the performance on both simulated and real data from strain S288C of <em>S. cerevisiae</em>. We obtained a reference genome from Saccharomyces Genome Database (<a href="http://www.yeastgenome.org/">www.yeastgenome.org</a>),
 which was created through extensive, systematic sequencing to produce a
 very accurate assembly, including the telomeric
                        regions. As mentioned earlier, because of 
ancient genome duplication and complex yeast telomere structure, the 
telomeres of
                        different chromosomes typically share highly 
similar repetitive regions, which poses challenges to assembly.
                     </p>
                     
                     <p id="p-48">We considered different types of data to test the robustness of the algorithms and to study the effect of insert distributions
                        on performance:
                     </p>
                     
                     <p id="p-49"><strong>Simulated Data D1</strong> consisted of read-pairs with two insert distributions, one short and one long. The read length was 101 bp for both types.
                        The <em>short-insert</em> reads had coverage depth 100× and an insert distribution with mean 400 bp and variance 75 bp. The <em>long-insert</em> reads had coverage depth 20× and an insert distribution with mean 10 kb and variance 1 kb. Simulation details are provided
                        in the <a href="http://bioinformatics.oxfordjournals.org/cgi/content/full/bts399/DC1">Supplementary Material</a>.
                     </p>
                     
                     <p id="p-50"><strong>Simulated Data D2</strong> consisted of two read-pair datasets with the same insert distributions and coverages as D1, but with a reduced read length
                        of 50 bp.
                     </p>
                     
                     <p id="p-51"><strong>Real Data D3</strong> consisted of Illumina read-pairs from a sequencing library preparation using Cre-Lox recombination. The reads, as described
                        in Van <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Nieuwerburgh <em>et al.</em> (2012)</a>, were sorted using DeLoxer into reads categorized as <em>short-insert</em> (0–400 bp fragments, mean 220 bases) or <em>long-insert</em>
 (1–5 kb, mean 2.3 kb). The reads varied in length from 30–100 bp. We 
truncated reads to 50 bases in order to provide algorithms
                        with high-quality, uniform-length reads. We used
 coverage 120× for the short-insert data and 40× for the long-insert 
data.
                        The performance of Telescoper does not degrade 
with higher coverage data.
                     </p>
                     
                     <p id="p-52">We sought to assess assembly for the 40-kb telomeric regions at the ends of each of <em>S. cerevisiae</em>'s
 16 chromosomes. To this end, we simulated data only from this region. 
For the real data, we used the full dataset, but
                        restricted evaluation statistics of the produced
 contigs to those alignable to the 32 telomeres, each of length 40 kb.
                     </p>
                     
                     <p id="p-53">Details of running the various algorithms, including parameter settings and runtimes, can be found in the <a href="http://bioinformatics.oxfordjournals.org/cgi/content/full/bts399/DC1">Supplementary Material</a>.
 To optimize the performance of the other algorithms, insert 
distribution and coverage parameters were provided where appropriate.
                        We did not include SGA for D2 and D3 since it 
was designed for reads of at least 100 bp.
                     </p>
                     
                  </div>
                  
                  <div id="sec-10" class="subsection">
                     <h3>4.2 Assembly performance</h3>
                     
                     <p id="p-54">Several standard metrics exist for 
measuring assembly performance in the absence of a reference genome. 
They include the length
                        of the largest contig, the total length of all 
contigs, and N50 (which is equal to the longest contig length such that 
the
                        sum of the lengths of all longer contigs is half
 the total output assembly). An additional metric is NG50 (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Earl <em>et al.</em>, 2011</a>),
 which is similar to N50 but more comparable across assembly algorithms.
 When the genome length is known, then rather than
                        using each algorithm's estimate of the genome 
size, which can fluctuate widely depending on the threshold at which 
small contigs
                        are output, one can use the true genome size. 
Thus NG50 is defined as the length of the longest contig such that the 
sum of
                        all longer contigs is half the total genome 
size. We considered the above-mentioned metrics in our study.
                     </p>
                     
                     <p id="p-55">To investigate assembly accuracy, we mapped each contig to the reference genome using NUCmer from the MUMMER package (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Delcher <em>et al.</em>, 2002</a>).
 For each contig, we determined to which telomere it maps best according
 to the total number of aligned bases. The number
                        of aligned bases in each contig forms a more 
useful foundation for accuracy-informed continuity statistics than the 
direct
                        number of bases in each contig. Therefore, we 
also computed the aforementioned metrics using these aligned lengths.
                     </p>
                     
                     <p id="p-56">The results of our study for simulated data are summarized in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, while the results for the real data are shown in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. These results are for the 32 telomeric regions, each of length 40 kb. As the tables show, Telescoper exhibited the best
                        performance under most metrics, with notable margins from the second best method. As shown in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>, reducing the read length from 101 to 50 bp while keeping all other parameters the same worsened the performance of most
                        algorithms, with ABySS being the most affected.
                        
                        
                     </p>
                     <div class="table pos-float" id="T1">
                        <div class="table-inline">
                           <div class="table-expansion" id="T1" itemprop="articleBody"><span class="highwire-journal-article-marker-start"></span><div class="table-caption"><span class="table-label">Table 1.</span> 
                  <p id="p-57">Summary of assembly results based on simulated data from 32 telomeric regions each of length 40 kb. ‘%Aligned’ is the ratio
                     of Total Aligned to Total Produced, while ‘%Covered’ is the fraction of the telomeric regions covered by contigs
                  </p>
                  <div class="sb-div caption-clear"></div>
               </div>
               <table frame="hsides" rules="groups" id="table-1">
                  <thead id="thead-1">
                     <tr id="tr-1">
                        <th colspan="11" rowspan="1" id="th-1" class="table-bborder">Results for simulated data D1 (read length = 101 bp)</th>
                     </tr>
                     <tr id="tr-2">
                        <th rowspan="1" colspan="1" id="th-2"></th>
                        <th colspan="4" rowspan="1" id="th-3" class="table-bborder">Produced (kb) </th>
                        <th colspan="4" rowspan="1" id="th-4" class="table-bborder">Aligned (kb) </th>
                        <th rowspan="1" colspan="1" id="th-5"></th>
                        <th rowspan="1" colspan="1" id="th-6"></th>
                     </tr>
                     <tr id="tr-3">
                        <th rowspan="1" colspan="1" id="th-7">Assembler</th>
                        <th rowspan="1" colspan="1" id="th-8">N50</th>
                        <th rowspan="1" colspan="1" id="th-9">NG50</th>
                        <th rowspan="1" colspan="1" id="th-10">Max</th>
                        <th rowspan="1" colspan="1" id="th-11">Total</th>
                        <th rowspan="1" colspan="1" id="th-12">N50</th>
                        <th rowspan="1" colspan="1" id="th-13">NG50</th>
                        <th rowspan="1" colspan="1" id="th-14">Max</th>
                        <th rowspan="1" colspan="1" id="th-15">Total</th>
                        <th rowspan="1" colspan="1" id="th-16">%Aligned</th>
                        <th rowspan="1" colspan="1" id="th-17">%Covered</th>
                     </tr>
                  </thead>
                  <tbody id="tbody-1">
                     <tr id="tr-4">
                        <td id="td-1">Telescoper</td>
                        <td id="td-2">40.0</td>
                        <td id="td-3">40.0</td>
                        <td id="td-4">41.0</td>
                        <td id="td-5">1208</td>
                        <td id="td-6">40.0</td>
                        <td id="td-7">40.0</td>
                        <td id="td-8">40.0</td>
                        <td id="td-9">1172</td>
                        <td id="td-10">97.0</td>
                        <td id="td-11">90.4</td>
                     </tr>
                     <tr id="tr-5">
                        <td id="td-12">ABySS</td>
                        <td id="td-13">31.0</td>
                        <td id="td-14">31.0</td>
                        <td id="td-15">39.0</td>
                        <td id="td-16">1296</td>
                        <td id="td-17">31.8</td>
                        <td id="td-18">31.8</td>
                        <td id="td-19">39.3</td>
                        <td id="td-20">1244</td>
                        <td id="td-21">95.9</td>
                        <td id="td-22">84.7</td>
                     </tr>
                     <tr id="tr-6">
                        <td id="td-23">ALLPATHS2</td>
                        <td id="td-24">35.2</td>
                        <td id="td-25">33.0</td>
                        <td id="td-26">39.0</td>
                        <td id="td-27">1047</td>
                        <td id="td-28">35.2</td>
                        <td id="td-29">33.4</td>
                        <td id="td-30">40.0</td>
                        <td id="td-31">1032</td>
                        <td id="td-32">98.5</td>
                        <td id="td-33">80.6</td>
                     </tr>
                     <tr id="tr-7">
                        <td id="td-34">SOAPdenovo</td>
                        <td id="td-35">25.0</td>
                        <td id="td-36">24.0</td>
                        <td id="td-37">39.0</td>
                        <td id="td-38">1149</td>
                        <td id="td-39">28.6</td>
                        <td id="td-40">24.6</td>
                        <td id="td-41">40.0</td>
                        <td id="td-42">1068</td>
                        <td id="td-43">92.9</td>
                        <td id="td-44">82.3</td>
                     </tr>
                     <tr id="tr-8">
                        <td id="td-45">Velvet</td>
                        <td id="td-46">13.9</td>
                        <td id="td-47">9.0</td>
                        <td id="td-48">31.0</td>
                        <td id="td-49">964</td>
                        <td id="td-50">13.9</td>
                        <td id="td-51">9.5</td>
                        <td id="td-52">31.6</td>
                        <td id="td-53">947</td>
                        <td id="td-54">98.2</td>
                        <td id="td-55">73.7</td>
                     </tr>
                     <tr id="tr-9">
                        <td id="td-56">SGA</td>
                        <td id="td-57">31.2</td>
                        <td id="td-58">27.0</td>
                        <td id="td-59">39.0</td>
                        <td id="td-60">1110</td>
                        <td id="td-61">31.6</td>
                        <td id="td-62">27.2</td>
                        <td id="td-63">40.0</td>
                        <td id="td-64">1075</td>
                        <td id="td-65">96.8</td>
                        <td id="td-66">82.0</td>
                     </tr>
                  </tbody>
               </table>
               <table frame="hsides" rules="groups" id="table-2">
                  <thead id="thead-2">
                     <tr id="tr-10">
                        <th colspan="11" rowspan="1" id="th-18" class="table-bborder">Results for simulated data D2 (read length = 50 bp)</th>
                     </tr>
                     <tr id="tr-11">
                        <th rowspan="1" colspan="1" id="th-19"></th>
                        <th colspan="4" rowspan="1" id="th-20" class="table-bborder">Produced (kb) </th>
                        <th colspan="4" rowspan="1" id="th-21" class="table-bborder">Aligned (kb)</th>
                        <th rowspan="1" colspan="1" id="th-22"></th>
                        <th rowspan="1" colspan="1" id="th-23"></th>
                     </tr>
                     <tr id="tr-12">
                        <th rowspan="1" colspan="1" id="th-24">Assembler</th>
                        <th rowspan="1" colspan="1" id="th-25">N50</th>
                        <th rowspan="1" colspan="1" id="th-26">NG50</th>
                        <th rowspan="1" colspan="1" id="th-27">Max</th>
                        <th rowspan="1" colspan="1" id="th-28">Total</th>
                        <th rowspan="1" colspan="1" id="th-29">N50</th>
                        <th rowspan="1" colspan="1" id="th-30">NG50</th>
                        <th rowspan="1" colspan="1" id="th-31">Max</th>
                        <th rowspan="1" colspan="1" id="th-32">Total</th>
                        <th rowspan="1" colspan="1" id="th-33">%Aligned</th>
                        <th rowspan="1" colspan="1" id="th-34">%Covered</th>
                     </tr>
                  </thead>
                  <tbody id="tbody-2">
                     <tr id="tr-13">
                        <td id="td-67">Telescoper</td>
                        <td id="td-68">39.0</td>
                        <td id="td-69">38.0</td>
                        <td id="td-70">39.0</td>
                        <td id="td-71">1162</td>
                        <td id="td-72">38.8</td>
                        <td id="td-73">38.3</td>
                        <td id="td-74">39.8</td>
                        <td id="td-75">1155</td>
                        <td id="td-76">99.4</td>
                        <td id="td-77">90.3</td>
                     </tr>
                     <tr id="tr-14">
                        <td id="td-78">ABySS</td>
                        <td id="td-79">12.1</td>
                        <td id="td-80">8.0</td>
                        <td id="td-81">31.0</td>
                        <td id="td-82">1097</td>
                        <td id="td-83">13.7</td>
                        <td id="td-84">8.9</td>
                        <td id="td-85">31.6</td>
                        <td id="td-86">966</td>
                        <td id="td-87">88.0</td>
                        <td id="td-88">75.0</td>
                     </tr>
                     <tr id="tr-15">
                        <td id="td-89">ALLPATHS2</td>
                        <td id="td-90">32.0</td>
                        <td id="td-91">27.0</td>
                        <td id="td-92">39.0</td>
                        <td id="td-93">968</td>
                        <td id="td-94">32.8</td>
                        <td id="td-95">27.7</td>
                        <td id="td-96">40.0</td>
                        <td id="td-97">950</td>
                        <td id="td-98">98.2</td>
                        <td id="td-99">74.3</td>
                     </tr>
                     <tr id="tr-16">
                        <td id="td-100">SOAPdenovo</td>
                        <td id="td-101">25.0</td>
                        <td id="td-102">21.0</td>
                        <td id="td-103">39.0</td>
                        <td id="td-104">988</td>
                        <td id="td-105">24.6</td>
                        <td id="td-106">20.8</td>
                        <td id="td-107">40.0</td>
                        <td id="td-108">954</td>
                        <td id="td-109">96.5</td>
                        <td id="td-110">74.3</td>
                     </tr>
                     <tr id="tr-17">
                        <td id="td-111">Velvet</td>
                        <td id="td-112">14.0</td>
                        <td id="td-113">9.0</td>
                        <td id="td-114">31.0</td>
                        <td id="td-115">955</td>
                        <td id="td-116">14.2</td>
                        <td id="td-117">9.5</td>
                        <td id="td-118">31.9</td>
                        <td id="td-119">939</td>
                        <td id="td-120">98.3</td>
                        <td id="td-121">73.2</td>
                     </tr>
                  </tbody>
               </table>
               <div class="table-foot"></div><span class="highwire-journal-article-marker-end"></span></div>
                   </div>
                        <div class="table-caption"><span class="table-label">Table 1.</span> 
                           <p id="p-57" class="first-child">Summary of 
assembly results based on simulated data from 32 telomeric regions each 
of length 40 kb. ‘%Aligned’ is the ratio
                              of Total Aligned to Total Produced, while 
‘%Covered’ is the fraction of the telomeric regions covered by contigs
                           </p>
                           <div class="sb-div caption-clear"></div>
                        </div>
                     </div>
                     <div class="table pos-float" id="T2">
                        <div class="table-inline">
                        <div class="table-expansion" id="T2" itemprop="articleBody"><span class="highwire-journal-article-marker-start"></span><div class="table-caption"><span class="table-label">Table 2.</span> 
                  <p id="p-58">Summary of results for real data D3. The contigs produced by each algorithm were aligned to the 32 telomeric regions each
                     of length 40 kb. As before, ‘%Covered’ is the fraction of the telomeric regions covered by contigs
                  </p>
                  <div class="sb-div caption-clear"></div>
               </div>
               <table frame="hsides" rules="groups" id="table-3">
                  <thead id="thead-3">
                     <tr id="tr-18">
                        <th rowspan="1" colspan="1" id="th-35"></th>
                        <th colspan="4" rowspan="1" id="th-36" class="table-bborder">Aligned (kb)</th>
                        <th rowspan="1" colspan="1" id="th-37"></th>
                     </tr>
                     <tr id="tr-19">
                        <th rowspan="1" colspan="1" id="th-38">Assembler</th>
                        <th rowspan="1" colspan="1" id="th-39">N50</th>
                        <th rowspan="1" colspan="1" id="th-40">NG50</th>
                        <th rowspan="1" colspan="1" id="th-41">Max</th>
                        <th rowspan="1" colspan="1" id="th-42">Total</th>
                        <th rowspan="1" colspan="1" id="th-43">%Covered</th>
                     </tr>
                  </thead>
                  <tbody id="tbody-3">
                     <tr id="tr-20">
                        <td id="td-122">Telescoper</td>
                        <td id="td-123">34.5</td>
                        <td id="td-124">32.8</td>
                        <td id="td-125">39.2</td>
                        <td id="td-126">980</td>
                        <td id="td-127">75.8</td>
                     </tr>
                     <tr id="tr-21">
                        <td id="td-128">ABySS</td>
                        <td id="td-129">12.0</td>
                        <td id="td-130">8.3</td>
                        <td id="td-131">31.3</td>
                        <td id="td-132">971</td>
                        <td id="td-133">75.3</td>
                     </tr>
                     <tr id="tr-22">
                        <td id="td-134">ALLPATHS2</td>
                        <td id="td-135">26.3</td>
                        <td id="td-136">16.5</td>
                        <td id="td-137">40.0</td>
                        <td id="td-138">923</td>
                        <td id="td-139">70.1</td>
                     </tr>
                     <tr id="tr-23">
                        <td id="td-140">SOAPdenovo</td>
                        <td id="td-141">21.4</td>
                        <td id="td-142">16.2</td>
                        <td id="td-143">39.3</td>
                        <td id="td-144">879</td>
                        <td id="td-145">68.6</td>
                     </tr>
                     <tr id="tr-24">
                        <td id="td-146">Velvet</td>
                        <td id="td-147">11.8</td>
                        <td id="td-148">6.9</td>
                        <td id="td-149">31.3</td>
                        <td id="td-150">928</td>
                        <td id="td-151">72.2</td>
                     </tr>
                  </tbody>
               </table>
               <div class="table-foot"></div><span class="highwire-journal-article-marker-end"></span></div>
                           
                        </div>
                        <div class="table-caption"><span class="table-label">Table 2.</span> 
                           <p id="p-58" class="first-child">Summary of results for real data D3. The contigs produced by each algorithm were aligned to the 32 telomeric regions each
                              of length 40 kb. As before, ‘%Covered’ is the fraction of the telomeric regions covered by contigs
                           </p>
                           <div class="sb-div caption-clear"></div>
                        </div>
                     </div>
                     
                     <p id="p-59"><a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> provides a more detailed picture of contig length distribution. These plots show the cumulative proportion for all aligned
                        contigs exceeding the contig size indicated on the <em>x</em>-axis. NG50 can be read from the plots as the <em>x</em>-coordinates at which each curve hits the 50% mark of bases output relative to the reference. The best possible curve is the
                        constant function <em>y</em> = 1, so the closer a curve is to that line, the better the performance. Note that for any given minimum contig size (the
                        <em>x</em>-axis value), Telescoper produced more alignable bases than all other methods compared, for all three datasets. Furthermore,
                        <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figures 5a and b</a> illustrate that Telescoper is more robust to a decrease in read length than are the other algorithms.
                        
                        
                     </p>
                     <div id="F5" class="fig pos-float odd">
                        <img src="bioinformaticsSupFiles/F5.large.jpg" width=800/>
                        </div>
                        <div class="fig-caption"><span class="fig-label">Fig. 5.</span> 
                           <p id="p-60" class="first-child">The cumulative proportion of all aligned contigs exceeding the contig size indicated on the <em>x</em>-axis. These plots illustrate the continuity and completeness of different assemblies. For any given minimum contig length,
                              Telescoper produced more aligned bases. NG50 can be read from this graph as the <em>x</em>-coordinates at which each curve hits the 50% mark of bases output relative to the reference. (<strong>a</strong>) Results on simulated data D1. (<strong>b</strong>) Results on simulated data D2. (<strong>c</strong>) Results on real data D3
                           </p>
                           <div class="sb-div caption-clear"></div>
                        </div>
                     </div>
                     
                     <p id="p-61">For Telescoper, the observed difference between the corresponding curve in <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5b</a> and that in <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5c</a>
 is largely attributable to the difference in the insert-size 
distribution. On simulated 50 bp data with long inserts with
                        mean length 2.2 kb and short inserts with mean 
length 400 bp, the performance of Telescoper was similar to that shown 
in <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5c</a> (see <a href="http://bioinformatics.oxfordjournals.org/cgi/content/full/bts399/DC1">Supplementary Material</a>), suggesting that Telescoper is robust the complications of real data and that the observed good performance of Telescoper
                        in <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5b</a> is due to its improved ability to take advantage of a longer (10 kb instead of 2.2 kb) long-insert distribution.
                     </p>
                     
                     <p id="p-62">Of further importance is the extent to
 which an algorithm produces false bases or contigs. Because we forced 
each contig to
                        align to a single telomere, chimeric contigs 
created by joining portions of different telomeres were penalized as 
having bases
                        that do not align. As shown in the ‘% Aligned’ 
column of <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, Telescoper was the top performer in this regard for D2, and followed ALLPATHS 2 and Velvet closely for D1.
                     </p>
                     
                     <p id="p-63">Finally, we considered visually examining the alignments of contigs onto each telomeric region. <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>
 shows the results for two chromosomes, with contigs from each assembly 
algorithm aligned to them. For each algorithm, each
                        contig is represented by a different colour, so 
more colours per method indicates a larger number of contigs. For each 
telomeric
                        region shown, Telescoper produced a single 
contig for almost the entire region, while other algorithms often 
produced many
                        small contigs.
                        
                        
                     </p>
                     <div id="F6" class="fig pos-float odd">
                        <img src="bioinformaticsSupFiles/F6.large.jpg" width=800/>
                        </div>
                        <div class="fig-caption"><span class="fig-label">Fig. 6.</span> 
                           <p id="p-64" class="first-child">Contig 
continuity results for real data D3. The left and right telomeric 
regions (separated by the dotted line) for two different
                              chromosomes are shown, with the aligned 
contigs displayed for each assembly algorithm. Different colours 
represent different
                              contigs in the produced assembly, so more 
colours per method indicates a larger number of contigs. For each 
telomeric region
                              shown, Telescoper produced a single contig
 for almost the entire region, while other algorithms often produced 
many small
                              contigs
                           </p>
                           <div class="sb-div caption-clear"></div>
                        </div>
                     </div>
                     
                  </div>
                  
               </div>
               <div class="section conclusion" id="sec-11">
                  <div class="section-nav"><a href="#sec-8" title="4 EMPIRICAL RESULTS" class="prev-section-link"><span>Previous Section</span></a><a href="#ack-1" title="ACKNOWLEDGMENTS" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>5 CONCLUSION</h2>
                  
                  <p id="p-65">We have introduced several new ideas for <em>de novo</em>
 genome assembly, geared towards highly repetitive regions. Our 
preliminary assembler, Telescoper, proceeds by iteratively
                     extending paths and selecting between them using 
the empirical distributions formed by both long-insert and short-insert 
paired-end
                     reads.
                  </p>
                  
                  <p id="p-66">The utility of Telescoper was validated in a study on both real and simulated data from the 40 kb telomeric regions of each
                     chromosome of <em>S. cerevisiae</em>. For all three
 datasets tested, Telescoper produced more continuous assemblies than 
the other algorithms considered. In our
                     evaluations, we tried to include the strongest and 
most popular algorithms with available implementation. Unfortunately, 
ALLPATHS-LG
                     (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Gnerre <em>et al.</em>, 2011</a>) could not be included, because of its small-fragment library requirement mentioned in <a id="xref-sec-1-1" class="xref-sec" href="#sec-1">Section 1.</a> We considered several standard metrics for comparing assemblies, but we note that the task of comparing genome assemblies
                     is a large one, with several papers exclusively devoted to it (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Earl <em>et al.</em>, 2011</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Salzberg <em>et al.</em>, 2012</a>).
                  </p>
                  
                  <p id="p-67">Other researchers are currently working 
on algorithms for identifying assembly errors using features derived 
from read mapping.
                     Rather than having this be a downstream process, we
 believe that it would help to incorporate such features directly into
                     an assembly algorithm. Here, we make an effort in 
this direction by scoring assembly extensions according to read-mapping
                     statistics. Although the scoring scheme used in 
this article may not be optimal, we have demonstrated that the idea of 
simultaneously
                     pursuing multiple extensions, and concurrently 
using multiple libraries to score and select among them is promising.
                  </p>
                  
                  <p id="p-68">The current implementation of Telescoper can be used as a finishing algorithm to extend contigs into repetitive regions and
                     produce better assemblies for telomeres. Other applications include targeted <em>de novo</em>
 assembly of structural variants and highly variant regions such as 
human leukocyte antigen. Future work will include extending
                     the ideas presented here to whole-genome assembly, 
improving error correction, producing more exhaustive listings of 
potential
                     paths and more thorough evaluation of the alternate
 paths. Also, additional validation metrics such as those explored by <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Salzberg <em>et al.</em> (2012)</a> can be incorporated as well.
                  </p>
                  
                  <p id="p-69">We often see cases where, if we took the 
union of all assemblies, we could produce a much better final product. 
This suggests
                     that assembly is not a solved problem, and that the
 strengths of different algorithms can potentially be combined to 
produce
                     better assemblies. We believe the ideas behind 
Telescoper have the potential to improve <em>de novo</em> assembly significantly and provide a comprehensive picture of previously unresolved repetitive regions.
                  </p>
                  
               </div>
               <div class="section ack" id="ack-1">
                  <div class="section-nav"><a href="#sec-11" title="5 CONCLUSION" class="prev-section-link"><span>Previous Section</span></a><a href="#ref-list-1" title="REFERENCES" class="next-section-link"><span>Next Section</span></a></div>
                  <h2>ACKNOWLEDGMENTS</h2>
                  
                  <p id="p-70">We thank Jasper Rine, Oliver Zill and Devin Scannell for their motivation of the problem and for useful discussions. We also
                     thank Paul Jenkins and Jeremy Maitin-Shepard for helpful discussions.
                  </p>
                  
                  <p id="p-71"><em>Funding:</em> In part by NSF Graduate Research Fellowships (to M.B. and S.S.), an NDSEG Graduate Fellowship (to A.H.C.), and an NSF CAREER
                     Grant DBI-0846015 (to Y.S.S.).
                  </p>
                  
                  <p id="p-72"><em>Conflict of Interest:</em> none declared.
                  </p>
                  
               </div>
               <ul class="copyright-statement"><a class="terms" href="http://www.oxfordjournals.org/our_journals/bioinformatics/terms.html">
                  <li class="fn" id="copyright-statement-1">© The Author(s) (2012). Published by Oxford University Press.</li>
               </a></ul>
               <a class="terms" href="http://www.oxfordjournals.org/our_journals/bioinformatics/terms.html"><div class="license" id="license-1">
                  <p id="p-1">This is an Open Access article distributed
 under the terms of the Creative Commons Attribution License 
(http://creativecommons.org/licenses/by/3.0),
                     which permits unrestricted use, distribution, and 
reproduction in any medium, provided the original work is properly 
cited.
                  </p>
               </div></a>
               <div class="section ref-list" id="ref-list-1">
                  <div class="section-nav"><a href="#ack-1" title="ACKNOWLEDGMENTS" class="prev-section-link"><span>Previous Section</span></a><div class="nav-placeholder">&nbsp;</div>
                  </div>
                  <h2>REFERENCES</h2>
                  <ol class="cit-list ref-use-labels">
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.1">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Alkan</span> <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">Limitations of next-generation genome sequence assembly</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-pub-date">2011</span>;<span class="cit-vol">8</span>:<span class="cit-fpage">61</span>-<span class="cit-lpage">65</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=10.1038/nmeth.1527&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=21102452&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=000285712000023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.2">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Ariyaratne</span> <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sung</span> <span class="cit-name-given-names">W. K.</span></span></li></ol><cite> <span class="cit-article-title">PE-assembler: de novo assembly using short paired-end reads</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-pub-date">2011</span>;<span class="cit-vol">27</span>:<span class="cit-fpage">167</span>-<span class="cit-lpage">174</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/cgi/ijlink?linkType=ABST&amp;journalCode=bioinfo&amp;resid=27/2/167" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.3">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Chaisson</span> <span class="cit-name-given-names">M. J. P.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">De novo fragment assembly with short mate-paired reads: does the read length matter?</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-pub-date">2009</span>;<span class="cit-vol">19</span>:<span class="cit-fpage">336</span>-<span class="cit-lpage">346</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=19/2/336" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.4">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Delcher</span> <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">Fast algorithms for large-scale genome alignment and comparision</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-pub-date">2002</span>;<span class="cit-vol">30</span>:<span class="cit-fpage">2478</span>-<span class="cit-lpage">2483</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/cgi/ijlink?linkType=ABST&amp;journalCode=nar&amp;resid=30/11/2478" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.5">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Drmanac</span> <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">Human genome sequencing using unchained base reads on self-assembling DNA nanoarrays</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-pub-date">2010</span>;<span class="cit-vol">327</span>:<span class="cit-fpage">78</span>-<span class="cit-lpage">81</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=327/5961/78" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.6">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Earl</span> <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">Assemblathon 1: a competitive assessment of de novo short-read assembly methods</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-pub-date">2011</span>;<span class="cit-vol">21</span>:<span class="cit-fpage">2224</span>-<span class="cit-lpage">2241</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=21/12/2224" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.7">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Gnerre</span> <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">High-quality draft assemblies of mammalian genomes from massively parallel sequence data</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci USA</abbr> <span class="cit-pub-date">2011</span>;<span class="cit-vol">108</span>:<span class="cit-fpage">1513</span>-<span class="cit-lpage">1518</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=108/4/1513" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.8">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Harris</span> <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">Single-molecule DNA sequencing of a viral genome</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-pub-date">2008</span>;<span class="cit-vol">320</span>:<span class="cit-fpage">106</span>-<span class="cit-lpage">109</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=320/5872/106" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.9">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Iqbal</span> <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title"><em>De novo</em> assembly and genotyping of variants using colored de Bruijn graphs</span>. <abbr class="cit-jnl-abbrev">Nature Genet.</abbr> <span class="cit-pub-date">2012</span>;<span class="cit-vol">44</span>:<span class="cit-fpage">226</span>-<span class="cit-lpage">232</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=10.1038/ng.1028&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=22231483&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.10">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Kellis</span> <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">Proof and evolutionary analysis of ancient genome duplication in the yeast <em>Saccharomyces cerevisiae</em></span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2004</span>;<span class="cit-vol">428</span>:<span class="cit-fpage">617</span>-<span class="cit-lpage">624</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=10.1038/nature02424&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=15004568&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.11">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Li</span> <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">De novo assembly of human genomes with massively parallel short-read sequencing</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-pub-date">2010</span>;<span class="cit-vol">20</span>:<span class="cit-fpage">265</span>-<span class="cit-lpage">272</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=20/2/265" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.12">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">MacCallum</span> <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">ALLPATHS 2: small genomes assembled accurately and with high continuity from short paired reads</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-pub-date">2009</span>;<span class="cit-vol">10</span>:<span class="cit-fpage">R103</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=10.1186/gb-2009-10-10-r103&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=19796385&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.13">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Margulies</span> <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">Genome sequencing in microfabricated high-density picolitre reactors</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2005</span>;<span class="cit-vol">437</span>:<span class="cit-fpage">376</span>-<span class="cit-lpage">380</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=16056220&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.14">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">McEachern</span> <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">Telomeres and their control</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-pub-date">2000</span>;<span class="cit-vol">34</span>:<span class="cit-fpage">331</span>-<span class="cit-lpage">358</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=10.1146/annurev.genet.34.1.331&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=11092831&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=000166299400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.15">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">McKernan</span> <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">Sequence and structural variation in a human genome uncovered by short-read, massively parallel ligation sequencing using
                                    two-base encoding</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-pub-date">2009</span>;<span class="cit-vol">19</span>:<span class="cit-fpage">1527</span>-<span class="cit-lpage">1541</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=19/9/1527" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.16">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Medvedev</span> <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">Paired de Bruijn graphs: a novel approach for incorporating mate pair information into genome assemblers</span>. <abbr class="cit-jnl-abbrev">Proceedings of Research in Computational Molecular Biology</abbr> <span class="cit-pub-date">2011</span>;<span class="cit-vol">Vol. 6577</span>:<span class="cit-fpage">238</span>-<span class="cit-lpage">251</span>. <span class="cit-comment">Lecture Notes in Computer Science</span>.</cite></div>
                           <div class="cit-extra"><a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Paired%20de%20Bruijn%20graphs%3A%20a%20novel%20approach%20for%20incorporating%20mate%20pair%20information%20into%20genome%20assemblers&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Medvedev&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
                                 						Search Google Scholar
                                 					</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.17">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Myers</span> <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">A whole-genome assembly of <em>Drosophila</em></span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-pub-date">2000</span>;<span class="cit-vol">287</span>:<span class="cit-fpage">2196</span>-<span class="cit-lpage">2204</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/cgi/ijlink?linkType=ABST&amp;journalCode=sci&amp;resid=287/5461/2196" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.18">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Parrish</span> <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">Assembly of non-unique insertion content using next-generation sequencing</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-pub-date">2011</span>;<span class="cit-vol">12</span>(<span class="cit-issue">Suppl. 6</span>):<span class="cit-fpage">S3</span>.</cite></div>
                           <div class="cit-extra"><a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Assembly%20of%20non-unique%20insertion%20content%20using%20next-generation%20sequencing&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Parrish&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
                                 						Search Google Scholar
                                 					</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.19">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Peng</span> <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">Generation of long insert pairs using a Cre-LoxP inverse PCR approach</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-pub-date">2012</span>;<span class="cit-vol">7</span>:<span class="cit-fpage">e29437</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=10.1371/journal.pone.0029437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=22253722&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.20">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Pevzner</span> <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">An Eulerian path approach to DNA fragment assembly</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci USA</abbr> <span class="cit-pub-date">2001</span>;<span class="cit-vol">98</span>:<span class="cit-fpage">9748</span>-<span class="cit-lpage">9753</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/cgi/ijlink?linkType=ABST&amp;journalCode=pnas&amp;resid=98/17/9748" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.21">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Rothberg</span> <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">An integrated semiconductor device enabling non-optical genome sequencing</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-pub-date">2011</span>;<span class="cit-vol">475</span>:<span class="cit-fpage">348</span>-<span class="cit-lpage">352</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=10.1038/nature10242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">CrossRef</a><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=21776081&amp;link_type=MED" class="cit-ref-sprinkles cit-ref-sprinkles-medline">Medline</a><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=000292911200036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-webofscience">Web of Science</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.22">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span> <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">GAGE: a critical evaluation of genome assemblies and assembly algorithms</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-pub-date">2012</span>;<span class="cit-vol">22</span>:<span class="cit-fpage">557</span>-<span class="cit-lpage">567</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=22/3/557" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.23">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Simpson</span> <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">ABySS: a parallel assembler for short-read sequence data</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-pub-date">2009</span>;<span class="cit-vol">19</span>:<span class="cit-fpage">1117</span>-<span class="cit-lpage">1123</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=19/6/1117" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.24">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Simpson</span> <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span> <span class="cit-name-given-names">R.</span></span></li></ol><cite> <span class="cit-article-title">Efficient de novo assembly of large genomes using compressed data structures</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-pub-date">2012</span>;<span class="cit-vol">3</span>:<span class="cit-fpage">549</span>-<span class="cit-lpage">556</span>.</cite></div>
                           <div class="cit-extra"><a target="_blank" href="http://scholar.google.com/scholar?as_q=&amp;as_epq=Efficient%20de%20novo%20assembly%20of%20large%20genomes%20using%20compressed%20data%20structures&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=Simpson&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;btnG=&amp;hl=en&amp;sciui=1&amp;as_sdt=0%2C5">
                                 						Search Google Scholar
                                 					</a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.25">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Van Nieuwerburgh</span> <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al</span></li></ol><cite>. <span class="cit-article-title">Illumina mate-paired dna sequencing-library preparation using Cre-Lox recombination</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-pub-date">2012</span>;<span class="cit-vol">40</span>:<span class="cit-fpage">e24</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/cgi/ijlink?linkType=ABST&amp;journalCode=nar&amp;resid=40/3/e24" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                     <li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-28.18.i311.26">
                           <div class="cit-metadata">
                              <ol class="cit-auth-list"> <li><span class="cit-auth"><span class="cit-name-surname">Zerbino</span> <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birney</span> <span class="cit-name-given-names">E.</span></span></li></ol><cite> <span class="cit-article-title">Velvet: algorithms for de novo short-read assembly using de Bruijn graphs</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-pub-date">2008</span>;<span class="cit-vol">18</span>:<span class="cit-fpage">821</span>-<span class="cit-lpage">829</span>.</cite></div>
                           <div class="cit-extra"><a href="http://bioinformatics.oxfordjournals.org/cgi/ijlink?linkType=ABST&amp;journalCode=genome&amp;resid=18/5/821" class="cit-ref-sprinkles cit-ref-sprinkles-ijlinks"><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></a></div>
                        </div>
                     </li>
                  </ol>
               </div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div>
         <div style="height: 15164px;" id="col-2">
            
            <div class="article-nav sidebar-nav">
               <a href="http://bioinformatics.oxfordjournals.org/content/28/18/i306.short" title="Previous article" class="previous">« Previous</a><span class="article-nav-sep"> | </span><a href="http://bioinformatics.oxfordjournals.org/content/28/18/i318.short" title="Next article" class="next">Next Article »</a>
               
               <span class="toc-link">
                  				<a href="http://bioinformatics.oxfordjournals.org/content/28/18.toc" title="Table of Contents">Table of Contents</a>
                  			</span>
               
               
            </div>
            <div class="content-box" id="article-cb-main">
               <div class="cb-contents">
                  <h3 class="cb-contents-header"><span>This Article</span></h3>
                  <div class="cb-section cb-slug">
                     <ol>
                        <li>
                           <div id="slugline">
                              
                              <cite>
                                 <abbr title="Bioinformatics" class="slug-jnl-abbrev">
                                    <nlm:abbrev-journal-title xmlns:nlm="http://schema.highwire.org/NLM/Journal" abbrev-type="publisher">Bioinformatics</nlm:abbrev-journal-title></abbr><span class="slug-pub-date" itemprop="datePublished">
                                    (2012) 
                                    </span>
                                 
                                 <span class="slug-vol">
                                    28
                                    </span><span class="slug-issue">
                                    (18):
                                    </span><span class="slug-pages">
                                    i311-i317.
                                    </span>
                                 <span class="slug-doi-wrapper">
                                    
                                    doi:
                                    
                                    <span title="10.1093/bioinformatics/bts399" class="slug-doi">10.1093/bioinformatics/bts399</span>
                                    </span>
                                 </cite>
                              
                              
                              
                              
                              
                              
                              
                              
                              
                              <span class="issue-title">This article appears in: <a href="http://bioinformatics.oxfordjournals.org/content/28/18.toc" title="Table of Contents">ECCB 2012 PROCEEDINGS PAPERS COMMITTEE SEPTEMBER 9 TO SEPTEMBER 12, 2012, CONFERENCE CENTER BASEL, SWITZERLAND</a></span>
                              
                              
                           </div>
                        </li>
                     </ol>
                  </div>
                  <div class="cb-section cb-views">
                     <ol>
                        <li><span class="creative-commons-article">This article is Open Access<span>OA</span></span></li>
                        <li class="abstract-view-link primary"><a href="http://bioinformatics.oxfordjournals.org/content/28/18/i311.abstract" rel="view-abstract">Abstract</a><span class="free">Free</span></li>
                        <li class="notice full-text-view-link"><span class="variant-indicator">» <span>Full Text (HTML)</span></span><span class="free">Free</span></li>
                        <li class="notice full-text-pdf-view-link"><a href="http://bioinformatics.oxfordjournals.org/content/28/18/i311.full.pdf+html" rel="view-full-text.pdf">Full Text (PDF)</a><span class="free">Free</span></li>
                     </ol>
                  </div>
                  <div class="cb-section collapsible" id="cb-art-cat">
                     <h4 class="cb-section-header"><a href="#" class="collapse-toggle"><span class="view-more">-</span> <span>Classifications</span></a></h4>
                     <ol>
                        <li>
                           <ul class="subject-headings">
                              <li>Original Paper
                                 <ul class="subject-sub-headings last-child">
                                    <li><a class="tocsection-search" href="http://bioinformatics.oxfordjournals.org/search?tocsectionid=SEQUENCING+AND+SEQUENCE+ANALYSIS&amp;sortspec=date&amp;submit=Submit">SEQUENCING AND SEQUENCE ANALYSIS</a></li>
                                 </ul>
                              </li>
                           </ul>
                        </li>
                     </ol>
                  </div>
                  <div class="cb-section collapsible" id="cb-art-svcs">
                     <h4 class="cb-section-header"><a href="#" class="collapse-toggle"><span class="view-more">-</span> <span>Services</span></a></h4>
                     <ol>
                        <li class="alert-link icon-link"><a href="http://bioinformatics.oxfordjournals.org/cgi/alerts/ctalert?alertType=citedby&amp;addAlert=cited_by&amp;cited_by_criteria_resid=bioinfo;28/18/i311&amp;saveAlert=no&amp;return-type=article&amp;return_url=http://bioinformatics.oxfordjournals.org/content/28/18/i311.full">Alert me when cited</a></li>
                        <li class="alert-link icon-link"><a href="http://bioinformatics.oxfordjournals.org/cgi/alerts/ctalert?alertType=correction&amp;addAlert=correction&amp;correction_criteria_value=28/18/i311&amp;saveAlert=no&amp;return-type=article&amp;return_url=http://bioinformatics.oxfordjournals.org/content/28/18/i311.full">Alert me if corrected</a></li>
                        <li><a href="http://bioinformatics.oxfordjournals.org/cgi/alerts/ctalert?alertType=eletter&amp;addAlert=eletter&amp;eletter_criteria_value=28/18/i311&amp;saveAlert=no&amp;return-type=article&amp;return_url=http://bioinformatics.oxfordjournals.org/content/28/18/i311.full">Alert me if commented</a></li>
                        <li class="similar-link"><a href="http://bioinformatics.oxfordjournals.org/search?qbe=bioinfo;bts399&amp;citation=Bresler%20et%20al.%2028%20%2818%29:%20i311&amp;submit=yes">Find similar articles</a></li>
                        <li class="similar-link nodata"><div id="cb-loaded-isi-related-none">No Web of Science related articles</div></li>
                        <li class="similar-link"><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=22962446&amp;link_type=MED_NBRS" class="similar-link">Similar articles in PubMed</a></li>
                        <li class="folder-link icon-link"><a href="http://bioinformatics.oxfordjournals.org/cgi/folders?action=addtofolder&amp;wherefrom=JOURNALS&amp;wrapped_id=bioinfo;28/18/i311">Add to my archive</a></li>
                        <li class="cit-man-link icon-link cite-link"><a href="http://bioinformatics.oxfordjournals.org/citmgr?gca=bioinfo;28/18/i311">Download citation</a></li>
                        <li><a class="request-permissions" href="https://s100.copyright.com/AppDispatchServlet?publisherName=oup&amp;publication=bioinformatics&amp;title=Telescoper%3A%20de%20novo%20assembly%20of%20highly%20repetitive%20regions%3A%20&amp;publicationDate=09%2F15%2F2012&amp;author=Ma%27ayan%20Bresler%2C%20Sara%20Sheehan%2C%20Andrew%20H.%20Chan%2C%20Yun%20S.%20Song&amp;copyright=Oxford%20University%20Press&amp;contentID=10.1093%2Fbioinformatics%2Fbts399&amp;volumeNum=28&amp;issueNum=18&amp;startPage=i311&amp;endPage=i317&amp;orderBeanReset=true" target="_blank">Request Permissions</a></li>
                     </ol>
                  </div>
                  <div class="cb-section default-closed collapsed" id="cb-letter-cit">
                     <h4 class="cb-section-header"><a href="#" class="collapse-toggle"><span class="view-more">+</span> <span>Responses</span></a></h4>
                     <ol style="display: none;">
                        <li><a href="http://bioinformatics.oxfordjournals.org/letters/submit/bioinfo;28/18/i311">Submit a response</a></li>
                        <li><span class="eletters-no-responses-published">No responses published</span></li>
                     </ol>
                  </div>
                  <div class="cb-section default-closed collapsed" id="cb-art-cit">
                     <h4 class="cb-section-header"><a href="#" class="collapse-toggle"><span class="view-more">+</span> <span>Citing Articles</span></a></h4>
                     <ol style="display: none;">
                        <li class="hw-citing-link nodata"><div id="cb-loaded-hw-cited-none">No citing articles</div></li>
                        <li><a title="[opens in a new window]" target="_blank" href="http://bioinformatics.oxfordjournals.org/cgi/crossref-forward-links/bioinfo/28/18/i311" rel="external-nw" id="cb-crossref-citing-articles">Citing articles via CrossRef</a></li>
                        <li class="nodata"><div id="cb-loaded-scopus-cited-none">No Scopus citing articles</div></li>
                        <li class="nodata"><div id="cb-loaded-isi-cited-none">No Web of Science citing articles</div></li>
                     </ol>
                  </div>
                  <div class="cb-section default-closed collapsed" id="cb-art-gs">
                     <h4 class="cb-section-header"><a href="#" class="collapse-toggle"><span class="view-more">+</span> <span>Google Scholar</span></a></h4>
                     <ol style="display: none;">
                        <li class="cb-art-gs-auth author-link"><a href="http://scholar.google.com/scholar?q=%22author%3ABresler%20author%3AM.%22" class="cb-art-gs-auth author-link">Articles by  Bresler, M.</a></li>
                        <li class="cb-art-gs-auth author-link"><a href="http://scholar.google.com/scholar?q=%22author%3ASong%20author%3AY.S.%22" class="cb-art-gs-auth author-link">Articles by  Song, Y. S.</a></li>
                     </ol>
                  </div>
                  <div class="cb-section default-closed collapsed" id="cb-art-pm">
                     <h4 class="cb-section-header"><a href="#" class="collapse-toggle"><span class="view-more">+</span> <span>PubMed</span></a></h4>
                     <ol style="display: none;">
                        <li class="cb-art-pm-cite cite-link icon-link"><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=22962446&amp;link_type=PUBMED" class="cb-art-pm-cite cite-link icon-link">PubMed citation</a></li>
                        <li class="cb-art-pm-auth author-link"><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=Bresler%20M&amp;link_type=AUTHORSEARCH" class="cb-art-pm-auth author-link">Articles by  Bresler, M.</a></li>
                        <li class="cb-art-pm-auth author-link"><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=Sheehan%20S&amp;link_type=AUTHORSEARCH" class="cb-art-pm-auth author-link">Articles by  Sheehan, S.</a></li>
                        <li class="cb-art-pm-auth author-link"><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=Chan%20AH&amp;link_type=AUTHORSEARCH" class="cb-art-pm-auth author-link">Articles by  Chan, A. H.</a></li>
                        <li class="cb-art-pm-auth author-link"><a href="http://bioinformatics.oxfordjournals.org/external-ref?access_num=Song%20YS&amp;link_type=AUTHORSEARCH" class="cb-art-pm-auth author-link">Articles by  Song, Y. S.</a></li>
                     </ol>
                  </div>
                  <div class="cb-section default-closed collapsed nodata" id="cb-art-rel">
                     <h4 class="cb-section-header"><a href="#" class="collapse-toggle"><span class="view-more">+</span> <span>Related Content</span></a></h4>
                     <ol style="display: none;">
                        <li class="nodata"><div id="cb-loaded-related-urls-none">No related web pages</div></li>
                     </ol>
                  </div>
                  <div class="cb-section collapsible" id="cb-art-soc">
                     <h4 class="cb-section-header"><a href="#" class="collapse-toggle"><span class="view-more">-</span> <span>Share</span></a></h4>
                     <ol>
                        <li><a title="email this" href="http://bioinformatics.oxfordjournals.org/email?gca=bioinfo;28/18/i311&amp;current-view-path=/content/28/18/i311.full"></a></li>
                        <li>
                           <div class="social-bookmarking">
                              
                              <ul class="social-bookmark-links">
                                 <li class="social-bookmarking-item social-bookmarking-item-citeulike"><a target="_BLANK" href="http://bioinformatics.oxfordjournals.org/external-ref?tag_url=http://bioinformatics.oxfordjournals.org/cgi/content/long/28/18/i311&amp;title=Telescoper%3A%20de%20novo%20assembly%20of%20highly%20repetitive%20regions+--+Bresler%20et%20al.%2028%20%2818%29%3A%20i311+--+bioinformatics&amp;doi=10.1093/bioinformatics/bts399&amp;link_type=CITEULIKE"><img src="bioinformaticsTest_files/citeulike.gif" alt="Add to CiteULike" title="CiteULike"></a><span class="soc-bm-link-text">CiteULike</span></li>
                                 <li class="social-bookmarking-item social-bookmarking-item-connotea"><a target="_BLANK" href="http://bioinformatics.oxfordjournals.org/external-ref?tag_url=http://bioinformatics.oxfordjournals.org/cgi/content/long/28/18/i311&amp;title=Telescoper%3A%20de%20novo%20assembly%20of%20highly%20repetitive%20regions+--+Bresler%20et%20al.%2028%20%2818%29%3A%20i311+--+bioinformatics&amp;doi=10.1093/bioinformatics/bts399&amp;link_type=CONNOTEA"><img src="bioinformaticsTest_files/connotea.gif" alt="Add to Connotea" title="Connotea"></a><span class="soc-bm-link-text">Connotea</span></li>
                                 <li class="social-bookmarking-item social-bookmarking-item-delicious"><a target="_BLANK" href="http://bioinformatics.oxfordjournals.org/external-ref?tag_url=http://bioinformatics.oxfordjournals.org/cgi/content/long/28/18/i311&amp;title=Telescoper%3A%20de%20novo%20assembly%20of%20highly%20repetitive%20regions+--+Bresler%20et%20al.%2028%20%2818%29%3A%20i311+--+bioinformatics&amp;doi=10.1093/bioinformatics/bts399&amp;link_type=DEL_ICIO_US"><img src="bioinformaticsTest_files/delicious.gif" alt="Add to Delicious" title="Delicious"></a><span class="soc-bm-link-text">Delicious</span></li>
                                 <li class="social-bookmarking-item social-bookmarking-item-facebook"><a title="[opens in a new window]" target="_BLANK" href="http://bioinformatics.oxfordjournals.org/external-ref?tag_url=http://bioinformatics.oxfordjournals.org/cgi/content/short/28/18/i311&amp;title=Telescoper%3A%20de%20novo%20assembly%20of%20highly%20repetitive%20regions+--+Bresler%20et%20al.%2028%20%2818%29%3A%20i311+--+bioinformatics&amp;doi=10.1093/bioinformatics/bts399&amp;link_type=FACEBOOK" class="sb-facebook" rel="external-nw"><img src="bioinformaticsTest_files/facebook.gif" alt="Add to Facebook" title="Facebook"></a><span class="soc-bm-link-text">Facebook</span></li>
                                 <li class="social-bookmarking-item social-bookmarking-item-googleplus"><div id="___plusone_0" style="height: 15px; width: 24px; display: inline-block; text-indent: 0px; margin: 0px; padding: 0px; background: none repeat scroll 0% 0% transparent; border-style: none; float: none; line-height: normal; font-size: 1px; vertical-align: baseline;"><iframe title="+1" src="bioinformaticsTest_files/fastbutton.htm" name="I0_1348190395935" id="I0_1348190395935" vspace="0" tabindex="0" style="position: static; top: 0px; width: 24px; margin: 0px; border-style: none; left: 0px; visibility: visible; height: 15px;" marginwidth="0" marginheight="0" hspace="0" allowtransparency="true" frameborder="0" scrolling="no" width="100%"></iframe></div><a style="display: none;" target="_BLANK" href="http://bioinformatics.oxfordjournals.org/external-ref?tag_url=http://bioinformatics.oxfordjournals.org/cgi/content/long/28/18/i311&amp;title=Telescoper%3A%20de%20novo%20assembly%20of%20highly%20repetitive%20regions+--+Bresler%20et%20al.%2028%20%2818%29%3A%20i311+--+bioinformatics&amp;doi=10.1093/bioinformatics/bts399&amp;link_type=GOOGLEPLUS&amp;log_only=yes"><img src="bioinformaticsTest_files/googleplus.jpg" alt="Add to Google+" title="Google+"></a><span class="soc-bm-link-text">Google+</span></li>
                                 <li class="social-bookmarking-item social-bookmarking-item-mendeley"><a target="_BLANK" href="http://bioinformatics.oxfordjournals.org/external-ref?tag_url=http://bioinformatics.oxfordjournals.org/cgi/content/long/28/18/i311&amp;title=Telescoper%3A%20de%20novo%20assembly%20of%20highly%20repetitive%20regions+--+Bresler%20et%20al.%2028%20%2818%29%3A%20i311+--+bioinformatics&amp;doi=10.1093/bioinformatics/bts399&amp;link_type=MENDELEY"><img src="bioinformaticsTest_files/mendeley.gif" alt="Add to Mendeley" title="Mendeley"></a><span class="soc-bm-link-text">Mendeley</span></li>
                                 <li class="social-bookmarking-item social-bookmarking-item-twitter"><a target="_BLANK" href="http://bioinformatics.oxfordjournals.org/external-ref?tag_url=http://bioinformatics.oxfordjournals.org/cgi/content/long/28/18/i311&amp;title=Telescoper%3A%20de%20novo%20assembly%20of%20highly%20repetitive%20regions+--+Bresler%20et%20al.%2028%20%2818%29%3A%20i311+--+bioinformatics&amp;doi=10.1093/bioinformatics/bts399&amp;link_type=TWITTER"><img src="bioinformaticsTest_files/twitter.gif" alt="Add to Twitter" title="Twitter"></a><span class="soc-bm-link-text">Twitter</span></li>
                              </ul>
                              
                              <p class="social-bookmarking-help"><a href="http://bioinformatics.oxfordjournals.org/help/social_bookmarks.dtl">What's this?</a></p>
                              
                           </div>
                        </li>
                     </ol>
                  </div>
               </div>
            </div>
            
            
            <div class="content-box" id="article-dyn-nav">
               <div class="cb-contents">
                  <h3 class="cb-contents-header"><span>Navigate This Article</span></h3>
                  <div class="cb-section" id="cb-art-nav">
                     <ol>
                        <li><a href="#content-block">Top</a></li>
                        <li><a href="#abstract-1">Abstract</a></li>
                        <li><a href="#sec-1">1 INTRODUCTION</a></li>
                        <li><a href="#sec-2">2 TERMINOLOGY</a></li>
                        <li><a href="#sec-3">3 METHODS</a></li>
                        <li><a href="#sec-8">4 EMPIRICAL RESULTS</a></li>
                        <li><a href="#sec-11">5 CONCLUSION</a></li>
                        <li><a href="#ack-1">ACKNOWLEDGMENTS</a></li>
                        <li><a href="#ref-list-1">REFERENCES</a></li>
                     </ol>
                  </div>
               </div>
            </div>
            
            <div class="content-box" id="article-cb-main">
               
               <div class="cb-contents">
                  
                  <div class="cb-section collapsible default-open" id="cb-art-bn">
                     
                     <h4 class="cb-section-header"><a href="#" class="collapse-toggle"><span class="view-more">-</span> <span>People also read [Beta]</span></a></h4>
                     
                     <ul>
                        <li>
                           
                           <div id="bnArticleRecs"><h4 class="cb-contents-header">
<span style="margin-top:10px;"></span>
</h4><ol style="padding-right:5px;">
<li>
<div id="bn_g_resultbnArticleRecs_1" class="bn_g_result bn_g_result" style="margin:0 auto; padding:0;">
<div class="bn_g_result_title" style="margin:0 auto; padding:0 5px 4px 0;">
<a class="bn_g_result_link" baynote_req="ArticlePage" baynote_guide="NextStepMatch" baynote_bnrank="1" href="http://bioinformatics.oxfordjournals.org/cgi/doi/10.1093/bioinformatics/bts414">Long read alignment based on maximal exact match seeds<div class="bn_mouseoverdiv">
<span class="bn_journaltitle"><b>Bioinformatics</b> - 2012</span>
<br>
<span class="bn_authorslist"></span>
<br>
<span class="bn_viewCount">76 readers also viewed this
                     </span>
</div>
</a>
</div>
</div>
<div id="bn_g_resultbnArticleRecs_2" class="bn_g_result bn_g_result" style="margin:0 auto; padding:0;">
<div class="bn_g_result_title" style="margin:0 auto; padding:0 5px 4px 0;">
<a class="bn_g_result_link" baynote_req="ArticlePage" baynote_guide="NextStepMatch" baynote_bnrank="2" href="http://bioinformatics.oxfordjournals.org/cgi/doi/10.1093/bioinformatics/bts380">Indel-tolerant read mapping with trinucleotide frequencies using cache-oblivious kd-trees<div class="bn_mouseoverdiv">
<span class="bn_journaltitle"><b>Bioinformatics</b> - 2012</span>
<br>
<span class="bn_authorslist"></span>
<br>
<span class="bn_viewCount">37 readers also viewed this
                     </span>
</div>
</a>
</div>
</div>
<div id="bn_g_resultbnArticleRecs_3" class="bn_g_result bn_g_result" style="margin:0 auto; padding:0;">
<div class="bn_g_result_title" style="margin:0 auto; padding:0 5px 4px 0;">
<a class="bn_g_result_link" baynote_req="ArticlePage" baynote_guide="NextStepMatch" baynote_bnrank="3" href="http://bioinformatics.oxfordjournals.org/cgi/doi/10.1093/bioinformatics/bts378">DELLY: structural variant discovery by integrated paired-end and split-read analysis<div class="bn_mouseoverdiv">
<span class="bn_journaltitle"><b>Bioinformatics</b> - 2012</span>
<br>
<span class="bn_authorslist"></span>
<br>
<span class="bn_viewCount">95 readers also viewed this
                     </span>
</div>
</a>
</div>
</div>
</li>
</ol>
</div>
                           
                        </li>
                        <li>
                           
                           <p id="bnHelp">
                              <a href="http://www.oxfordjournals.org/our_journals/help.html">What's this?</a>
                              
                           </p>
                           
                        </li>
                     </ul>
                     
                  </div>
                  
               </div>
               
            </div>
            
            
         <div style="display: block;" id="docked-nav" class="dockblock"><div class="content-box" id="docked-cb"><div class="cb-contents"><h3>This Article</h3><div class="cb-section cb-slug"><ol id="docked-slug"><li><div id="slugline">
                              
                              <cite>
                                 <abbr title="Bioinformatics" class="slug-jnl-abbrev">
                                    <nlm:abbrev-journal-title xmlns:nlm="http://schema.highwire.org/NLM/Journal" abbrev-type="publisher">Bioinformatics</nlm:abbrev-journal-title></abbr><span class="slug-pub-date" itemprop="datePublished">
                                    (2012) 
                                    </span>
                                 
                                 <span class="slug-vol">
                                    28
                                    </span><span class="slug-issue">
                                    (18):
                                    </span><span class="slug-pages">
                                    i311-i317.
                                    </span>
                                 <span class="slug-doi-wrapper">
                                    
                                    doi:
                                    
                                    <span title="10.1093/bioinformatics/bts399" class="slug-doi">10.1093/bioinformatics/bts399</span>
                                    </span>
                                 </cite>
                              
                              
                              
                              
                              
                              
                              
                              
                              
                              <span class="issue-title">This article appears in: <a href="http://bioinformatics.oxfordjournals.org/content/28/18.toc" title="Table of Contents">ECCB 2012 PROCEEDINGS PAPERS COMMITTEE SEPTEMBER 9 TO SEPTEMBER 12, 2012, CONFERENCE CENTER BASEL, SWITZERLAND</a></span>
                              
                              
                           </div></li></ol></div><div class="cb-section"><ol id="docked-nav-views"><li><span class="creative-commons-article">This article is Open Access<span>OA</span></span></li><li class="abstract-view-link primary"><a href="http://bioinformatics.oxfordjournals.org/content/28/18/i311.abstract" rel="view-abstract">Abstract</a><span class="free">Free</span></li><li class="notice full-text-view-link"><span class="variant-indicator">» <span>Full Text (HTML)</span></span><span class="free">Free</span></li><li class="notice full-text-pdf-view-link"><a href="http://bioinformatics.oxfordjournals.org/content/28/18/i311.full.pdf+html" rel="view-full-text.pdf">Full Text (PDF)</a><span class="free">Free</span></li></ol></div></div></div><div class="content-box" id="article-dyn-nav">
               <div class="cb-contents">
                  <h3 class="cb-contents-header"><span>Navigate This Article</span></h3>
                  <div class="cb-section" id="cb-art-nav">
                     <ol>
                        <li><a href="#content-block">Top</a></li>
                        <li><a href="#abstract-1">Abstract</a></li>
                        <li><a href="#sec-1">1 INTRODUCTION</a></li>
                        <li><a href="#sec-2">2 TERMINOLOGY</a></li>
                        <li><a href="#sec-3">3 METHODS</a></li>
                        <li><a href="#sec-8">4 EMPIRICAL RESULTS</a></li>
                        <li><a href="#sec-11">5 CONCLUSION</a></li>
                        <li><a href="#ack-1">ACKNOWLEDGMENTS</a></li>
                        <li><a href="#ref-list-1">REFERENCES</a></li>
                     </ol>
                  </div>
               </div>
            </div></div></div>
         <div style="height: 15164px;" id="col-3">
            
            <div id="col-3-qs" class="sidebar-qs">
               
               
               <form class="searchbox" action="/search" method="get">
                  <p>Search this journal:</p>
                  <div>
                     	
                     	
                     	<input name="fulltext" id="header-qs-input" type="text">
                     <input name="submit" value="yes" type="hidden">
                     <input class="inline_button" alt="Go" src="bioinformaticsTest_files/go.gif" type="image">
                     
                     <div class="adv-search-link"><a href="http://bioinformatics.oxfordjournals.org/search">Advanced »</a></div>
                     
                     
                  </div>
               </form>
               
            </div>
            <div class="cb-contents">
               
               <div id="sidebar-current-issue" class="content-box">
                  	
                  <h3 class="cb-contents-header">Current Issue</h3>
                  	
                  <div class="cb-section">
                     	  
                     <ol>
                        <li>
                           	      <a href="http://bioinformatics.oxfordjournals.org/content/current"><span>
                                 		  <span class="current-issue-date">
                                    September 15, 2012
                                    </span><span class="toc-citation-volume">
                                    
                                    28
                                    </span><span class="toc-citation-issue">(18)</span>
                                 		  
                                 		</span></a>
                           	    
                        </li>
                     </ol>
                     	
                  </div>
                  	
                  <div class="cb-section">
                     	  
                     <ol>
                        <li>
                           <a href="http://bioinformatics.oxfordjournals.org/content/current"><img src="bioinformaticsTest_files/18.gif" alt="Bioinformatics" id="cover"></a>
                           
                        </li>
                     </ol>
                     	
                  </div>
                  	
                  <div class="cb-section sidebar-etoc-link">
                     	  
                     <ol>
                        <li><a href="http://bioinformatics.oxfordjournals.org/cgi/alerts/etoc">Alert me to new issues</a></li>
                     </ol>
                     	
                  </div>
                  
               </div>
               
            </div>
            
            <div id="second">
               
               <div class="features">
                  
                  <div class="feature separator_after first">
                     
                     <h2 id="the_journal" title="The Journal"><span>The Journal</span></h2>
                     
                     <ul class="emphasised">
                        <li>
                           <a href="http://www.oxfordjournals.org/bioinformatics/about.html" title="About this journal">About this journal</a>
                           
                        </li>
                        <li>
                           <a href="http://bioinformatics.oxfordjournals.org/cgi/calendarcontent/index.html?target=hw.dtl" title="Calendar of events">Calendar of events</a>
                           
                        </li>
                        <li>
                           <a href="http://www.oxfordjournals.org/access_purchase/rights_permissions.html" title="Rights &amp; Permissions">Rights &amp; Permissions</a>
                           
                        </li>
                        <li>
                           <a href="http://www.oxfordjournals.org/access_purchase/dispatch_dates.html#CABIOSP" title="Dispatch date of the next issue">Dispatch date of the next issue</a>
                           
                        </li>
                        <li>
                           <a href="http://publicationethics.org/about" title="This journal is a member of the Committee on Publication Ethics (COPE)">This journal is a member of the Committee on Publication Ethics (COPE)</a>
                           
                        </li>
                        <li>
                           <a href="http://bioinformatics.oxfordjournals.org/cgi/eletters?lookup=by_date&amp;days=84&amp;target=hw.dtl" title="Recent Comments">Recent Comments</a>
                           
                        </li>
                        <li>
                           <a href="http://www.oxfordjournals.org/mobile_faqs.html" title="We are mobile – find out more">We are mobile – find out more</a>
                           
                        </li>
                     </ul>
                     <a href="http://www.oxfordjournals.org/page/4638/1">
                        <img alt="Next Generation Sequencing " src="bioinformaticsTest_files/nextgensuq_sidebar_2012.jpg">
                        </a>
                     
                  </div>
                  
                  <div class="feature separator_after">
                     
                     <h3>An official journal of</h3>
                     
                     <ul class="emphasised">
                        <li>
                           <a href="http://www.iscb.org/" title="The International Society for Computational Biology">The International Society for Computational Biology</a>
                           
                        </li>
                     </ul>
                     
                  </div>
                  
                  <div class="feature separator_after">
                     
                     <h3>Impact factor:  5.468</h3>
                     
                  </div>
                  
                  <div class="feature">
                     
                     <h3>Editors-in-Chief</h3>
                     
                     <div class="contact">
                        
                        <h4>A Bateman</h4>
                        
                     </div>
                     
                     <div class="contact">
                        
                        <h4>A Valencia</h4>
                        
                     </div>
                     
                     <ul class="emphasised">
                        <li>
                           <a href="http://www.oxfordjournals.org/bioinformatics/editorial_board.html" title="View full editorial board">View full editorial board</a>
                           
                        </li>
                     </ul>
                     
                  </div>
                  
                  <div class="feature">
                     
                     <h2 id="for_authors" title="For Authors"><span>For Authors</span></h2>
                     
                     <ul class="emphasised">
                        <li>
                           <a href="http://www.oxfordjournals.org/bioinformatics/for_authors/index.html" title="Instructions to authors">Instructions to authors</a>
                           
                        </li>
                        <li>
                           <a href="http://www.oxfordjournals.org/our_journals/bioinformatics/for_authors/submission_online.html" title="Online submission">Online submission</a>
                           
                        </li>
                        <li>
                           <a href="http://mc.manuscriptcentral.com/bioinformatics" title="Submit Now!">Submit Now!</a>
                           
                        </li>
                        <li>
                           <a href="http://www.oxfordjournals.org/access_purchase/self-archiving_policyb.html" title="Self-archiving policy">Self-archiving policy</a>
                           
                        </li>
                     </ul>
                     <a href="http://www.oxfordjournals.org/oxfordopen/">
                        <img alt="Open access options for authors - visit Oxford Open" src="bioinformaticsTest_files/oxford_open_small.gif">
                        </a>
                     
                     <p>Open access options for authors - visit <a href="http://www.oxfordjournals.org/oxfordopen/">Oxford Open</a>
                        
                     </p>
                     <a href="http://publicaccess.nih.gov/submit_process_journals.htm">
                        <img alt="PMC Logo" src="bioinformaticsTest_files/pmc-logo.gif">
                        </a>
                     
                     <ul class="emphasised">
                        <li>
                           <a href="http://publicaccess.nih.gov/submit_process_journals.htm" title="This journal enables compliance with the NIH Public Access Policy">This journal enables compliance with the NIH Public Access Policy</a>
                           
                        </li>
                     </ul>
                     
                  </div>
                  
                  <div class="feature">
                     
                     <h2 id="alerting_services" title="Alerting Services"><span>Alerting Services</span></h2>
                     
                     <ul class="emphasised">
                        <li>
                           <a href="http://bioinformatics.oxfordjournals.org/cgi/alerts/etoc" title="Email table of contents">Email table of contents</a>
                           
                        </li>
                        <li>
                           <a href="http://bioinformatics.oxfordjournals.org/cgi/alerts/etoc" title="Email Advance Access">Email Advance Access</a>
                           
                        </li>
                        <li>
                           <a href="http://bioinformatics.oxfordjournals.org/help/citetrack" title="CiteTrack">CiteTrack</a>
                           
                        </li>
                        <li>
                           <a href="http://bioinformatics.oxfordjournals.org/rss" title="XML RSS feed">XML RSS feed</a>
                           
                        </li>
                     </ul>
                     
                  </div>
                  
                  <div class="feature">
                     
                     <h2 id="corporate_services" title="Corporate Services"><span>Corporate Services</span></h2>
                     
                     <ul class="emphasised">
                        <li>
                           <a href="http://www.oxfordjournals.org/corporate_services/advertising.html" title="Advertising sales">Advertising sales</a>
                           
                        </li>
                        <li>
                           <a href="http://www.oxfordjournals.org/corporate_services/reprints.html" title="Reprints">Reprints</a>
                           
                        </li>
                        <li>
                           <a href="http://www.oxfordjournals.org/corporate_services/supplements.html" title="Supplements">Supplements</a>
                           
                        </li>
                     </ul>
                     
                  </div>
                  
                  <div class="feature">
                     
                     <h2>Widget</h2>
                     
                     <ul class="emphasised">
                        <li>
                           <a href="http://www.oxfordjournals.org/our_journals/bioinformatics/widget.html" title="Get a widget">Get a widget</a>
                           
                        </li>
                     </ul>
                     
                  </div>
                  
               </div>
               
               <div class="ad_unhidden" id="oas_right1">
                  <iframe id="id_advertframe_right1" src="bioinformaticsTest_files/advert_002.htm" class="resize_ad" frameborder="0" height="600" scrolling="no" width="160"></iframe>
                  
               </div>
               
            </div>
            
            <div class="most-links-box ">
               
               <div class="most-header">
                  
                  <h3>Most</h3><ul class="most-headings"><li class="most-cur-sel"><a href="#">Most Read</a></li><li><a href="#">Most Cited</a></li></ul>
                  
               </div>
               
               <ul>
                  <li class="most-cur-sel">
                     <h4>Most Read</h4>
                     	  
                     <div class="most-list">
                        	    
                        <ol>
                           <li class="first-item"><a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/28/16/2097?rss=1&amp;amp;ssource=mfr">Genomic dark matter: the reliability of short read mapping illustrated by the genome mappability score</a></li>
                           <li><a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/bts477v1?rss=1&amp;amp;ssource=mfr">easyRNASeq: a bioconductor package for processing RNA-Seq data</a></li>
                           <li><a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/bts494v1?rss=1&amp;amp;ssource=mfr">BioJava: an open-source framework for bioinformatics in 2012</a></li>
                           <li><a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/28/16/2093?rss=1&amp;amp;ssource=mfr">Performance of computational tools in evaluating the functional impact of laboratory-induced amino acid mutations</a></li>
                           <li class="last-item"><a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/28/8/1086?rss=1&amp;amp;ssource=mfr">Oases: robust de novo RNA-seq assembly across the dynamic range of expression levels</a></li>
                        </ol>
                        	    <a href="http://bioinformatics.oxfordjournals.org/reports/mfr1.dtl" class="view-all">» View all Most Read articles</a>
                        	  
                     </div>
                     	
                  </li>
                  <li>
                     <h4>Most Cited</h4>
                     	
                     <div class="most-list">
                        	  
                        <ol>
                           <li class="first-item"><a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/27/3/431?rss=1&amp;amp;ssource=mfc">Cytoscape 2.8: new features for data integration and network visualization</a></li>
                           <li><a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/26/18/2347?rss=1&amp;amp;ssource=mfc">Cytoscape Web: an interactive web-based network browser</a></li>
                           <li><a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/26/19/2462?rss=1&amp;amp;ssource=mfc">RDP3: a flexible and fast computer program for analyzing recombination</a></li>
                           <li><a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/26/19/2460?rss=1&amp;amp;ssource=mfc">Search and clustering orders of magnitude faster than BLAST</a></li>
                           <li class="last-item"><a href="http://bioinformatics.oxfordjournals.org/cgi/content/short/26/18/2336?rss=1&amp;amp;ssource=mfc">LocusZoom: regional visualization of genome-wide association scan results</a></li>
                        </ol>
                        	  <a href="http://bioinformatics.oxfordjournals.org/reports/mfc1.dtl" class="view-all">» View all Most Cited articles</a>
                        	
                     </div>
                     
                  </li>
               </ul>
               
            </div>
            
            
            
         </div>
         
         
         
         <div id="oas_bottom" class="ad_unhidden">
            <iframe id="id_advertframe_bottom" src="bioinformaticsTest_files/advert_003.htm" class="resize_ad" frameborder="0" height="90" scrolling="no" width="728"></iframe>
            
         </div>
         
         
         
         
         
         
         <div id="secondary_footer">
            
            <div id="issn">Online ISSN 1460-2059 - Print ISSN 1367-4803</div>
            
            
            <div id="copyright"><a href="http://www.oxfordjournals.org/our_journals/terms.html">Copyright ©</a>&nbsp;
               <span>2012</span>&nbsp;Oxford University Press
               
            </div>
            
         </div>
         
         <div id="primary_footer">
            
            <div id="site_logo">
               <strong><span>Oxford Journals</span></strong>
               <em><span>Oxford University Press</span></em>
               
            </div>
            
            
            <div id="third_nav">
               
               <ul>
                  <li><a href="http://www.oxfordjournals.org/site_map.html">Site Map</a></li>
                  <li><a href="http://www.oup.co.uk/privacy/">Privacy Policy</a></li>
                  <li><a href="http://www.oxfordjournals.org/faq/">Frequently Asked Questions</a></li>
               </ul>
               
            </div>
            
            
            <div id="other_oup_sites">
               
               <form action="http://www.oxfordjournals.org/service/redirect">
                  <div>
                     <label for="site_list">Other Oxford University Press sites:</label>
                     <select name="url" id="site_list">
                        <option selected="selected" value="http://www.oup.com">Oxford University Press</option>
                        <option value="http://www.oxfordjournals.org/china/">Oxford Journals China</option>
                        <option value="http://www.oxfordjournals.org/japan/">Oxford Journals Japan</option>
                        <option value="http://www.oup.com/uk/academic">Academic &amp; Professional books</option>
                        <option value="http://www.oup.com/oxed">Children's &amp; Schools Books</option>
                        <option value="http://www.oup.com/uk/dictionaries">Dictionaries &amp; Reference</option>
                        <option value="http://www.oup.co.uk/newdnb/">Dictionary of National Biography</option>
                        <option value="http://www.oup.com/uk/digital_reference">Digital Reference</option>
                        <option value="http://www.oup.com/elt/">English Language Teaching</option>
                        <option value="http://www.oup.com/uk/textbooks">Higher Education Textbooks</option>
                        <option value="http://www.oup.co.uk/ieu/">International Education Unit</option>
                        <option value="http://www.oup.com/uk/law">Law</option>
                        <option value="http://www.oup.com/uk/medicine">Medicine</option>
                        <option value="http://www.oup.com/uk/music">Music</option>
                        <option value="http://www.oup.com/online">Online Products &amp; Publishing</option>
                        <option value="http://oxfordbibliographiesonline.com">Oxford Bibliographies Online</option>
                        <option value="http://oxforddictionaries.com">Oxford Dictionaries Online</option>
                        <option value="http://www.oed.com">Oxford English Dictionary</option>
                        <option value="http://www.oxfordlanguagedictionaries.com">Oxford Language Dictionaries Online</option>
                        <option value="http://www.oxfordscholarship.com">Oxford Scholarship Online</option>
                        <option value="http://www.oup.com/uk/reference">Reference</option>
                        <option value="http://www.oup.com/uk/rights">Rights and Permissions</option>
                        <option value="http://www.oup.com/uk/booksellers">Resources for Retailers &amp; Wholesalers</option>
                        <option value="http://www.oup.com/uk/corporate">Resources for the Healthcare Industry</option>
                        <option value="http://www.oup.com/vsi">Very Short Introductions</option>
                        <option value="http://www.oup.com/worldsclassics">World's Classics</option></select>
                     
                     <input src="bioinformaticsTest_files/go_002.gif" alt="Go" class="inline_button" type="image">
                     
                  </div>
               </form>
               
            </div>
            
         </div>
         <script type="text/javascript" src="bioinformaticsTest_files/hw-publisher-baynote.js" defer="defer"></script><script type="text/javascript">
        var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
        document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
        </script><script src="bioinformaticsTest_files/ga.js" type="text/javascript"></script><script type="text/javascript">
        try {
        var pageTracker = _gat._getTracker("UA-189672-16");
        pageTracker._setDomainName(".oxfordjournals.org");
        pageTracker._trackPageview();
        } catch(err) {}
        </script></div>
   
<div id="fancybox-tmp"></div><div id="fancybox-loading"><div></div></div><div id="fancybox-overlay"></div><div id="fancybox-wrap"><div id="fancybox-outer"><div class="fancybox-bg" id="fancybox-bg-n"></div><div class="fancybox-bg" id="fancybox-bg-ne"></div><div class="fancybox-bg" id="fancybox-bg-e"></div><div class="fancybox-bg" id="fancybox-bg-se"></div><div class="fancybox-bg" id="fancybox-bg-s"></div><div class="fancybox-bg" id="fancybox-bg-sw"></div><div class="fancybox-bg" id="fancybox-bg-w"></div><div class="fancybox-bg" id="fancybox-bg-nw"></div><div id="fancybox-content"></div><a id="fancybox-close"></a><div id="fancybox-title"></div><a href="javascript:;" id="fancybox-left"><span class="fancy-ico" id="fancybox-left-ico"></span></a><a href="javascript:;" id="fancybox-right"><span class="fancy-ico" id="fancybox-right-ico"></span></a></div></div><img style="display: none;" src="bioinformaticsTest_files/listener2_002.bmp"><img style="display: none;" src="bioinformaticsTest_files/listener2.bmp"><img style="display: none;" src="bioinformaticsTest_files/listener2_003.bmp"></body></html>